<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Nat Commun</span> <span class="iso-abbrev" title="iso-abbrev">Nat Commun</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Nature Communications</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2041-1723</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/4911614">4911614</a> </span> <span class="pii" title="pii">ncomms11842</span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/ncomms11842">10.1038/ncomms11842</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27296726">27296726</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Full circumpolar migration ensures evolutionary unity in the Emperor penguin</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cristofari</span> <span class="given-names" tagx="given-names" title="given-names">Robin</span> </span> <a href="#c1">a</a> <a href="#a1">1</a> <a href="#a2">2</a> <a href="#a3">3</a> <a href="#a4">4</a> <a href="#a5">5</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bertorelle</span> <span class="given-names" tagx="given-names" title="given-names">Giorgio</span> </span> <a href="#a6">6</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ancel</span> <span class="given-names" tagx="given-names" title="given-names">André</span> </span> <a href="#a1">1</a> <a href="#a2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Benazzo</span> <span class="given-names" tagx="given-names" title="given-names">Andrea</span> </span> <a href="#a6">6</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le Maho</span> <span class="given-names" tagx="given-names" title="given-names">Yvon</span> </span> <a href="#a1">1</a> <a href="#a2">2</a> <a href="#a3">3</a> <a href="#a4">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ponganis</span> <span class="given-names" tagx="given-names" title="given-names">Paul J.</span> </span> <a href="#a7">7</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stenseth</span> <span class="given-names" tagx="given-names" title="given-names">Nils Chr</span> </span> <a href="#a5">5</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trathan</span> <span class="given-names" tagx="given-names" title="given-names">Phil N.</span> </span> <a href="#a8">8</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whittington</span> <span class="given-names" tagx="given-names" title="given-names">Jason D.</span> </span> <a href="#a3">3</a> <a href="#a4">4</a> <a href="#a5">5</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zanetti</span> <span class="given-names" tagx="given-names" title="given-names">Enrico</span> </span> <a href="#a6">6</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zitterbart</span> <span class="given-names" tagx="given-names" title="given-names">Daniel P.</span> </span> <a href="#a9">9</a> <a href="#a10">10</a> <a href="#a11">11</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le Bohec</span> <span class="given-names" tagx="given-names" title="given-names">Céline</span> </span> <a href="#c2">b</a> <a href="#a1">1</a> <a href="#a2">2</a> <a href="#a3">3</a> <a href="#a4">4</a> <a href="#n1">*</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trucchi</span> <span class="given-names" tagx="given-names" title="given-names">Emiliano</span> </span> <a href="#c3">c</a> <a href="#a5">5</a> <a href="#a12">12</a> <a href="#n1">*</a> </span> <span class="citation_author_institution" id="a1">[1], <span class="institution" title="institution">Université de Strasbourg (UdS), Institut Pluridisciplinaire Hubert Curien</span> <span class="country" tagx="country" title="country">France</span> </span> <span class="citation_author_institution" id="a2">[2], <span class="institution" title="institution">Centre National de la Recherche Scientifique (CNRS), UMR 7178</span> <span class="country" tagx="country" title="country">France</span> </span> <span class="citation_author_institution" id="a3">[3], <span class="institution" title="institution">Centre Scientifique de Monaco (CSM), Département de Biologie Polaire</span> <span class="country" tagx="country" title="country">Principality of Monaco</span> </span> <span class="citation_author_institution" id="a4">[4], <span class="institution" title="institution">Laboratoire International Associé LIA-647 BioSensib, CSM-CNRS-UdS</span> <span class="country" tagx="country" title="country">Principality of Monaco</span> </span> <span class="citation_author_institution" id="a5">[5], <span class="institution" title="institution">Centre for Ecological and Evolutionary Synthesis (CEES), Department of Biosciences, University of Oslo, Postboks, Blindern </span> <span class="country" tagx="country" title="country">Norway</span> </span> <span class="citation_author_institution" id="a6">[6], <span class="institution" title="institution">Department of Life Sciences and Biotechnology, University of Ferrara</span> <span class="country" tagx="country" title="country">Italy</span> </span> <span class="citation_author_institution" id="a7">[7], <span class="institution" title="institution">Center for Marine Biotechnology and Biomedicine, Scripps Institution of Oceanography, University of California San Diego </span> <span class="country" tagx="country" title="country">USA</span> </span> <span class="citation_author_institution" id="a8">[8], <span class="institution" title="institution">British Antarctic Survey, Natutal Environment Research Council, High Cross</span> <span class="country" tagx="country" title="country">UK</span> </span> <span class="citation_author_institution" id="a9">[9], <span class="institution" title="institution">Ocean Acoustics Laboratory, Alfred Wegener Institute, Helmholtz Centre for Polar and Marine Research </span> <span class="country" tagx="country" title="country">Germany</span> </span> <span class="citation_author_institution" id="a10">[10], <span class="institution" title="institution">Applied Ocean Physics and Engineering Department, Woods Hole Oceanographic Institution</span> <span class="country" tagx="country" title="country">USA</span> </span> <span class="citation_author_institution" id="a11">[11], <span class="institution" title="institution">Biophysics Laboratory, Department of Physics, University of Erlangen-Nuremberg</span> <span class="country" tagx="country" title="country">Germany</span> </span> <span class="citation_author_institution" id="a12">[12], <span class="institution" title="institution">Department of Botany, and Biodiversity Research, University of Vienna</span> <span class="country" tagx="country" title="country">Austria</span> </span> </div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">a</span> <span class="email" tagx="email" title="email">robin.cristofari@iphc.cnrs.fr</span> </div> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">b</span> <span class="email" tagx="email" title="email">celine.lebohec@iphc.cnrs.fr</span> </div> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">c</span> <span class="email" tagx="email" title="email">emiliano.trucchi@ibv.uio.no</span> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">*</span> <p>These authors contributed equally to this work</p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-06-06</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2016</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">11842</span> <span class="history" title="history"> <span class="received" title="received">received: 2015-08-21</span> <span class="accepted" title="accepted">accepted: 2016-05-05</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Defining reliable demographic models is essential to understand the threats of ongoing environmental change. Yet, in the most remote and threatened areas, models are often based on the survey of a single population, assuming stationarity and independence in population responses. This is the case for the Emperor penguin <i>Aptenodytes forsteri</i>, a flagship Antarctic species that may be at high risk continent-wide before 2100. Here, using genome-wide data from the whole Antarctic continent, we reveal that this top-predator is organized as one single global population with a shared demography since the late Quaternary. We refute the view of the local population as a relevant demographic unit, and highlight that (i) robust extinction risk estimations are only possible by including dispersal rates and (ii) colony-scaled population size is rather indicative of local stochastic events, whereas the species' response to global environmental change is likely to follow a shared evolutionary trajectory. </p> </div> <div class="abstract" title="abstract"> <p> <span class="inline-graphic" title="inline-graphic" />Delimiting populations is crucial for conserving threatened species. Using genome-wide data from the whole of Antarctica, Cristofari <i>et al.</i> show that Emperor penguins are organised into a single global population that have shared demography since the late Quarternary. </p> </div> </div> </div> <div class="body" title="body"> <p>The rate of ongoing environmental change is now thought to exceed the rate at which most species <a href="#b1">1</a>, including humans <a href="#b2">2</a>, are able to adapt, with significant consequences for their resilience and for ecosystem sustainability. Consequently, an increasing number of studies seek to understand its impact on the world's ecosystems and to predict likely scenarios in response to climate projections <a href="#b3">3</a>, either in order to set up more efficient conservation strategies, or as means to prompt urgent political action <a href="#b4">4</a>. One of the main difficulties of this task lies in the fact that only a handful of species have been monitored for more than a few decades, and, in most cases, our knowledge of their demography is limited to local (that is, population-scale) and short-term (that is, generation-scale) responses <a href="#b5">5</a>. Yet, in order to establish reliable projections, larger-scale population parameters must be integrated into demographic models. Recent developments in high-throughput sequencing allow the analysis of genome-wide and population-scale data and provide a genomic insight into the important demographic parameters that can be used to accurately predict species responses to global change <a href="#b6">6</a>. </p> <p>Extracting species-wide projections from time-series collected on a single population requires a precise understanding of how local events relate to species-scale demographic processes. Indeed, in several instances, it appeared that observations made on a single breeding population could not be extended to larger areas. The clear phenological shift in response to climate change observed in a Great tit <i>Parus major</i> population in Southern Britain <a href="#b7">7</a>, for example, could not be observed under similar conditions in the Netherlands <a href="#b8">8</a>. In the Chinook salmon <i>Oncorrhynchus tshawytscha</i>, survival rate in different populations was shown to be correlated with different sets of climate variables, thus preventing the definition of a single response model for the species <a href="#b9">9</a>. In such cases, the confounding effect of local heterogeneity is often involved: for example, abiotic habitat characteristics <a href="#b9">9</a>, biotic interactor communities <a href="#b10">10</a> or the dynamics of particular interspecific interactions <a href="#b11">11</a> can be locally heterogeneous, and lead to population-specific responses to climate change. Such observations present a challenge for the establishment of large-scale climate envelope models <a href="#b12">12</a>, but also for the inference of species-wide demographic parameters from single-population surveys. Indeed, two independent assumptions need to be met: (i) that population dynamics are homogeneous in space (that is, that the focal population does not behave differently from the species as a whole), and (ii) most importantly, that populations are demographically independent, so that the observed local trends can be assumed parallel (and not complementary) to trends in other populations. </p> <p>Indeed, robust extrapolation requires populations to be closed systems, in which coupling with other populations is minimal. In other words, we need to establish whether the observed local extinctions or fluctuations occur as a consequence of mortality peaks, or massive dispersal events—or a combination of both. Adult mortality has traditionally been proposed as the primary factor (through changes in resource availability and subsequent starvation <a href="#b13">13</a>, or increased predation <a href="#b14">14</a>), and several models have been built on that basis <a href="#b15">15</a> <a href="#b16">16</a>. Yet, these models mostly rely on the explicit assumption that movement among populations is negligible <a href="#b15">15</a>, leaving adult and juvenile survival and breeding output as the sole factors driving population dynamics. This assumption, however, is not based on direct evidence, but is rather motivated by technical difficulties in discriminating emigration from mortality of tagged individuals <a href="#b16">16</a>. </p> <p>In a recent review, Chown <i>et al.</i> <a href="#b4">4</a> reported that despite the pristine appearance of Antarctica, its species and ecosystems are also under considerable threat. The Emperor penguin, the only winter-breeding top-predator species of the continent, stands at the forefront of the impacts of climate warming <a href="#b17">17</a>, and recent projections, based on the demographic trend observed at one colony in Adélie Land in Eastern Antarctica, suggest that it may be facing high extinction risk within the next 100 years <a href="#b15">15</a>. Emperor penguins breed nearly exclusively on sea ice: this unstable habitat makes the species immediately sensitive to local environmental changes. One of the northernmost colonies, located on the Antarctic Peninsula, vanished during the last few decades, as sea ice retreated in that area <a href="#b18">18</a>. Other colonies underwent a dramatic drop in breeding success and population size shortly after a modification in local sea ice topology <a href="#b19">19</a>. The colony breeding on the tongue of the Mertz glacier, for instance, disappeared after the 2010 calving of that glacier <a href="#b20">20</a>, and the following changes in local ice movements also had catastrophic consequences on the nearby Pointe Géologie colony, where the number of fledged chicks dropped from ∼2,500 in 2010 to ∼100 in 2014 (field observation). All of these events have in common an identified proximal cause, usually linked to modifications in the local sea ice landscape. Either increased sea ice forced adults to make longer trips to reach open waters for foraging (with subsequent breeding failure resulting from heightened energy expenditures) <a href="#b19">19</a>, or on the contrary glacier calving destroyed the usual colony location <a href="#b20">20</a>. On the other hand, recent empirical evidence increasingly points to an important effect of dispersal in the Emperor's response to habitat disruption <a href="#b21">21</a>. The rapid recovery of the Emperor penguin population in Coulman Island confirms this assumption and excludes a peak in adult mortality followed by re-growth <a href="#b19">19</a>. Recent satellite and ground surveys have also shown that whole Emperor penguin colonies are able to relocate with or without an identified cause <a href="#b20">20</a> <a href="#b22">22</a>. Finally, biologging experiments have emphasized the outstanding distances regularly travelled by adult and juvenile Emperor penguins <a href="#b23">23</a>. Here, we demonstrate that dispersal is a fundamental component of demography in this long-lived species, and that all the world's colonies behave as a single evolutionary unit sharing a common demographic history. We also propose that dispersal plays a central role in the species' adaptive response to environmental change at the continental scale. As such, migration among colonies needs to be incorporated into demographic models in order to achieve accurate projections. </p> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="genome-widesingle-nucleotidepolymorphism(snp)typing" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide single-nucleotide polymorphism (SNP) typing</div> <p>We produced genome-wide restriction-site associated DNA sequencing (RAD-sequencing) data <a href="#b24">24</a>, yielding a total of 59,037 highly confident polymorphic sites using a consensus-calling approach ( <a href="#S1">Supplementary Fig. 1</a>) for 110 individuals from 6 Emperor penguin colonies representing the whole species' range ( <a href="#f1">Fig. 1a</a>). To assess both fine- and large-scale processes, we sampled three colonies in a tight cluster around Adélie Land, in Eastern Antarctica: Eastern and Western Mertz colonies <a href="#b20">20</a> (‘MZE' and ‘MZW'), as well as the Pointe Géologie colony, near Dumont d'Urville research station (‘DDU'), all three within ∼300 km. Two colonies were sampled in the Weddell Sea area, across the continent: Atka Bay colony, near Neumayer research station (‘NEU'), ∼6,500 km away from Adélie Land, and Halley Bay colony (‘HAL'), ∼700 km further. Finally, one colony was sampled from the Ross Sea area (Cape Washington, ‘WSH'), ∼1,700 km from Adélie Land and ∼8,000 km from HAL. </p> </div> <div class="afullypanmicticspecies" title="sec"> <div class="title" tagx="title" title="title">
A fully panmictic species</div> <p>In striking contrast both with its fragmented geographical distribution and with our current knowledge about other marine predators <a href="#b25">25</a> <a href="#b26">26</a> <a href="#b27">27</a>, the Emperor penguin exhibits a remarkable degree of genetic homogeneity at the continent scale. Pairwise fixation index ( <i>Fst</i>) values, calculated either as a function of allele frequency covariance between populations over variable sites, or from called genotypes using Reich's estimator <a href="#b28">28</a>, are very low ( <a href="#S1">Supplementary Table 1</a>), and only ∼0.5% of total variance is explained by colony structure (as per analysis of molecular variance, or AMOVA). High genetic mixing is supported by all classical descriptors of genetic variation as estimated on our consensus SNP set. Hardy–Weinberg equilibrium holds across 466 out of 590 scaffolds with all colonies assessed together (out-of-equilibrium scaffolds all have less than 5 SNPs). The mean homozygosity across all individuals is low (F=0.051±0.094) and does not exhibit any inter-colony difference. The nucleotide diversity is low and highly similar across all colonies ( <i>π</i> <sub>DDU</sub>=0.0026; <i>π</i> <sub>MZE</sub>=0.0023; <i>π</i> <sub>MZW</sub>=0.0023; <i>π</i> <sub>NEU</sub>=0.0023, <i>π</i> <sub>HAL</sub>=0.0023; <i>π</i> <sub>WSH</sub>=0.0026), going along with the expectation for long-lived high-investment species <a href="#b29">29</a>, and despite very different colony census sizes. Finally, Tajima's <i>D</i>, as estimated for non-coding regions of the genome from down-sampled haplotypes, does not deviate from neutral expectations ( <i>D</i> <sub>DDU</sub>=−0.87; <i>D</i> <sub>MZE</sub>=−0.66; <i>D</i> <sub>MZW</sub>=−0.578; <i>D</i> <sub>NEU</sub>=−0.64; <i>D</i> <sub>HAL</sub>=−0.68; <i>D</i> <sub>WSH</sub>=−0.28). A neighbour-net based on pairwise Hamming distances shows considerable admixture between areas: individuals are trending towards geographical sorting according to geographical location, but inter-individual variability is largely dominant ( <a href="#f1">Fig. 1b</a>). Overall, the variance explained by geographical structure is extremely low. The first principal component analysis (PCA) component of variance ( <a href="#S1">Supplementary Fig. 2</a>) shows a weak general pattern of isolation-by-distance along the coast. Mertz colonies stand on the one end, Pointe Géologie in the centre, and Atka Bay and HAL at the other end, yet Ross sea samples do not stand out, as would be expected in case of strong isolation-by-distance. However, this pattern remains extremely marginal: the variance explained by the first component barely amounts to 1.4% of the total variance, and colonies cannot be distinguished on the basis of PCA. Inference of population split topology based on allele frequency variation among populations also supports this view: neutral genetic differentiation from the ancestral population increases eastward from WSH to MZE, but with numerous migration events inferred between most colonies ( <a href="#S1">Supplementary Fig. 3</a>). Finally, in clustering analyses performed either on called genotypes or on genotype likelihoods, the preferred model consistently had <i>k</i>=1, that is, no inferred clusters in the data. The very limited genetic drift observed between colonies separated by several thousands of kilometres suggests intense gene flow in this flightless seabird along the coast of Antarctica. </p> </div> <div class="long-termdemographicreconstruction" title="sec"> <div class="title" tagx="title" title="title">
Long-term demographic reconstruction</div> <p>The existence of a common, homogeneous gene pool for the entire species also implies that all present-day colonies share a common demographic history. To test this, we reconstructed past Emperor penguin population size changes in BEAST2 (ref. <a href="#b30">30</a>) under an extended Bayesian skyline plot model. In accordance with our expectation, reconstructions based either on a single colony or on haplotypes sampled randomly from the whole continent converge to the same estimate of effective population size and to the same demographic history ( <a href="#f2">Fig. 2a</a>). All reconstructions show a moderate increase in population size over the past 100,000 years (with some uncertainty as to the precise dating of the beginning of the expansion, because of the difficulty of precisely calibrating mutation rates in a multilocus approach), regardless of the very different present-day colony sizes ( <a href="#f2">Fig. 2b</a>). This trend is in accordance with the findings of Li <i>et al.</i> <a href="#b31">31</a> based on a single-genome pairwise sequentially Markovian coalescent approach. Emperor penguin population size does not appear to have been affected by the last glacial period. This is in stark contrast with the sudden post-glacial population expansion of the Emperor's sister species, the King penguin <i>A. patagonicus</i> from the sub-Antarctic region <a href="#b32">32</a>, a difference probably explained by the contrasting breeding habitats and strategies of the two species, the King penguin being dependent on year-round ice-free breeding sites. The inferred long-term demographic stability also excludes past genetic fragmentation, which is known to reduce effective population size <a href="#b33">33</a>. Thus, overall, the agreement of coalescent histories at both local and global sampling scales supports the idea that all extant colonies share the same genetic pool, and that the effects of neutral genetic drift between distant areas are durably counterbalanced by the intensity of gene flow. </p> </div> <div class="importanceofcontinent-widedispersal" title="sec"> <div class="title" tagx="title" title="title">
Importance of continent-wide dispersal</div> <p>To quantitatively assess the importance of continental dispersal, we modelled the inter-colony migration rates required to generate such a level of genetic admixture. We first co-estimated effective population size and bidirectional migration rates between a subset of four colonies representing our whole sampling area by simulating genetic data under a continuous-time Markovian coalescent model against the observed two-dimensional allele frequency spectra, using a composite-likelihood approach <a href="#b34">34</a>. Analysis was calibrated using a RADome mutation rate as estimated for the King penguin <a href="#b32">32</a> and a generation time of 16 years <a href="#b15">15</a> <a href="#b16">16</a>. Effective population sizes all converge to an average of ∼4,000 (from 1,120 to 7,640) breeding individuals (detailed estimates in <a href="#S1">Supplementary Table 2</a> and <a href="#f1">Fig. 1a</a>), in keeping with the observed median size of extant colonies <a href="#b35">35</a>. Each colony is estimated to receive, on average, between 0.7% and ∼4.2% of its effective population size in migrants every generation. Thus, if we were to scale these results to present-day observed census size, a colony such as Pointe Géologie (DDU), with a count of ∼7,000 breeding adults, would exchange, on average, between ∼260 and ∼300 migrants (3.8–4.2%) per generation with the rest of the continent. Remarkably, strongly asymmetric gene flow pattern is inferred between DDU and MZE ( <a href="#f1">Fig. 1a</a>), with nearly all dispersal occurring towards MZE. It is relevant that the DDU colony has indeed gone through a strong population reduction in the recent past, and environment-induced adult mortality was suggested to be the main cause <a href="#b13">13</a> <a href="#b15">15</a> <a href="#b36">36</a>. However, our results point towards a very central role of dispersal, as direct human disturbance (for example, effect of flipper-banding <a href="#b37">37</a>) on the DDU colony may have led to vastly increased emigration flow from DDU to MZE. We further validated these results using a haplotype-based multilocus Bayesian approach as implemented in Migrate-n <a href="#b38">38</a>. Model ranking using Bayes factor model choice gives clear support for a full-migration model with very high gene flow. Estimated migration rates ( <i>M</i>) and population sizes ( <i>Θ</i>) are highly homogeneous: mean mutation-scaled ( <i>μ</i>) migration rate <i>M</i>= <i>m</i>/ <i>μ</i>=2,358±130, mutation-scaled effective sizes <i>Θ</i> <sub>WSH</sub>=0.0017, <i>Θ</i> <sub>MZE</sub>=0.0018, <i>Θ</i> <sub>MZW</sub>=0.0019, <i>Θ</i> <sub>DDU</sub>=0.0019, <i>Θ</i> <sub>NEU</sub>=0.0018, <i>Θ</i> <sub>HAL</sub>=0.0019. Considering a conservative range of <i>μ</i> for our subset of loci <a href="#b32">32</a>, we can estimate that each colony receives, on average, from 5.40% (±0.22) to 10.00% (±0.42) of its total effective size as migrants from other colonies at each generation. Although slightly higher, this estimate is of the same order of magnitude as the one derived from the joint allele frequency spectra. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>It is important to note that the estimates we present here reflect the migration parameters averaged over many generations, and not the instantaneous dispersal rate (see <a href="#S1">Supplementary Note 1</a> for details). Therefore, neither population sizes nor migration rates should be interpreted as referring literally to the immediate state of extant colonies but rather to the average coalescence-based population size and migration rate over time. Dispersal itself is not necessarily a stationary process: it may be very heterogeneous in space and time (as was shown, for example, in the Little penguin <i>Eudyptula minor</i> <a href="#b39">39</a>), while individual colonies may show positive or negative dispersal balance according to the current local habitat conditions and modifications. </p> <p>It appears from our results that Emperor penguin colonies are largely open systems, in which demographic coupling with other colonies through dispersal is a fundamental mechanism. Recent observations suggest that this view may be extended to other seabird species <a href="#b26">26</a>. For example, we have repeatedly resighted micro-tagged King penguins born on one colony in the Crozet Archipelago, at different colonies in the same archipelago, and at colonies in the Kerguelen Archipelago, 1,500 km away (Le Bohec, personel communication, 2016). As pointed out by Mayr <a href="#b40">40</a>, ‘ <i>a high dispersal ability is a necessity for occupants of temporary habitats'</i>. This necessity appears to be twofold. First, from a mechanistic point of view, habitat instability can force dispersal. Indeed, dispersal has been the immediate response for colonies facing habitat disturbance in the recent past <a href="#b18">18</a> <a href="#b19">19</a> <a href="#b21">21</a>, and it can play an important role by adding flexibility to an otherwise rigid philopatric system, and allowing species to dynamically exploit the best breeding locations in rapidly changing polar environments. Second, high dispersal rates are expected to impose a high <i>migration load</i> on natural selection, and to counteract the effects of local adaptations <a href="#b41">41</a>, thus allowing the species to preserve a gene pool that is adapted to a wider range of possible environmental conditions: it may thus itself be adaptive in highly unstable habitats. Yet, a direct consequence is that colony-level demographic events are of local rather than global significance. Population trends extracted from a single colony reflect the immediate quality of the focal location on a generation scale, rather than species-wide parameters ( <a href="#f2">Fig. 2c</a>). In this context, the colony is not a truly relevant demographic unit, but rather a transient aggregation of individuals at a particular point of time. To distinguish this structure from panmictic and metapopulation systems, we propose the term <i>synnome</i> for this combination of an exceptionally fragmented space and a very fluid gene pool (see full definition in <a href="#S1">Supplementary Note 1</a>). </p> <p>Although exceptional if compared with current knowledge about other marine predators, such as turtles <a href="#b25">25</a>, marine mammals <a href="#b27">27</a> and most seabird species <a href="#b26">26</a>, intense dispersal is expected to be a common evolutionary strategy in unstable high-latitude environments <a href="#b26">26</a>, especially under ongoing climate warming. Importantly, most of the evidence accumulated up to now in seabirds has relied heavily on mitochondrial DNA <a href="#b26">26</a> (see also <a href="#S1">Supplementary Note 2</a> and <a href="#S1">Supplementary Fig. 5</a> for details), which is known to provide biased evidence due to its faster coalescence rate and non-recombining character <a href="#b42">42</a>. SNP data can also provide evidence of more intense gene flow than mitochondrial DNA (as has been shown, for example, in the Black-footed albatross <i>Phœbastria nigripes</i> <a href="#b43">43</a>). Therefore, several species that are currently treated as fragmented metapopulation systems, may in fact appear to be closer to the Emperor penguin's <i>synnome</i> structure once genome-wide data will be available. </p> <p>Our findings highlight the importance of adopting a cross-disciplinary approach, integrating population genomics and behavioural ecology, to the study of population dynamics. Such an approach accounts for the temporally and spatially complex ecological processes that shape the structure of worldwide populations, and should help in the development of more robust and accurate demographic projections at the whole species level. These refined projections will then be more likely to allow estimation of the extent of threats to vulnerable species and identifying the proximal causes of their decline. As a single genetic population, Emperor penguins will respond to climate change through a unified evolutionary trajectory. This insight thus presents new challenges to our understanding of how current climate scenarios will impact upon the future of the most cold-adapted species in the world, an iconic bio-indicator of the delicate Antarctic ecosystem. </p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="samplecollectionanddnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Sample collection and DNA extraction</div> <p>Material was collected from six locations of East Antarctica, between 2004 and 2012: Cape Washington in the Ross Sea area (‘WSH'), Pointe Géologie (‘DDU') in Adélie Land, ‘MZE' and ‘MZW' in George V Land, Atka Bay (‘NEU') in Dronning Maud Land area and ‘HAL' in the Weddell Sea area. All procedures employed during field work were approved by national Ethical Committees, and authorizations to enter the breeding site and to collect samples from living or dead birds were delivered by the US Antarctic Conservation Act (permit no 2004-010), the French ‘Ministère de l'Enseignement Supérieur et de la Recherche' and the ‘Terres Australes et Antarctiques Françaises' (permit nos 2012-126, 2012-111 and 2012-117), the German Environmental Agency (Umweltbundesamt, permit no I 3.5–94003-3/295) and the British Antarctic Survey/University of Cambridge Animal Ethics Review Committee. On the DDU colony, blood samples were collected from 23 chicks before fledging. In WSH, MZE, MZW, NEU and HAL, muscle samples were collected from frozen chick carcasses collected around the colony (WSH: <i>N</i>=4, MZE: <i>N</i>=25, MZW: <i>N</i>=14, NEU: <i>N</i>=24, HAL: <i>N</i>=20). DNA was extracted using a spin-column protocol (Qiagen DNEasy Blood and Tissue kit, Qiagen) with minor modifications. </p> </div> <div class="genome-widesnptyping" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide SNP typing</div> <p>SNP discovery and sequencing followed a single-digest RAD-sequencing protocol <a href="#b24">24</a>. Genomic DNA was checked for degradation on a 1.5% agarose gel, and only samples with consistently high molecular weight were retained and quantified by fluorometry (Life technologies Qubit). A total of 110 samples were retained and sequenced in 5 distinct libraries. (i) ∼150 ng of genomic DNA per sample were digested with the restriction enzyme <i>Sbf</i>-I-HF (NEB); (ii) each sample was then ligated to a unique barcoded P1 adapter before pooling in a single library. The library was then sheared by sonication (7 cycles of 30 s ON—30 s OFF); (iii) sonicated libraries were concentrated to 25 μl by DNA capture on magnetic beads (beads solution/DNA=0.8:1), thus further reducing the carry-over of non-ligated P1 adapters, and the target size range fraction (350–650 bp) was then selected by automated gel electrophoresis (BluePippin); (iv) capture on magnetic beads using the same beads/DNA ratio (0.8:1) was then employed in all following purification steps (after blunt-end repairing, poly-A tailing, P2 adapter ligation and library enrichment by PCR). Magnetic beads were kept together with the library throughout the pre-PCR steps, and DNA was re-bound to the beads for purification using a PEG-8000-binding solution; (v) PCR amplification was performed in 8 × 12.5 μl aliquots pooled after the amplification in order to reduce amplification bias on few loci because of random drift. PCR was performed using NEB Phusion polymerase with the following cycles: 30 s denaturation at 98 °C, 18 cycles of amplification (10 min at 98 °C, 30 s at 65 °C and 30 s at 72 °C), and a final elongation of 5 min at 72 °C; (vi) the library was then quantified by a fluorimetry-based method (Life technologies Qubit), and molarity was checked on an Agilent Bioanalyzer chip (Invitrogen). A final volume of 20 μl for each library was submitted for paired-end sequencing on an Illumina HiSeq2000 sequencer (V3 chemistry, libraries 1–3) or HiSeq2500 (V4 chemistry, libraries 4–5), at the Norwegian Sequencing Centre, University of Oslo, spiked with 20% PhiX control library in order to reduce low-diversity bias. </p> </div> <div class="sequencealignmentandgenotyping" title="sec"> <div class="title" tagx="title" title="title">
Sequence alignment and genotyping</div> <p>Data processing was performed using the following workflow: (i) <i>Sequence demultiplexing.</i> Read quality assessment was made in FastQC ( <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Samples were de-multiplexed according to in-line barcodes using Stacks v1.20 (ref. <a href="#b44">44</a>), low-quality reads were discarded and sequences trimmed to 95 bp. (ii) <i>Read mapping and filtering</i>. Demultiplexed fastq files were mapped to the published contigs of the Emperor penguin genome <a href="#b45">45</a> using Bowtie2 2.2.3 (ref. <a href="#b46">46</a>), with standard settings, allowing only end-to-end mapping. Resulting SAM files were filtered using Samtools 0.1.19 (ref. <a href="#b47">47</a>), PicardTools 1.113 ( <a href="http://picard.sorceforge.net">http://picard.sorceforge.net</a>), and custom R and shell scripts in order to discard unpaired reads and full read pairs where at least one mate has a mapping quality score below 30. The resulting BAM files were then filtered for PCR and optical duplicates by comparing mapping position and CIGAR string, using Picard MarkDuplicates. This process also allowed to filter out most sequencing errors, as MarkDuplicates only retains the read with the highest average Phred score in each duplicate cluster. (iii) <i>SNP calling and genotyping</i>. Three independent algorithms were used for SNP and genotype calling, all of them built on a maximum-likelihood framework. (i) First-in-pair reads were exported as BAM files for the maximum-likelihood-based genotype caller built in the Stacks pipeline (ref_map.pl), with a maximum of five mismatches allowed between alleles at a single locus (both within and between individuals), correcting genotype calls using the information from the whole data set in the rxstacks programme. (ii) We used the same cleaned BAM files to simultaneously call both mismatch and indel polymorphisms in all samples using the GATK HaplotypeCaller pipeline <a href="#b48">48</a>, with standard parameters, except for population heterozygosity which was set to 0.01. (iii) ANGSD 0.900 (ref. <a href="#b49">49</a>) was used to call SNPs and genotypes using a maximum-likelihood process with the Samtools mpileup/bcftools algorithm, using the complete sample allele frequency information as a prior. Genotype calls were only retained for comparison purposes with Stacks and GATK, however, downstream analysis was performed directly on raw genotype likelihoods, as this approach has been shown to be much more sensitive to weak structure than classical SNP-calling analysis <a href="#b50">50</a>. Genotype calls from all three processes were formatted into VCF format using software-specific (Stacks and GATK) or custom scripts (ANGSD); further filtering and manipulation were done using VCFtools 0.1.12 (ref. <a href="#b51">51</a>). For analyses relying on SNP calls, an additional filtering step was performed by (i) extracting the list of consensus loci called as SNPs by all three independent algorithms, (ii) discarding genotypes with a coverage under 3 × for analyses sensitive to sequencing depth, (iii) removing loci genotyped in less than 75% of all individuals, and finally (iv) thinning down loci to keep only polymorphism distant of at least 1 kb, in order to minimize linkage between markers. (iv) <i>Sex-linked marker analysis</i>. To check for sex-specific dispersal or structure patterns, we repeated analyses for each sex separately, using either autosomal loci only or Z-linked loci only. We used the published genome annotations for bird gametologs <a href="#b52">52</a> to identify potentially Z-linked scaffolds. As females are heterogametic in birds, we expect non pseudo-autosomal Z-linked regions to be fully homozygous in females, but to be neutrally heterozygous in males. We therefore retained only scaffolds that had a clearly bimodal distribution of heterozygosity, with one mode at or close to 0 (a slight tolerance was allowed to account for misalignment, sequencing errors mistyped as SNPs due to low coverage, or the presence of a transposable elements with autosomal homologues). Fifteen scaffolds were ultimately retained. Scaffolds containing candidate Z-linked coding DNA sequences, but with no visible bimodal heterozygosity distribution, were excluded altogether from the analysis. Sex assignment was performed independently from each of the non-recombining Z scaffolds, and consistency of sex calls between scaffolds was checked manually. </p> </div> <div class="raddatadescription" title="sec"> <div class="title" tagx="title" title="title">
RAD data description</div> <p>Each HiSeq sequencing lane yielded an ∼201,000,000 paired-end reads (±16,000) with a mean Phred score of 37, only part of which was dedicated to that project. After barcode demultiplexing and quality filtering, we retained an average 147,000,000 read pairs per library (±13,000,000). Concordant alignment rate was high (70.7±9.1%). However, after filtering, a large proportion of the reads was identified as duplicates and removed from further analysis. On average, 479,000 read pairs were retained per individual. We built an average 78,000 loci per individual (±17,000). Overlap between SNP calling methods was high: 111,686 SNPs were called by GATK (173,704 SNPs total), ANGSD (203,801 SNPs total) and Stacks (148,721 SNPs total; <a href="#S1">Supplementary Fig. 1</a>). After filtering by coverage (minimum 3 ×) and missing data (minimum 75% representation), 59,037 highly confident SNPs (hereafter our ‘consensus SNP set', as opposed to the ‘full data set', which includes all sequencing data that passed our quality checks and aligned successfully to autosomal or pseudo-autosomal scaffolds) were retained for analysis, with a mean depth of 6.8 × . Of these, 582, spread across 15 scaffolds, were identified as unambiguously belonging to the non-recombining region of the Z chromosome. </p> </div> <div class="pairwisefst,amovaandsummarystatistics" title="sec"> <div class="title" tagx="title" title="title">
Pairwise FST, AMOVA and summary statistics</div> <p>Pairwise fixation index was estimated by two independent methods. We calculated Reich's estimator <a href="#b28">28</a> on called genotypes of our consensus SNP set, using custom R scripts. Without calling genotypes, we calculated Reynolds' estimator <a href="#b53">53</a> on the full data set, taking into account uncertainty in site-allele-frequency, according to the method implemented in ngsTools <a href="#b50">50</a> <a href="#b54">54</a>. Both methods were performed on a per-site basis and averaged over 1-kb non-overlapping sliding windows, as the ratio of the windowed sum of inter-population variance over the windowed sum of total variance. For the genotype-call-free method, only sites with a probability of being variant at least equal to 0.95 were included in the estimation. Mean homozygosity, nucleotide diversity and Tajima's <i>D</i> were calculated on a per-locus basis, using the consensus SNP set. To avoid possible biases due to low coverage in estimating nucleotide diversity and Tajima's <i>D</i>, we randomly sampled one haplotype for each individual, and performed calculations on this haploid subset. Calculations were made using <i>adegenet</i> <a href="#b55">55</a> and <i>pegas</i> <a href="#b56">56</a> packages, as well as custom R scripts. To quantify the proportion of variation at each organization level, AMOVA was performed on a 22,875 unlinked SNP data using Arlequin <a href="#b57">57</a>, on a per-locus basis, with 1,000 permutations. </p> </div> <div class="identity-by-state(ibs)andidentity-by-descent(ibd)" title="sec"> <div class="title" tagx="title" title="title">
Identity-by-state (IBS) and identity-by-descent (IBD)</div> <p>Pairwise indicators of IBS and IBD were calculated in PLINK v1.9 (ref. <a href="#b58">58</a>) based on the consensus SNP set. Allelic distance for pairs of individuals is rather sensitive to uneven coverage ( <a href="#S1">Supplementary Fig. 4</a>, lower triangle), with lower-depth individuals appearing more similar to all other individuals than higher-depth ones. However, IBD inference, which takes into account the total genetic variance in the whole sample, appears more robust to coverage variation ( <a href="#S1">Supplementary Fig. 4</a>, upper triangle). Three individuals in the HAL colony (HAL13, HAL14 and HAL16, visible on <a href="#S1">Supplementary Fig. 4</a>), as well as two in the MZE colony (MZE01 and MZE04, visible on <a href="#S1">Supplementary Fig. 4</a>), appear markedly more related to each other than to the rest of the sample. Of these two clusters, only one individual was retained for inbreeding-sensitive analyses, such as PCA. Pairwise IBS (Hamming) distances were used to generate a neighbour-net using SplitsTree <a href="#b59">59</a>. </p> </div> <div class="principalcomponentanalysis" title="sec"> <div class="title" tagx="title" title="title">
Principal component analysis</div> <p>PCA was performed on called genotypes in the consensus SNP set using R library <i>adegenet</i> <a href="#b55">55</a>, and on genotype likelihoods for the full data set using ngsCovar <a href="#b54">54</a>. Analysis was performed either on all samples or keeping only one individual for each highly related cluster. When performed on all samples, analysis was mostly driven by the HAL relatedness cluster, with the first principal component of variance (PC) accounting for 2.84% of total variance. Interestingly, these outliers are also identified as a separate group in admixture analyses (see below), with high repeatability, although with only a slight gain in model fit compared with a single-population model (Evanno's Δ <i>K</i> over 10 replicates). However, discriminant analysis of principal components <a href="#b60">60</a> performed in <i>adegenet</i> on the consensus SNP set, and contrasting these outliers and the main sample group, showed that single-locus contributions were very low and evenly spread across the genome, thus excluding any strong directional selective phenomenon. After removal of all samples but one in each high-relatedness cluster, variation did not appear to be driven by outliers anymore. </p> </div> <div class="populationsplitsandmigrationtopology" title="sec"> <div class="title" tagx="title" title="title">
Population splits and migration topology</div> <p>We used TreeMix <a href="#b61">61</a> to infer the topology of population splits and migrations from allele frequency variation among colonies. We produced RAD-sequencing data for the Emperor's sister species, the King penguin and processed it with the same protocol as for our Emperor penguin samples. We genotyped 15 King penguins from the Baie du Marin colony, on Crozet Archipelago, that were used as an outgroup for the analysis. We restricted our data set to ca 15,000 highly confident, unlinked SNPs shared between the two species. Three-population and four-population tests did not allow us to reject a tree-like topology in our data. Analysis was performed first using the King penguin outgroup, and bootstrapping over blocks of 500 SNPs, in order to assess the topology of relationships between our six Emperor penguin colonies. We used this topology to re-run TreeMix without the King penguin samples, but fixing the root according to our first results, in order to increase the resolution of the analysis ( <a href="#S1">Supplementary Fig. 3</a>). </p> </div> <div class="clusteringanalysis" title="sec"> <div class="title" tagx="title" title="title">
Clustering analysis</div> <p>Clustering analyses were performed both on called genotypes, using fastStructure <a href="#b62">62</a>, and on genotype likelihoods, using ngsAdmix <a href="#b63">63</a> and a stricter filtering was applied: as input in fastStructure we selected unlinked 6,825 SNPs with a maximum amount of missing data of 10%. In ngsAdmix, we filtered the data set to 3,020 high-confidence sites on the basis of probability of being variable, coverage and phred score. We tested a number of components ranging from 1 to 10, with three independent replicates. For fastStructure, we used a simple prior for <i>K</i> values ranging from 1 to 10, and a logistic prior for values from 1 to 3. Most likely number of components was chosen using fastStructure's chooseK.py script for fastStructure models and Evanno's Δ <i>K</i> method for ngsAdmix models. </p> </div> <div class="coalescent-basedanalysisusingbeast2" title="sec"> <div class="title" tagx="title" title="title">
Coalescent-based analysis using BEAST2</div> <p>An independent estimate of population size changes through time was performed in BEAST2 (ref. <a href="#b30">30</a>). We proceeded as above for locus selection and haplotype down-sampling. To remain agnostic as to population structure, we performed analysis on each colony separately, as well as on the whole data set, as the lack of strong population structure, evidenced by all other analyses, allowed us to sample haplotypes from the whole species without violating the model's assumptions. We used an extended bayesian skyline plot model <a href="#b64">64</a> in order to co-estimate present-day <i>Θ</i> and possible past fluctuations. We followed the protocol of Trucchi and colleagues <a href="#b32">32</a>, but reduced the parameter space by defining only one site-model per locus class (five or six SNPs), using HKY models with empirical base frequencies, and allowing for rate variation in four discrete gamma categories. Kappa was linked across site models, according to our expectation for neutral variation, in order to alleviate computational load. All chains were run in duplicate to check for convergence and for a sufficient length to gather effective sample sizes (ESS)&amp;gt;200 for all parameters, which necessitated ca 1,000,000,000 steps on all models. We used the same estimates for mutation rate and generation time as in Migrate-n analyses. Reconstruction for WSH colony is much less precise because of the very small number of haplotypes ( <i>N</i>=4) sampled per locus. However, present-day population size estimate converges with reconstructions based on the other colonies. </p> </div> <div class="fastsimcoal2analysis" title="sec"> <div class="title" tagx="title" title="title">
Fastsimcoal2 analysis</div> <p>Joint derived-allele frequency spectra were generated from the full data set in ANGSD 0.900 (ref. <a href="#b49">49</a>) for a subset of four colonies that encompass the whole continent (MZE, DDU, NEU and HAL). To polarize these spectra, we reconstructed the most likely ancestral base for all positions in the RADome. We selected 12 high-quality Emperor penguin samples covering the whole species' range, as well as 12 high-quality King penguin samples processed according to the same protocol. States at all positions were determined using GATK's Haplotype Caller pipeline <a href="#b48">48</a>. We used BEDtools <a href="#b65">65</a> and VCFtools' vcf-consensus script <a href="#b51">51</a> to update the published Emperor penguin genome and establish a reference RADome for both the King penguin and the Emperor penguin, using only high-quality polymorphisms (phred-scale genotype quality ≥80), and including variable sites as ambiguity codes according to the International Union of Pure and Applied Chemistry (IUPAC) standard notation. We aligned this RADome to the Adélie penguin genome <i>Pygoscelis adeliae</i> <a href="#b66">66</a> using Bowtie2 (ref. <a href="#b46">46</a>), and extracted the corresponding regions. For each RAD locus, a maximum-likelihood unrooted tree was built in PhyML <a href="#b67">67</a>, and maximum-likelihood ancestral sequence for crown- <i>Aptenodytes</i> was reconstructed using PAML <a href="#b68">68</a> and Lazarus ( <a href="https://project-lazarus.googlecode.com/">https://project-lazarus.googlecode.com/</a>), using PhyML tree topology as a prior. Ancestral states were then used to determine the ancestral and derived alleles in the Emperor penguin. </p> <p>Reconstruction of population sizes and migration events was performed through composite-likelihood maximization <a href="#b34">34</a>, by simulating joint-spectra under a continuous-time Markovian coalescent model in fastsimcoal2.5.11 (ref. <a href="#b69">69</a>). For each run, we performed a maximum of 80 expectation-conditional maximisation (ECM) optimization cycles over the 12 retained parameters (population sizes and asymmetric migration rates between the four analysed colonies), each parameter optimization step requiring the generation of 100,000 simulated joint-spectra. We generated 50 non-parametric bootstrap replicates for each spectrum. For each bootstrap data set, and for the original data set, we ran 50 independent replicates, and retained the one with the highest log-likelihood. We assumed a mutation rate of 2.6e <sup>−7</sup> subsitutions per site per generation as calculated for the King penguin <a href="#b32">32</a>, and a generation time of 16 years <a href="#b15">15</a>. Computation was performed on the high-performance Abel cluster at the University of Oslo, and required a total of ∼30,000 CPU hours. </p> <p>We chose to restrict our analysis to a stepwise migration model for two main reasons. First, computational load increases rapidly with the number of estimated parameters, and higher complexity models could not be run with the necessary amount of replication. Second, as our model is not supposed to represent precisely the present-day state of the colonies, but rather parameters averaged over a long period of time, we do not expect the intensity of the migration flow to be much affected by the structure of the connectivity. </p> </div> <div class="coalescent-basedanalysisusingmigrate-n" title="sec"> <div class="title" tagx="title" title="title">
Coalescent-based analysis using Migrate-n</div> <p>All methods used above provide a robust framework for identifying groups even in weakly structured populations. However, all are, to some extent, functions of the covariance of allele frequencies between populations. In the hypothesis that our estimate of the frequency spectrum may be biased is some way, for example, by moderate sample size, we also performed a coalescent-based structure analysis using Migrate-n <a href="#b38">38</a>. As opposed to frequency-spectrum-based approaches (see above), coalescent-based analysis relies on phased polymorphisms to infer population parameters. After verifying that the number of polymorphisms in each RAD locus followed a Poisson distribution of <i>λ</i> equal to the mean number of polymorphisms per locus, we selected 50 random loci comprising between five and six polymorphisms as an unbiased representation of the neutrally evolving part of the genome <a href="#b32">32</a>. To correct for potential over-representation of sequences in case a heterozygous individual was mis-called as homozygous, we randomly sampled one allele only for each individual. For all colonies except WSH, we randomly picked a set of haplotypes in the population (16 in MZE, DDU, HAL and NEU, 14 in MZW, and 4 in WSH). We ran a cold chain and three heated chain of 50,000,000 generations, recording every 500 generations, with a 5,000,000-generation burn-in. We used a static heating scheme, raising the cold chain to a power of 1.5, 3 and 1e6, and proposing chain swapping every 100 steps. We used a uniform prior for population sizes ( <i>Θ</i>), bounded between 0 and 0.1 (with a <i>δ</i> of 0.01), and for the migration rates ( <i>M</i>), bounded at 4,000 with a <i>δ</i> of 400. Proper mixing under these conditions was ensured using the highest parametrization model (model 2, see below). We compared five models of increasing complexity: (i) a panmictic model, in which all colonies were gathered in one population, estimating only a general <i>Θ</i>, (ii) a full-matrix model, in which asymmetric migrations were allowed between all pairs of colonies, (iii) a stepwise model, in which asymmetric migrations were allowed, but only between neighbouring colonies, in a closed circle, (iv) a first meta-population model, in which the Ross Sea (WSH), George V and Adélie Lands (MZE, MZW and DDU) and the South Atlantic (NEU and HAL) were treated as three populations, with an asymmetric migrations between them, (v) a second meta-population model, in which the northernmost colonies (MZE, MZW, DDU and NEU) were separated from the Weddell Sea (HAL) and the Ross Sea (WSH). Models were ordered by log Bayes factor defined by Kass and Raftery as <i>ln</i>BF=2[ <i>ln</i>(mL(model1))- <i>ln</i>(mL(model2))], with mL(model1) and mL(model2) being the marginal likelihoods for the two compared models, as calculated by thermodynamic integration. Under this model, each migration rate can also be expressed as a proportion of the receiving population's effective size as <i>m</i>=4· <i>M</i>· <i>μ.</i> Hence, the relative demographic importance of immigration for any given colony can also be expressed as Σ <sub>i</sub>= <i>n</i>(4· <i>Mi·μ</i>), with <i>n</i> being the total number of populations identified as gene sources for the focal population. To convert mutation-scaled estimates of <i>M</i> and <i>Θ</i>, we therefore need an estimate of <i>μ</i> (the mean number of substitutions·per site per·Myr) for the set of loci used in the analysis. As the number of sites in each locus is mainly due to the stochastic nature of the mutational process, and follows a Poisson distribution of parameter <i>λ</i> equal to the mean SNP density per locus (see above), we can consider that a single true mutation rate ( <i>μ</i>) applies to the whole RADome. However, as our analysis is restricted to loci containing five to six SNPs, our estimates of <i>Θ</i> and <i>M</i> are not directly scaled by <i>μ</i>, but rather by posterior probabilities of <i>μ</i> conditional on the number of SNPs in each locus class. We used the class-specific mutation rate posterior probabilities as calculated for the King penguin <a href="#b32">32</a>. As a conservative estimate, we used a range of rates fitting the 3-SNPs to 6-SNPs class loci. Using a generation time of 16 years <a href="#b15">15</a>, these are <i>μ</i> <sub>3snps</sub>=1.14e <sup>−6</sup> and <i>μ</i> <sub>6snps</sub>=2.16e <sup>−6</sup>. As these were estimated from a subset of 16 haplotypes, and we included 82 haplotypes in our analysis, we considered that variability was likely to be underestimated by Trucchi and colleagues <a href="#b32">32</a> compared with our sample. Thus, we did not consider higher estimates than those made for 6-SNPs loci. </p> </div> <div class="dataavailability" title="sec"> <div class="title" tagx="title" title="title">
Data availability</div> <p>Demultiplexed sequencing data are available from the Short Read Archive with the accession number <a href="SRP070516">SRP070516</a>. Input files used in the main analyses are available from figshare.com ( <a href="https://dx.doi.org/10.6084/m9.figshare.2949508.v1">https://dx.doi.org/10.6084/m9.figshare.2949508.v1</a>). </p> </div> </div> <div class="additionalinformation" title="sec"> <div class="title" tagx="title" title="title">
Additional information</div> <p> <b>How to cite this article</b>: Cristofari, R. <i>et al.</i> Full circumpolar migration ensures evolutionary unity in the Emperor penguin. <i>Nat. Commun.</i> 7:11842 doi: 10.1038/ncomms11842 (2016). </p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data">  <div class="title" tagx="title" title="title">
Supplementary Information</div> <p>Supplementary Figures 1-5, Supplementary Tables 1-2, Supplementary Notes 1-2 and Supplementary References </p>  <div class="media" title="media"> <a href="ncomms11842-s1.pdf">LINK</a> </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>This study was undertaken within the framework of the Programme 137 of the Institut Polaire Français Paul-Emile Victor (IPEV), with additional support from the French National Research Agency (ANR) ‘PICASO' grant (ANR-2010-BLAN-1728-01, PI: Y.L.M.), from Marie Curie Intra European Fellowships (FP7-PEOPLE-IEF-2008, European Commission; project no. 235962 to CLB and FP7-PEOPLE-IEF-2010, European Commission; project no. 252252 to E.T.), from the Centre Scientifique de Monaco through budget allocated to the Laboratoire International Associé 647 ‘BioSensib' (CSM/CNRS-University of Strasbourg), from the Centre National de la Recherche Scientifique (Programme Zone Atelier de Recherches sur l'Environnement Antarctique et Subantarctique) and from the British Antarctic Survey Ecosystems Programme, NERC (P.N.T.). Logistic and field costs of research were supported by IPEV (C.L.B.), the British Antarctic Survey Ecosystems Programme, NERC (P.N.T.), AWI (D.P.Z.), US NSF grant number NSF 0229638 (P.J.P.). This work was performed on the Abel Cluster, owned by the University of Oslo and the Norwegian metacenter for High Performance Computing (NOTUR), and operated by the Department for Research Computing at USIT, the University of Oslo. We are very grateful to Morten Skage, Ave Tooming-Klunderud, Marianne Selander-Hansen, and the Norwegian Sequencing Center for their very valuable help in the laboratory, as well as Lex Nederbragt and Michael Matschiner for their assistance with the Abel cluster, Matteo Fumagalli and Thorfinn Korneliussen for their precious advice regarding ngsTools and ANGSD, and Alexei J. Drummond for his insights on BEAST2 parametrization. We thank the IPEV Programme 109 (head: Henri Weimerskirch) for shared data collection on egg and chick mortality in the Emperor penguin colony of Pointe Géologie and Christophe Barbraud and Karine Delord for data management, Matthieu Boureau, and the British Antarctic Survey staff at Halley Bay for the sample collection. Last but not the least, we thank Romain Loriol for his help with Greek lexicology. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="b1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoffmann</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sgrò</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Climate change and evolutionary adaptation</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">470</span>, <span class="fpage" tagx="fpage" title="fpage">479</span>– <span class="lpage" tagx="lpage" title="lpage">485</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21350480">21350480</a> </span> </span> </li> <li tag="ref"> <a name="b2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Díaz</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fargione</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chapin</span> <span class="given-names" tagx="given-names" title="given-names">F. S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tilman</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Biodiversity loss threatens human well-being</span>. <span class="source" tagx="source" title="source">PLoS Biol.</span> <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">1300</span>– <span class="lpage" tagx="lpage" title="lpage">1305</span> ( <span class="year" tagx="year" title="year">2006</span>). </span> </li> <li tag="ref"> <a name="b3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomas</span> <span class="given-names" tagx="given-names" title="given-names">C. D.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Extinction risk from climate change</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">427</span>, <span class="fpage" tagx="fpage" title="fpage">145</span>– <span class="lpage" tagx="lpage" title="lpage">148</span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14712274">14712274</a> </span> </span> </li> <li tag="ref"> <a name="b4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chown</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">The changing form of Antarctic biodiversity</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">522</span>, <span class="fpage" tagx="fpage" title="fpage">431</span>– <span class="lpage" tagx="lpage" title="lpage">438</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26108852">26108852</a> </span> </span> </li> <li tag="ref"> <a name="b5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saether</span> <span class="given-names" tagx="given-names" title="given-names">B. E.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Generation time and temporal scaling of bird population dynamics</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">436</span>, <span class="fpage" tagx="fpage" title="fpage">99</span>– <span class="lpage" tagx="lpage" title="lpage">102</span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16001068">16001068</a> </span> </span> </li> <li tag="ref"> <a name="b6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allendorf</span> <span class="given-names" tagx="given-names" title="given-names">F. W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohenlohe</span> <span class="given-names" tagx="given-names" title="given-names">P. A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luikart</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genomics and the future of conservation genetics</span>. <span class="source" tagx="source" title="source">Nat. Rev. Genet.</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">697</span>– <span class="lpage" tagx="lpage" title="lpage">709</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20847747">20847747</a> </span> </span> </li> <li tag="ref"> <a name="b7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charmantier</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Adaptive phenotypic plasticity in response to climate change in a wild bird population</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">320</span>, <span class="fpage" tagx="fpage" title="fpage">800</span>– <span class="lpage" tagx="lpage" title="lpage">803</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18467590">18467590</a> </span> </span> </li> <li tag="ref"> <a name="b8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Visser</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Noordwijk</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tinbergen</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lessells</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Warmer springs lead to mistimed reproduction in great tits ( <i>Parus major</i>) </span>. <span class="source" tagx="source" title="source">P. Roy. Soc. Lond. B Bio.</span> <span class="volume" tagx="volume" title="volume">265</span>, <span class="fpage" tagx="fpage" title="fpage">1867</span>– <span class="lpage" tagx="lpage" title="lpage">1870</span> ( <span class="year" tagx="year" title="year">1998</span>). </span> </li> <li tag="ref"> <a name="b9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crozier</span> <span class="given-names" tagx="given-names" title="given-names">L. G.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zabel</span> <span class="given-names" tagx="given-names" title="given-names">R. W.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Climate impacts at multiple scales: evidence for differential population responses in juvenile Chinook salmon </span>. <span class="source" tagx="source" title="source">J. Anim. Ecol.</span> <span class="volume" tagx="volume" title="volume">75</span>, <span class="fpage" tagx="fpage" title="fpage">1100</span>– <span class="lpage" tagx="lpage" title="lpage">1109</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16922845">16922845</a> </span> </span> </li> <li tag="ref"> <a name="b10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenkinson</span> <span class="given-names" tagx="given-names" title="given-names">L. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lawton</span> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shorrocks</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wood</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Making mistakes when predicting shifts in species range in response to global warming</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">391</span>, <span class="fpage" tagx="fpage" title="fpage">783</span>– <span class="lpage" tagx="lpage" title="lpage">786</span> ( <span class="year" tagx="year" title="year">1998</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9486646">9486646</a> </span> </span> </li> <li tag="ref"> <a name="b11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stenseth</span> <span class="given-names" tagx="given-names" title="given-names">N. C.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Testing for effects of climate change on competitive relationships and coexistence between two bird species </span>. <span class="source" tagx="source" title="source">P. R. Soc. Lond. B Bio</span> <span class="volume" tagx="volume" title="volume">282</span>, <span class="fpage" tagx="fpage" title="fpage">20141958</span> ( <span class="year" tagx="year" title="year">2015</span>). </span> </li> <li tag="ref"> <a name="b12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pearson</span> <span class="given-names" tagx="given-names" title="given-names">R. G.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dawson</span> <span class="given-names" tagx="given-names" title="given-names">T. P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Predicting the impacts of climate change on the distribution of species: are bioclimate envelope models useful </span>. <span class="source" tagx="source" title="source">Global Ecol. Biogeogr.</span> <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">361</span>– <span class="lpage" tagx="lpage" title="lpage">371</span> ( <span class="year" tagx="year" title="year">2003</span>). </span> </li> <li tag="ref"> <a name="b13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbraud</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weimerskirch</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Emperor penguins and climate change</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">411</span>, <span class="fpage" tagx="fpage" title="fpage">183</span>– <span class="lpage" tagx="lpage" title="lpage">186</span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11346792">11346792</a> </span> </span> </li> <li tag="ref"> <a name="b14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ainley</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Impacts of cetaceans on the structure of Southern Ocean food webs</span>. <span class="source" tagx="source" title="source">Mar. Mammal Sci.</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">482</span>– <span class="lpage" tagx="lpage" title="lpage">498</span> ( <span class="year" tagx="year" title="year">2010</span>). </span> </li> <li tag="ref"> <a name="b15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenouvrier</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Projected continent-wide declines of the emperor penguin under climate change</span>. <span class="source" tagx="source" title="source">Nat. Clim. Change</span> <span class="volume" tagx="volume" title="volume">4</span>, <span class="fpage" tagx="fpage" title="fpage">715</span>– <span class="lpage" tagx="lpage" title="lpage">718</span> ( <span class="year" tagx="year" title="year">2014</span>). </span> </li> <li tag="ref"> <a name="b16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ehrlén</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morris</span> <span class="given-names" tagx="given-names" title="given-names">W. F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Predicting changes in the distribution and abundance of species under environmental change </span>. <span class="source" tagx="source" title="source">Ecol. Lett.</span> <span class="volume" tagx="volume" title="volume">18</span>, <span class="fpage" tagx="fpage" title="fpage">303</span>– <span class="lpage" tagx="lpage" title="lpage">314</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25611188">25611188</a> </span> </span> </li> <li tag="ref"> <a name="b17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ainley</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Antarctic penguin response to habitat change as Earth's troposphere reaches 2° C above preindustrial levels </span>. <span class="source" tagx="source" title="source">Ecol. Monogr.</span> <span class="volume" tagx="volume" title="volume">80</span>, <span class="fpage" tagx="fpage" title="fpage">49</span>– <span class="lpage" tagx="lpage" title="lpage">66</span> ( <span class="year" tagx="year" title="year">2010</span>). </span> </li> <li tag="ref"> <a name="b18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trathan</span> <span class="given-names" tagx="given-names" title="given-names">P. N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fretwell</span> <span class="given-names" tagx="given-names" title="given-names">P. T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stonehouse</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="mixed-article-title" title="mixed-article-title">First recorded loss of an emperor penguin colony in the recent period of Antarctic regional warming: implications for other colonies </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">6</span>, <span class="fpage" tagx="fpage" title="fpage">e14738</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21386883">21386883</a> </span> </span> </li> <li tag="ref"> <a name="b19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kooyman</span> <span class="given-names" tagx="given-names" title="given-names">G. L.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ponganis</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Chick production at the largest emperor penguin colony decreases by 50% from 2008–10</span>. <span class="source" tagx="source" title="source">Antarct. Sci.</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">33</span>– <span class="lpage" tagx="lpage" title="lpage">37</span> ( <span class="year" tagx="year" title="year">2014</span>). </span> </li> <li tag="ref"> <a name="b20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ancel</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Emperors in hiding: when ice-breakers and satellites complement each other in Antarctic exploration </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e100404</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24963661">24963661</a> </span> </span> </li> <li tag="ref"> <a name="b21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forcada</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trathan</span> <span class="given-names" tagx="given-names" title="given-names">P. N.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Penguin responses to climate change in the Southern Ocean</span>. <span class="source" tagx="source" title="source">Glob. Chang. Biol</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">1618</span>– <span class="lpage" tagx="lpage" title="lpage">1630</span> ( <span class="year" tagx="year" title="year">2009</span>). </span> </li> <li tag="ref"> <a name="b22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">LaRue</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kooyman</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lynch</span> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fretwell</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Emigration in emperor penguins: implications for interpretation of long-term studies</span>. <span class="source" tagx="source" title="source">Ecography</span> <span class="volume" tagx="volume" title="volume">38</span>, <span class="fpage" tagx="fpage" title="fpage">114</span>– <span class="lpage" tagx="lpage" title="lpage">120</span> ( <span class="year" tagx="year" title="year">2015</span>). </span> </li> <li tag="ref"> <a name="b23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thiebot</span> <span class="given-names" tagx="given-names" title="given-names">J.-B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lescroël</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbraud</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bost</span> <span class="given-names" tagx="given-names" title="given-names">C.-A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Three-dimensional use of marine habitats by juvenile emperor penguins <i>Aptenodytes forsteri</i> during post-natal dispersal </span>. <span class="source" tagx="source" title="source">Antarct. Sci.</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">536</span>– <span class="lpage" tagx="lpage" title="lpage">544</span> ( <span class="year" tagx="year" title="year">2013</span>). </span> </li> <li tag="ref"> <a name="b24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baird</span> <span class="given-names" tagx="given-names" title="given-names">N. A.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">e3376</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878">18852878</a> </span> </span> </li> <li tag="ref"> <a name="b25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Molfetti</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Recent demographic history and present fine-scale structure in the Northwest Atlantic leatherback ( <i>Dermochelys coriacea</i>) turtle population </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">e58061</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23516429">23516429</a> </span> </span> </li> <li tag="ref"> <a name="b26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Friesen</span> <span class="given-names" tagx="given-names" title="given-names">V. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Speciation in seabirds: why are there so many species… and why aren't there more</span>. <span class="source" tagx="source" title="source">J. Ornithol.</span> <span class="volume" tagx="volume" title="volume">156</span>, <span class="fpage" tagx="fpage" title="fpage">S27</span>– <span class="lpage" tagx="lpage" title="lpage">S39</span> ( <span class="year" tagx="year" title="year">2015</span>). </span> </li> <li tag="ref"> <a name="b27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hoelzel</span> <span class="given-names" tagx="given-names" title="given-names">A. R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genetic structure of cetacean populations in sympatry, parapatry, and mixed assemblages: implications for conservation policy </span>. <span class="source" tagx="source" title="source">J. Hered.</span> <span class="volume" tagx="volume" title="volume">89</span>, <span class="fpage" tagx="fpage" title="fpage">451</span>– <span class="lpage" tagx="lpage" title="lpage">458</span> ( <span class="year" tagx="year" title="year">1998</span>). </span> </li> <li tag="ref"> <a name="b28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reich</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thangaraj</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patterson</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">A. L.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Reconstructing Indian population history</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">461</span>, <span class="fpage" tagx="fpage" title="fpage">489</span>– <span class="lpage" tagx="lpage" title="lpage">494</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19779445">19779445</a> </span> </span> </li> <li tag="ref"> <a name="b29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Romiguier</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Comparative population genomics in animals uncovers the determinants of genetic diversity</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">515</span>, <span class="fpage" tagx="fpage" title="fpage">261</span>– <span class="lpage" tagx="lpage" title="lpage">263</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25141177">25141177</a> </span> </span> </li> <li tag="ref"> <a name="b30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bouckaert</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">BEAST 2: a software platform for Bayesian evolutionary analysis</span>. <span class="source" tagx="source" title="source">PLoS Comput. Biol.</span> <span class="volume" tagx="volume" title="volume">10</span>, <span class="fpage" tagx="fpage" title="fpage">e1003537</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24722319">24722319</a> </span> </span> </li> <li tag="ref"> <a name="b31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Two Antarctic penguin genomes reveal insights into their evolutionary history and molecular changes related to the Antarctic environment </span>. <span class="source" tagx="source" title="source">GigaScience</span> <span class="volume" tagx="volume" title="volume">3</span>, <span class="fpage" tagx="fpage" title="fpage">27</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25671092">25671092</a> </span> </span> </li> <li tag="ref"> <a name="b32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trucchi</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">King penguin demography since the last glaciation inferred from genome-wide data</span>. <span class="source" tagx="source" title="source">P. Roy. Soc. Lond. B Bio.</span> <span class="volume" tagx="volume" title="volume">281</span>, <span class="fpage" tagx="fpage" title="fpage">20140528</span> ( <span class="year" tagx="year" title="year">2014</span>). </span> </li> <li tag="ref"> <a name="b33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chikhi</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sousa</span> <span class="given-names" tagx="given-names" title="given-names">V. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luisi</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goossens</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beaumont</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">The confounding effects of population structure, genetic diversity and the sampling scheme on the detection and quantification of population size changes </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">186</span>, <span class="fpage" tagx="fpage" title="fpage">983</span>– <span class="lpage" tagx="lpage" title="lpage">995</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20739713">20739713</a> </span> </span> </li> <li tag="ref"> <a name="b34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dupanloup</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huerta-Sanchez</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sousa</span> <span class="given-names" tagx="given-names" title="given-names">V. C.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foll</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Robust demographic inference from genomic and SNP data</span>. <span class="source" tagx="source" title="source">PLoS Genet.</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e1003905</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24204310">24204310</a> </span> </span> </li> <li tag="ref"> <a name="b35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fretwell</span> <span class="given-names" tagx="given-names" title="given-names">P. T.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">An emperor penguin population estimate: the first global, synoptic survey of a species from space </span>. <span class="source" tagx="source" title="source">PLoS ONE</span> <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">e33751</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22514609">22514609</a> </span> </span> </li> <li tag="ref"> <a name="b36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbraud</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gavrilo</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mizin</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weimerskirch</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of emperor penguin declines between Pointe Géologie and Haswell Island over the past 50 years </span>. <span class="source" tagx="source" title="source">Antarct. Sci.</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">461</span>– <span class="lpage" tagx="lpage" title="lpage">468</span> ( <span class="year" tagx="year" title="year">2011</span>). </span> </li> <li tag="ref"> <a name="b37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Le Maho</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">An ethical issue in biodiversity science: the monitoring of penguins with flipper bands </span>. <span class="source" tagx="source" title="source">C. R. Biol.</span> <span class="volume" tagx="volume" title="volume">334</span>, <span class="fpage" tagx="fpage" title="fpage">378</span>– <span class="lpage" tagx="lpage" title="lpage">384</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21640946">21640946</a> </span> </span> </li> <li tag="ref"> <a name="b38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beerli</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palczewski</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Unified framework to evaluate panmixia and migration direction among multiple sampling locations </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">185</span>, <span class="fpage" tagx="fpage" title="fpage">313</span>– <span class="lpage" tagx="lpage" title="lpage">326</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20176979">20176979</a> </span> </span> </li> <li tag="ref"> <a name="b39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burridge</span> <span class="given-names" tagx="given-names" title="given-names">C. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peucker</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valautham</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Styan</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dann</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Nonequilibrium conditions explain spatial variability in genetic structuring of little penguin ( <i>Eudyptula minor</i>) </span>. <span class="source" tagx="source" title="source">J. Hered.</span> <span class="volume" tagx="volume" title="volume">106</span>, <span class="fpage" tagx="fpage" title="fpage">228</span>– <span class="lpage" tagx="lpage" title="lpage">237</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25833231">25833231</a> </span> </span> </li> <li tag="ref"> <a name="b40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mayr</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="source" tagx="source" title="source">Populations, Species, and Evolution: An Abridgment of Animal Species and Evolution</span> Harvard Univ. ( <span class="year" tagx="year" title="year">1970</span>). </span> </li> <li tag="ref"> <a name="b41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lenormand</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Gene flow and the limits to natural selection</span>. <span class="source" tagx="source" title="source">Trends Ecol. Evol.</span> <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">183</span>– <span class="lpage" tagx="lpage" title="lpage">189</span> ( <span class="year" tagx="year" title="year">2002</span>). </span> </li> <li tag="ref"> <a name="b42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zink</span> <span class="given-names" tagx="given-names" title="given-names">R. M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barrowclough</span> <span class="given-names" tagx="given-names" title="given-names">G. F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Mitochondrial DNA under siege in avian phylogeography</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">17</span>, <span class="fpage" tagx="fpage" title="fpage">2107</span>– <span class="lpage" tagx="lpage" title="lpage">2121</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18397219">18397219</a> </span> </span> </li> <li tag="ref"> <a name="b43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dierickx</span> <span class="given-names" tagx="given-names" title="given-names">E. G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shultz</span> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sato</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiraoka</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edwards</span> <span class="given-names" tagx="given-names" title="given-names">S. V.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Morphological and genomic comparisons of Hawaiian and Japanese Black footed Albatrosses ( <i>Phoebastria nigripes</i>) using double digest RADseq: implications for conservation </span>. <span class="source" tagx="source" title="source">Evol. Appl</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">662</span>– <span class="lpage" tagx="lpage" title="lpage">678</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26240604">26240604</a> </span> </span> </li> <li tag="ref"> <a name="b44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Catchen</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hohenlohe</span> <span class="given-names" tagx="given-names" title="given-names">P. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bassham</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Amores</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cresko</span> <span class="given-names" tagx="given-names" title="given-names">W. A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Stacks: an analysis tool set for population genomics</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">22</span>, <span class="fpage" tagx="fpage" title="fpage">3124</span>– <span class="lpage" tagx="lpage" title="lpage">3140</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23701397">23701397</a> </span> </span> </li> <li tag="ref"> <a name="b45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lambert</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genomic data from the Emperor penguin ( <i>Aptenodytes forsteri</i>) </span>. <span class="source" tagx="source" title="source">GigaScience</span> ( <span class="year" tagx="year" title="year">2011</span>). </span> </li> <li tag="ref"> <a name="b46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langmead</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salzberg</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span>. <span class="source" tagx="source" title="source">Nat. Methods</span> <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">357</span>– <span class="lpage" tagx="lpage" title="lpage">359</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a> </span> </span> </li> <li tag="ref"> <a name="b47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">The sequence alignment/map format and SAMtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">2078</span>– <span class="lpage" tagx="lpage" title="lpage">2079</span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a> </span> </span> </li> <li tag="ref"> <a name="b48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DePristo</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data </span>. <span class="source" tagx="source" title="source">Nature Genet.</span> <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">491</span>– <span class="lpage" tagx="lpage" title="lpage">498</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889">21478889</a> </span> </span> </li> <li tag="ref"> <a name="b49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korneliussen</span> <span class="given-names" tagx="given-names" title="given-names">T. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albrechtsen</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">ANGSD: analysis of next generation sequencing data</span>. <span class="source" tagx="source" title="source">BMC Bioinform.</span> <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">356</span> ( <span class="year" tagx="year" title="year">2014</span>). </span> </li> <li tag="ref"> <a name="b50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fumagalli</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Quantifying population genetic differentiation from next-generation sequencing data</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">195</span>, <span class="fpage" tagx="fpage" title="fpage">979</span>– <span class="lpage" tagx="lpage" title="lpage">992</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23979584">23979584</a> </span> </span> </li> <li tag="ref"> <a name="b51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Danecek</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">The variant call format and VCFtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">2156</span>– <span class="lpage" tagx="lpage" title="lpage">2158</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522">21653522</a> </span> </span> </li> <li tag="ref"> <a name="b52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">Complex evolutionary trajectories of sex chromosomes across bird taxa</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">346</span>, <span class="fpage" tagx="fpage" title="fpage">1246338</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25504727">25504727</a> </span> </span> </li> <li tag="ref"> <a name="b53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reynolds</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weir</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cockerham</span> <span class="given-names" tagx="given-names" title="given-names">C. C.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimation of the coancestry coefficient: basis for a short-term genetic distance</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">105</span>, <span class="fpage" tagx="fpage" title="fpage">767</span>– <span class="lpage" tagx="lpage" title="lpage">779</span> ( <span class="year" tagx="year" title="year">1983</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17246175">17246175</a> </span> </span> </li> <li tag="ref"> <a name="b54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fumagalli</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vieira</span> <span class="given-names" tagx="given-names" title="given-names">F. G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Linderoth</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nielsen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="mixed-article-title" title="mixed-article-title">ngsTools: methods for population genetics analyses from Next-Generation Sequencing data </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">1486</span>– <span class="lpage" tagx="lpage" title="lpage">1487</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24458950">24458950</a> </span> </span> </li> <li tag="ref"> <a name="b55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jombart</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="mixed-article-title" title="mixed-article-title">adegenet: a R package for the multivariate analysis of genetic markers</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">1403</span>– <span class="lpage" tagx="lpage" title="lpage">1405</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18397895">18397895</a> </span> </span> </li> <li tag="ref"> <a name="b56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paradis</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="mixed-article-title" title="mixed-article-title">pegas: an R package for population genetics with an integrated-modular approach</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">419</span>– <span class="lpage" tagx="lpage" title="lpage">420</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20080509">20080509</a> </span> </span> </li> <li tag="ref"> <a name="b57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laval</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneider</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Arlequin (version 3.0): an integrated software package for population genetics data analysis </span>. <span class="source" tagx="source" title="source">Evol. Bioinform</span> <span class="volume" tagx="volume" title="volume">1</span>, <span class="fpage" tagx="fpage" title="fpage">47</span>– <span class="lpage" tagx="lpage" title="lpage">50</span> ( <span class="year" tagx="year" title="year">2005</span>). </span> </li> <li tag="ref"> <a name="b58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purcell</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="source" tagx="source" title="source">Am. J. Hum. Genet.</span> <span class="volume" tagx="volume" title="volume">81</span>, <span class="fpage" tagx="fpage" title="fpage">559</span>– <span class="lpage" tagx="lpage" title="lpage">575</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17701901">17701901</a> </span> </span> </li> <li tag="ref"> <a name="b59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huson</span> <span class="given-names" tagx="given-names" title="given-names">D. H.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Application of phylogenetic networks in evolutionary studies</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">254</span>– <span class="lpage" tagx="lpage" title="lpage">267</span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16221896">16221896</a> </span> </span> </li> <li tag="ref"> <a name="b60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jombart</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Devillard</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Balloux</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Discriminant analysis of principal components: a new method for the analysis of genetically structured populations </span>. <span class="source" tagx="source" title="source">BMC Genet.</span> <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">94</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20950446">20950446</a> </span> </span> </li> <li tag="ref"> <a name="b61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pickrell</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pritchard</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Inference of population splits and mixtures from genome-wide allele frequency data</span>. <span class="source" tagx="source" title="source">PLoS Genet.</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">e1002967</span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23166502">23166502</a> </span> </span> </li> <li tag="ref"> <a name="b62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Raj</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephens</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pritchard</span> <span class="given-names" tagx="given-names" title="given-names">J. K.</span> </span> <span class="mixed-article-title" title="mixed-article-title">fastSTRUCTURE: variational inference of population structure in large SNP data sets</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">197</span>, <span class="fpage" tagx="fpage" title="fpage">573</span>– <span class="lpage" tagx="lpage" title="lpage">589</span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24700103">24700103</a> </span> </span> </li> <li tag="ref"> <a name="b63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Skotte</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Korneliussen</span> <span class="given-names" tagx="given-names" title="given-names">T. S.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albrechtsen</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Estimating individual admixture proportions from next generation sequencing data</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">195</span>, <span class="fpage" tagx="fpage" title="fpage">693</span>– <span class="lpage" tagx="lpage" title="lpage">702</span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24026093">24026093</a> </span> </span> </li> <li tag="ref"> <a name="b64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heled</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Drummond</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Bayesian inference of population size history from multiple loci</span>. <span class="source" tagx="source" title="source">BMC Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">289</span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18947398">18947398</a> </span> </span> </li> <li tag="ref"> <a name="b65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quinlan</span> <span class="given-names" tagx="given-names" title="given-names">A. R.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">I. M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">841</span>– <span class="lpage" tagx="lpage" title="lpage">842</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278">20110278</a> </span> </span> </li> <li tag="ref"> <a name="b66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lambert</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Genomic data from the Adélie penguin (Pygoscelis adeliae)</span>. <span class="source" tagx="source" title="source">GigaScience</span> ( <span class="year" tagx="year" title="year">2011</span>). </span> </li> <li tag="ref"> <a name="b67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guindon</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <i>et al.</i> <span class="mixed-article-title" title="mixed-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0 </span>. <span class="source" tagx="source" title="source">Syst. Biol.</span> <span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">307</span>– <span class="lpage" tagx="lpage" title="lpage">321</span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638">20525638</a> </span> </span> </li> <li tag="ref"> <a name="b68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="mixed-article-title" title="mixed-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span>. <span class="source" tagx="source" title="source">Mol. Biol. Evol.</span> <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">1586</span>– <span class="lpage" tagx="lpage" title="lpage">1591</span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113">17483113</a> </span> </span> </li> <li tag="ref"> <a name="b69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Excoffier</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foll</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Fastsimcoal: a continuous-time coalescent simulator of genomic diversity under arbitrarily complex evolutionary scenarios </span>. <span class="source" tagx="source" title="source">Bioinformatics</span> <span class="volume" tagx="volume" title="volume">27</span>, <span class="fpage" tagx="fpage" title="fpage">1332</span>– <span class="lpage" tagx="lpage" title="lpage">1334</span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21398675">21398675</a> </span> </span> </li> <li tag="ref"> <a name="b70" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbraud</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Delord</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> &amp;amp; <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weimerskirch</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="mixed-article-title" title="mixed-article-title">Extreme ecological response of a seabird community to unprecedented sea ice cover</span>. <span class="source" tagx="source" title="source">R. Soc. Open Sci</span> <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">140456</span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26064653">26064653</a> </span> </span> </li> </ul> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Author contributions</b> R.C., E.T. and C.L.B. conceived the project, designed the study and co-wrote the manuscript. A.A. brought ideas to the experiment design. R.C. and E.T. performed library preparation, RAD mapping and main analyses. G.B. designed the joint-SFS analysis, A.B., E.Z., R.C. and E.T. performed the joint-SFS demographic reconstructions. A.A., C.L.B., P.J.P., P.N.T., and D.P.Z. provided the samples. N.C.S. hosted the project. A.A., A.B., G.B., Y.L.M., P.J.P., N.C.S., P.N.T., J.D.W., E.Z. and D.P.Z. contributed to the development of the manuscript. </p> </div> </div> </div>  </body></html>