<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Evol Lett</span> <span class="iso-abbrev" title="iso-abbrev">Evol Lett</span> <span class="doi" title="doi">10.1002/(ISSN)2056-3744</span> <span class="publisher-id" title="publisher-id">EVL3</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Evolution Letters</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2056-3744</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley and Sons Inc.</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6591552">6591552</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1002/evl3.103">10.1002/evl3.103</a> </span> <span class="publisher-id" title="publisher-id">EVL3103</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Letter</span>: <span class="subject" title="subject">Letters</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Genomic selection on breeding time in a wild bird population</div> <div class="alt-title" title="alt-title">
P. GIENAPP ET AL.</div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gienapp</span> <span class="given-names" tagx="given-names" title="given-names">Phillip</span> </span> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-9368-8769</span> <span class="address" title="address"> <span class="email" tagx="email" title="email">p.gienapp@nioo.knaw.nl</span> </span> <a href="#evl3103-aff-0001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Calus</span> <span class="given-names" tagx="given-names" title="given-names">Mario P. L.</span> </span> <a href="#evl3103-aff-0002"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Laine</span> <span class="given-names" tagx="given-names" title="given-names">Veronika N.</span> </span> <a href="#evl3103-aff-0001"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Visser</span> <span class="given-names" tagx="given-names" title="given-names">Marcel E.</span> </span> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1456-1939</span> <a href="#evl3103-aff-0001"> <sup>1</sup> </a> <a href="#evl3103-aff-0002"> <sup>2</sup> </a> </span> </div> <span class="citation_author_institution" id="evl3103-aff-0001">[1], <span class="named-content" title="named-content">Department of Animal Ecology</span> <span class="institution" title="institution">Netherlands Institute of Ecology (NIOO‐KNAW)</span> <span class="named-content" title="named-content">Wageningen</span> <span class="country" tagx="country" title="country">The Netherlands</span> </span> <span class="citation_author_institution" id="evl3103-aff-0002">[2], <span class="named-content" title="named-content">Animal Breeding and Genomics</span> <span class="institution" title="institution">Wageningen University &amp;amp; Research</span> <span class="named-content" title="named-content">Wageningen</span> <span class="country" tagx="country" title="country">The Netherlands</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-3-3</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-4</span> </span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="issue-id" tagx="issue-id" title="issue-id">10.1002/evl3.2019.3.issue-2</span> <span class="fpage" tagx="fpage" title="fpage">142</span> <span class="lpage" tagx="lpage" title="lpage">151</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-6-29</span> <span class="accepted" title="accepted">accepted: 2019-1-30</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <a class="self-uri" href="file:EVL3-3-142.pdf" title="self-uri">file:EVL3-3-142.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Artificial selection experiments are a powerful tool in evolutionary biology. Selecting individuals based on multimarker genotypes (genomic selection) has several advantages over phenotype‐based selection but has, so far, seen very limited use outside animal and plant breeding. Genomic selection depends on the markers tagging the causal loci that underlie the selected trait. Because the number of necessary markers depends, among other factors, on effective population size, genomic selection may be in practice not feasible in wild populations as most wild populations have much higher effective population sizes than domesticated populations. However, the current possibilities of cost‐effective high‐throughput genotyping could overcome this limitation and thereby make it possible to apply genomic selection also in wild populations. Using a unique dataset of about 2000 wild great tits ( <i>Parus major</i>), a small passerine bird, genotyped on a 650 k SNP chip we calculated genomic breeding values for egg‐laying date using the so‐called GBLUP approach. In this approach, the pedigree‐based relatedness matrix of an "animal model," a special form of the mixed model, is replaced by a marker‐based relatedness matrix. Using the marker‐based relatedness matrix, the model seemed better able to disentangle genetic and permanent environmental effects. We calculated the accuracy of genomic breeding values by correlating them to the phenotypes of individuals whose phenotypes were excluded from the analysis when estimating the genomic breeding values. The obtained accuracy was about 0.20, with very little effect of the used genomic relatedness estimator but a strong effect of the number of SNPs. The obtained accuracy is lower than typically seen in domesticated species but considerable for a trait with low heritability (∼0.2) as avian breeding time. Our results show that genomic selection is possible also in wild populations with potentially many applications, which we discuss here. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
Animal model</div> <div class="kwd" title="kwd">
GBLUP</div> <div class="kwd" title="kwd">
genomic breeding values</div> <div class="kwd" title="kwd">
phenology</div> <div class="kwd" title="kwd">
quantitative genetic</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
European Research Council</div> <span class="award-id" tagx="award-id" title="award-id">339092 ‐ E‐Response</span> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="source-schema-version-number" value="2.0" /> <meta name="component-id" value="evl3103" /> <meta name="cover-date" value="April 2019" /> <meta name="details-of-publishers-convertor" value="Converter:WILEY_ML3GV2_TO_NLMPMC version:5.6.5 mode:remove_FC converted:25.06.2019" /> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>All data will be deposited in a public archive (Dryad) after acceptance and before publication of the manuscript. </p> </div> </div> </div> <div class="body" title="body"> <div class="boxed-text" tagx="boxed-text" title="boxed-text">  <div class="title" tagx="title" title="title">
Impact summary</div>  <p>Selecting individuals based on their genotypes instead of their phenotypes is already widely applied in animal and plant breeding. This "genomic selection" has several advantages over "traditional" selection based on phenotypes, for example, individuals can be selected before they express their phenotype, which can considerably speed up selection in phenotypes that take a long time to measure or are difficult to obtain. Selection experiments can be a powerful tool in evolutionary biology and genomic selection could obviously be useful here. However, it has so far been unclear whether genomic selection will be feasible in natural populations that differ in important parameters, for example, effective population size, from populations of domestic breeds. We here explored whether genomic selection worked in a natural population of great tits ( <i>Parus major</i>), a small, common songbird species. We could show that the accuracy, an important parameter determining the efficiency of genomic selection, of "genomic breeding values" for egg‐laying date in this population was moderate but lower than typical for animal and plant breeding. Despite this reduced accuracy, caused by the high effective population size of our large great tit study population, our results show that genomic selection can be possible in natural populations and we discuss a number of potential applications next to selection experiments. </p> </div> <p>Dissecting a trait's genetic architecture, i.e. how many (and also which) loci affect the trait and how their effect sizes are distributed, is important to fully understand and predict its evolution. Quantitative genetics assumes that a trait is determined by many loci of small effects. This assumption, the infinitesimal model (Barton et al. <a href="#evl3103-bib-0005">2017</a>), allows to model the traits’ genetic (co)variances and its evolutionary change, the response to selection, based on phenotypic resemblance among individuals of known relatedness without any molecular genetic information. Despite its unrealistic assumptions that, for example, totally ignore gene‐by‐gene interactions, for which quantitative genetics and the infinitesimal model have been criticized (Nelson et al. <a href="#evl3103-bib-0058">2013</a>), this framework has been highly successful in animal and plant breeding (Hill <a href="#evl3103-bib-0040">2012</a>) but also in natural populations (Charmantier et al. <a href="#evl3103-bib-0012">2014</a>). </p> <p>The advances in molecular genetic techniques have led to increasing information about the molecular genetic architecture of traits. Gene mapping studies have been highly successful in mapping the genes underlying a variety of traits in humans (Visscher et al. <a href="#evl3103-bib-0074">2017</a>) but also other traits even in natural populations, as, for example, bill morphology in Darwin finches (Abzhanov et al. <a href="#evl3103-bib-0002">2004</a>, <a href="#evl3103-bib-0001">2006</a>), defensive armament traits in sticklebacks (Colosimo et al. <a href="#evl3103-bib-0013">2004</a>) or life‐history in salmon (Barson et al. <a href="#evl3103-bib-0004">2015</a>). Such gene mapping studies have, however, not been universally successful. In many cases, the identified causal loci could only explain a small part of the phenotypic variance (Manolio et al. <a href="#evl3103-bib-0052">2009</a>). This problem of "missing heritability" is now mostly resolved by an increasing number of markers included in the analyses (Visscher et al. <a href="#evl3103-bib-0074">2017</a>). The advances in molecular genetic techniques have now made it possible to undertake gene mapping studies in wild populations of "nonmodel" species. Unsurprisingly, however, these studies had limited power and often failed to identify any locus associated with the analyzed traits (see Supplementary Table in Gienapp et al. <a href="#evl3103-bib-0026">2017b</a>) indicating that many traits likely have a polygenic architecture. It hence seems that, despite its simplistic and not necessarily realistic assumptions (Nelson et al. <a href="#evl3103-bib-0058">2013</a>), the infinitesimal model of classical quantitative genetic theory is a reasonable approximation for the genetic architecture of many traits. </p> <p>Quantitative genetics allows the prediction of individual "breeding values," that is the sum of the additive genetic effects on the trait (Lynch and Walsh <a href="#evl3103-bib-0051">1998</a>; Mrode <a href="#evl3103-bib-0057">2014</a>), which are commonly used to select individuals in animal breeding but also allow testing for genetic differentiation, with some caveats (Hadfield et al. <a href="#evl3103-bib-0035">2010</a>), in space or time in natural populations (e.g., Garant et al. <a href="#evl3103-bib-0022">2004</a>; Gienapp and Merilä <a href="#evl3103-bib-0028">2014</a>). Quantitative genetic analysis requires information about relatedness among individuals, for example, from a pedigree. Establishing a pedigree from observational field work data is, however, only feasible in taxa with parental care where individuals can be individually and uniquely marked at the age when parental care still occurs. This means that quantitative genetic studies are generally biased toward certain bird and mammal species (Charmantier et al. <a href="#evl3103-bib-0012">2014</a>). An alternative, proposed already some time ago (e.g., Ritland <a href="#evl3103-bib-0061">1996</a>), is to use molecular markers to estimate relatedness among individuals instead of pedigrees. Early applications of this approach generally suffered from the limited number of markers that were available (Coltman <a href="#evl3103-bib-0014">2005</a>; Garant and Kruuk <a href="#evl3103-bib-0021">2005</a>) but using the currently available high‐throughput genotyping in "nonmodel" species may overcome this problem (Gienapp et al. <a href="#evl3103-bib-0024">2017a</a>). </p> <p>In domesticated species high‐density marker panels have been available for quite some time. Starting with the influential paper by Meuwissen et al. ( <a href="#evl3103-bib-0054">2001</a>) the marker‐based prediction of breeding values, called "genomic selection" or "genomic prediction," became a very widely accepted and applied tool in animal and plant breeding (Jannink et al. <a href="#evl3103-bib-0042">2010</a>; Meuwissen et al. <a href="#evl3103-bib-0055">2016</a>). "Genomic" breeding values (GEBVs), i.e. breeding values predicted from high‐density markers, are generally more accurate than pedigree‐based breeding values (Meuwissen et al. <a href="#evl3103-bib-0055">2016</a>). Furthermore, GEBVs have the advantage that they allow reasonably accurate predictions about the performance of individuals in the absence of own phenotypes (or offspring phenotypes in case of traits expressed only in the other sex). Genomic prediction relies on that the markers "tag" a sufficiently large number of the loci that determine the trait, that is each QTL (quantitative trait locus) is in linkage disequilibrium with at least one marker. This means that the accuracy of GEBVs depends, among other variables, on the number of phenotypic records and the extent of linkage disequilibrium that can be reflected by the number of independently segregating chromosome segments ( <i>M <sub>e</sub> </i>) (e.g., Goddard <a href="#evl3103-bib-0032">2009</a>; Daetwyler et al. <a href="#evl3103-bib-0017">2010</a>). </p> <p>The development of the necessary genomic tools for genomic selection in non‐domesticated species has generally been lagging behind but now high‐throughput high‐density genotyping has become possible in virtually any species. This potentially widens the scope of quantitative genetic studies in wild populations by allowing to use marker‐based relatedness instead of pedigree‐based relatedness and could enable us also to apply genomic selection in non‐domesticated species (Gienapp et al. <a href="#evl3103-bib-0024">2017a</a>). As pointed out above, the accuracy of GEBVs, and hence the feasibility of this approach, depends on <i>M <sub>e</sub> </i>, which in turns depends on the effective population size ( <i>N <sub>e</sub> </i>) (e.g., Visscher et al. <a href="#evl3103-bib-0073">2006</a>). The effective population size is, however, generally much larger in natural than in domestic populations due to, for example, larger population census sizes, different mating systems and less skewed reproductive success. This will impair the feasibility of this approach and, so far, very few studies successfully predicted GEBVs in wild populations (e.g., Beaulieu et al. <a href="#evl3103-bib-0006">2014</a>). </p> <p>We here applied genomic selection on seasonal breeding time in a wild population of great tits ( <i>Parus major</i>). Knowledge about a trait's genetic architecture is also important to understand and predict its evolutionary response to selection. Climate change is generally expected to lead to selection on phenology (Gienapp et al. <a href="#evl3103-bib-0030">2014</a>) and it has been shown to lead to selection on phenology by disrupting synchrony between trophic levels in great tits (Visser et al. <a href="#evl3103-bib-0076">1998</a>) and other species (Visser and Both <a href="#evl3103-bib-0075">2005</a>). Whether populations or species will be able to successfully respond to selection depends, among other factors, on the genetic variance of the trait under selection (e.g., Lynch and Lande <a href="#evl3103-bib-0050">1993</a>; Bürger and Lynch <a href="#evl3103-bib-0010">1995</a>; Gienapp et al. <a href="#evl3103-bib-0027">2013a</a>). Avian breeding time is heritable in great tits (e.g., McCleery et al. <a href="#evl3103-bib-0053">2004</a>; Gienapp et al. <a href="#evl3103-bib-0029">2006</a>) but it is currently unclear which part of the "physiological cascade" underlying this trait varies genetically. By creating individuals with extreme phenotypes and studying their physiology we hope to be able to address this question. To create these extreme individuals we applied genomic selection in laboratory selection lines of great tits. </p> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="phenotypes/trainingpopulation" title="sec"> <div class="title" tagx="title" title="title">
PHENOTYPES/TRAINING POPULATION</div> <p>Our "training population," that is the set of individuals for which phenotypes and genotypes are recorded, consisted of several long‐term study populations of wild great tits ( <i>Parus major</i>) in the Southern part of the Veluwe area close to Arnhem (52° 00′ N, 5° 50′ E, the Netherlands). These populations are located in close vicinity (max. distance: 5 km) within a large contiguous woodland area. Great tits are small passerines that breed in natural cavities but readily accept artificial nest boxes. In all study populations nest boxes are supplied in overabundance so that almost all great tits breed in the supplied nest boxes. Nest‐boxes are checked weekly for signs of nest building and clutch initiation starting in the beginning of April. When a nest with eggs is found, the date of the first egg laid (hereafter egg‐laying date) is back‐calculated on the assumption that one egg is laid per day. All nestlings are ringed with standard aluminium bird rings at an age of seven days. Adults are caught in the nest boxes during the chick feeding period and identified by their rings or ringed if still unringed. This allowed the construction of a pedigree based on this observational data. In the recent decades adults and chicks have also been blood‐sampled at capture. Blood samples were stored in either 1 mL Cell Lysis Solution (Gentra Puregene Kit, Qiagen, USA) or Queens buffer (Seutin et al. <a href="#evl3103-bib-0064">1991</a>). </p> </div> <div class="genotypingandqualitycontrol" title="sec"> <div class="title" tagx="title" title="title">
GENOTYPING AND QUALITY CONTROL</div> <p>A total of 2015 female great tits were genotyped using a custom made Affymetrix great tit 650K SNP chip (Kim et al. <a href="#evl3103-bib-0043">2018</a>) at Edinburgh Genomics (Edinburgh, United Kingdom). Axiom Analysis Suite 1.1 was used for SNP calling following the Affymetrix best practices workflow. Total of 32,716 SNPs located on unassigned reads, that is without known genomic position, and on the Z chromosome were excluded. Altogether 503,199 SNPs passed initial quality control. From these, 248 nonpolymorphic SNPs were excluded. SNPs were not filtered for Hardy–Weinberg equilibrium or minor allele frequency. </p> </div> <div class="calculationofgebvs" title="sec"> <div class="title" tagx="title" title="title">
CALCULATION OF GEBVs</div> <p>There are two conceptually different ways to calculate genomic breeding values (GEBVs). In so‐called "whole genome regression" phenotypes are regressed simultaneously against all markers and the GEBV is calculated as the sum of all estimated marker effects multiplied with the corresponding genotypes (de los Campos et al. <a href="#evl3103-bib-0018">2013</a>). However, the fact that the number of estimated parameters, the marker effects, exceeds the number of observations prevents the use of "simple" least‐squares multiple regression and instead variable selection or shrinkage estimation procedures need to be used. Various approaches as, for example, ridge regression (Hoerl and Kennard <a href="#evl3103-bib-0041">1970</a>), LASSO (Tibshirani <a href="#evl3103-bib-0068">1996</a>) or Bayesian methods with various priors (e.g., Gianola <a href="#evl3103-bib-0023">2013</a>) have been proposed and used. The other approach is replacing the pedigree‐based relatedness matrix in an animal model by the marker‐based relatedness matrix (VanRaden <a href="#evl3103-bib-0071">2008</a>; Yang et al. <a href="#evl3103-bib-0079">2010</a>), generally known as GBLUP. While conceptually very different, this approach can be shown to be mathematically identical to ridge regression or a Bayesian approach with a Gaussian distribution of marker effects. Generally, the accuracies of GEBVs that can be obtained with these different approaches are comparable (e.g., Hayes et al. <a href="#evl3103-bib-0036">2009a</a>; Daetwyler et al. <a href="#evl3103-bib-0017">2010</a>; Gao et al. <a href="#evl3103-bib-0020">2013</a>). </p> <p>We here calculated GEBVs using the GBLUP approach. The marker‐based or "genomic" relatedness matrix (GRM) was calculated using the pairwise relatedness estimators of VanRaden ( <a href="#evl3103-bib-0071">2008</a>), method 1, and Yang et al. ( <a href="#evl3103-bib-0079">2010</a>) using the program calc_grm (Calus and Vandenplas <a href="#evl3103-bib-0011">2016</a>). When using the option VanRaden 1, the GRM is calculated following VanRaden ( <a href="#evl3103-bib-0071">2008</a>) as </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
ZZ</div> </div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
∑</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <div class="mspace_UNKNOWN" /> </div> </div> </div>with <b> <i>G</i> </b> being the GRM, <b> <i>Z</i> </b> being a centered matrix of marker genotypes of all individuals, and <i>p <sub>i</sub> </i> the frequency of the second allele at locus <i>i</i>. <b> <i>Z</i> </b> is calculated from the matrix of marker genotypes, coded as –1, 0, 1 for the homozygote, heterozygote, and other homozygote, by subtracting 2( <i>p <sub>i</sub> </i> – 0.5). Dividing <b> <i>ZZ</i> </b>′ by <div class="math_UNKNOWN"> <div class="mstyle_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="mo_UNKNOWN">
∑</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> scales <b> <i>G</i> </b> to be analogous to the relatedness matrix obtained from a pedigree. <p /> <p>When using the option Yang, the GRM is calculated following Yang et al. ( <a href="#evl3103-bib-0079">2010</a>) as </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
WW</div> </div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mi_UNKNOWN">
n</div> </div> <div class="mspace_UNKNOWN" /> </div> </div> </div>with <b> <i>G</i> </b> being the GRM, <b> <i>W</i> </b> being a matrix containing the scaled and centered marker genotypes and <i>n</i> being the number of markers. The elements of <b> <i>W</i> </b> are computed as <div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mspace_UNKNOWN" /> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
w</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
x</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
−</div> <div class="mn_UNKNOWN">
2</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> </div> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div> <div class="mspace_UNKNOWN" /> </div> </div> </div>with <i>x <sub>ij</sub> </i> being the marker genotype of individual <i>j</i> at locus <i>i</i>. <p /> <p>When a pedigree is known calc_grm offers the option to scale the GRM to the level of inbreeding in the pedigree following Powell et al. ( <a href="#evl3103-bib-0060">2010</a>). Finally, the GRM is adjusted for sampling error in the relatedness due to the limited number of markers following Yang et al. ( <a href="#evl3103-bib-0079">2010</a>). We here calculated the GRM with and without scaling according to the pedigree. </p> <p>To explore the relative importance of relationship information contained in the pedigree and in the genomic relatedness matrix, we calculated a "weighted" (genomic) relatedness matrix <b> <i>G</i> </b> <sup>*</sup> as </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
G</div> </div> <div class="mo_UNKNOWN">
∗</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
G</div> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
A</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
−</div> <div class="mi_UNKNOWN">
α</div> <div class="mo_UNKNOWN">
)</div> </div> </div> </div> </div>with <b> <i>G</i> </b> being the GRM, <b> <i>A</i> </b> being the pedigree‐based relatedness matrix and α the weighting factor ranging from 0 to 1. Here, we used values for α of 0.05, 0.5, 0.8, and 0.95. <p /> <p>Because egg‐laying date is affected by spring temperature (e.g., Gienapp et al. <a href="#evl3103-bib-0025">2005</a>) and can also differ among habitats, we fitted the following model to all recorded egg‐laying dates and used the year and area estimates from this model to "precorrect" the recorded phenotypes of the genotyped individuals: </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
y</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
a</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
age</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
ind</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
ε</div> </div> </div> </div>with <i>y <sub>i,j</sub> </i> being the phenotype of individual <i>i</i> in year <i>j</i>, <i>μ</i> the overall intercept, <i>yr <sub>j</sub> </i> and <i>ar <sub>a</sub> </i> the fixed effects for year (as factor) and area, respectively, <i>age <sub>i</sub> </i> the age of individual <i>i</i> (as factor, 1st year breeder vs older) and <i>ind <sub>i</sub> </i> the random effect of individual <i>i</i>. We did this, instead of fitting year and area in our GBLUP model, because not all individuals in all years were genotyped, which could have led to biases in the estimates for year‐area combinations with few genotyped individuals. <p /> <p>To estimate variance components and predict the GEBV at the same time we ran the following mixed model, our GBLUP model, in ASReml‐R: </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
μ</div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
age</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
p</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
e</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
ε</div> </div> </div> </div>with <i>y′ <sub>i,j</sub> </i> being the pre‐corrected phenotype of individual <i>i</i> in year <i>j</i>, that is <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="msubsup_UNKNOWN"> <div class="mrow_UNKNOWN" /> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
′</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
j</div> </div> </div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
y</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
j</div> </div> <div class="mo_UNKNOWN">
−</div> <div class="msub_UNKNOWN"> <div class="mover_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
r</div> </div> <div class="mo_UNKNOWN">
^</div> </div> <div class="mi_UNKNOWN">
a</div> </div> </div> </div>, <i>age <sub>i</sub> </i> the age of individual <i>i</i>, <i>pe <sub>i</sub> </i> the nongenetic (permanent environment) random effect of individual <i>i</i>, and <i>a <sub>i</sub> </i> the additive genetic random effect of individual <i>i</i>. The covariance between the additive genetic effects was given by the GRM or the weighted GRM ( <b> <i>G</i> </b> <sup>*</sup>). For comparison, we also fitted a pedigree‐based model where the covariance between the additive genetic effects was given the pedigree‐based relatedness matrix, while the model was otherwise identical. The scripts to calculate the GRM and to run the animal model are available as Supplementary Material. <p /> </div> <div class="validationofgebvs" title="sec"> <div class="title" tagx="title" title="title">
VALIDATION OF GEBVs</div> <p>In the "validation" step a subset of individuals that have both phenotypes and genotypes are excluded from the training population, and their precorrected phenotypes (see above) are regressed against the GEBVs predicted from their genotypes. The correlation between the GEBVs and phenotypes of these individuals is used to compute the accuracy of the GEBVs, which is defined as the correlation between the true and the estimated breeding values. Since the true breeding values are not known, phenotypes (P) that were precorrected for all fixed effects and averaged per individual are used instead: </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
accuracy</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
o</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
P</div> <div class="mo_UNKNOWN">
,</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
GEBV</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="msqrt_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> </div> </div> </div>where division by <div class="math_UNKNOWN"> <div class="msqrt_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div> corrects for the fact that the maximum correlation between P and GEBV is equal to <div class="math_UNKNOWN"> <div class="msqrt_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mn_UNKNOWN">
2</div> </div> </div> </div>, which is achieved if the accuracy is 1, that is when GEBV are equal to the true breeding values. To allow comparison of the accuracy obtained using different numbers of SNPs we used the <i>h</i> <sup>2</sup> obtained from the VanRaden model with all SNPs for scaling. For this comparison, we could hence equally well have used the correlation between phenotype and GEBV but choose to present the (identically scaled) accuracies to minimize potential confusion. <p /> <p>Since the standard error of the calculated accuracy depends on the size of the set of individuals excluded from the training population (Daetwyler et al. <a href="#evl3103-bib-0016">2013</a>), we followed a modified "leave‐one‐out" approach. In this approach each individual is in turn excluded from the training population, that is in practice its phenotype is set to missing and the standard GBLUP model is run. This means that this individual's GEBV is predicted based on all phenotypes except its own. To reduce computation time, we here did not exclude each individual separately from the training population but instead excluded 20 individuals at once, which reduced computation time by a factor 20. We randomly excluded 20 individuals from the training population, predicted their GEBVs and then selected the next 20 individuals (each individual was only excluded once) until all individuals were excluded once and their GEBVs predicted. When GEBVs for all individuals had been predicted, the GEBVs were correlated with the individuals’ phenotypes. 95% confidence intervals were obtained by bootstrapping. The 1000 bootstrap samples were obtained by randomly drawing—with replacement—2015 observations from the data. We also report the (approximate) standard errors of the accuracy and the root mean square error (RMSE) of the GEBVs. The standard error was based on the sampling variance of a correlation coefficient calculated as the square root of the sampling variance divided by the square root of the heritability. The RMSE was calculated as the square root of the mean squared difference between an individual's phenotype and its GEBV estimated including the individual's phenotype. The scripts that were used to run the validation analyses are available as Supplementary Material. </p> <p>Since our data are from a natural population, there is some relatedness structure in the sampled individuals. Out of the 2015 individuals included in the analysis more than half (1520) had no known relatives within the genotyped individuals. This may be partly due to parents not being captured but overall capture rates of breeding adult individuals are at least 90%. Based on the recorded pedigree information, there were 185 individuals related as full‐sibs, 79 individuals related as maternal half‐sibs and 80 individuals related as paternal half‐sibs in the pedigree. Overall mean pairwise pedigree‐based relatedness was 0.00033. </p> <p>To test the effect of the number of SNPs used to estimate relatedness we reran our analysis for subsets of 1000, 50,000, 100,000, 250,000, and 400,000 SNPs. For 100 replicates the corresponding number of SNPs was sampled randomly from the total number of SNPs, the GRM calculated using the VanRaden‐estimator, and GEBVs predicted using the approach described above for each random sample. That is, for the first replicate the phenotypes of the first 20 individuals were excluded and their GEBVs predicted, for the second replicate the phenotypes of the next 20 individuals were excluded and their GEBVs predicted and so forth. </p> </div> <div class="expectedaccuracy" title="sec"> <div class="title" tagx="title" title="title">
EXPECTED ACCURACY</div> <p>We also calculated the expected accuracy of the GEBVs, following Daetwyler et al. ( <a href="#evl3103-bib-0017">2010</a>): </p><div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
r</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
g</div> <div class="mo_UNKNOWN">
,</div> <div class="mover_UNKNOWN"> <div class="mi_UNKNOWN">
g</div> <div class="mo_UNKNOWN">
^</div> </div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="msqrt_UNKNOWN"> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
h</div> <div class="mn_UNKNOWN">
2</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mi_UNKNOWN">
e</div> </div> </div> </div> </div> <div class="mspace_UNKNOWN" /> </div> </div> </div>with <i>N</i> being size of the training population, <i>h</i> <sup>2</sup> heritability of the trait, and <i>M <sub>e</sub> </i> the number of independently segregating genome segments. We approximated <i>M <sub>e</sub> </i> using the equation of Goddard ( <a href="#evl3103-bib-0032">2009</a>): <div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mn_UNKNOWN">
2</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mi_UNKNOWN">
L</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
l</div> <div class="mi_UNKNOWN">
n</div> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
4</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
N</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mi_UNKNOWN">
L</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mspace_UNKNOWN" /> </div> </div> </div>with <i>N <sub>e</sub> </i> being effective population size and <i>L</i> genome length in Morgans. The total map length in the great tit study population is 2009.85 cM (van Oers et al. <a href="#evl3103-bib-0070">2014</a>). The effective population size estimated from pairwise sequential Markovian coalescent analysis is ∼5.7 × 10 <sup>5</sup> individuals (Laine et al. <a href="#evl3103-bib-0047">2016</a>). We also computed <i>M <sub>e</sub> </i> empirically from our data following Goddard et al. ( <a href="#evl3103-bib-0033">2011</a>): <div class="disp-formula" title="disp-formula"> <div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
M</div> <div class="mi_UNKNOWN">
e</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
v</div> <div class="mi_UNKNOWN">
a</div> <div class="mi_UNKNOWN">
r</div> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
G</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mspace_UNKNOWN" /> </div> </div> </div>with <i>var</i>( <i>G</i>) being the variance across all off‐diagonal relationships in the GRM. <p /> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>The pairwise relatedness estimated from the pedigree and the GRM corresponded well (Fig. <a href="#evl3103-fig-0001">1</a>). The mean relatedness estimated from all SNPs, using VanRaden ( <a href="#evl3103-bib-0071">2008</a>) and scaled according to the pedigree (see Methods) for individuals with a pedigree relatedness of 0.5 (full sibs and parent‐offspring pairs) and 0.25 (half‐sibs in our dataset) was 0.49 and 0.22, respectively. A number of individuals that is unrelated according to the pedigree show "genomic" relatedness of up to 0.5. We checked these cases of unexpectedly high relatedness against our extensive data base and concluded that these could potentially be "missing links" in the pedigree, i.e. these individuals having a close common ancestor that is not recorded in the pedigree. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Correspondence between pairwise relatedness estimates from the pedigree and SNPs. The genomic relatedness estimated using all SNPs and the vanRaden ( <a href="#evl3103-bib-0071">2008</a>) estimator (including pedigree information, see Methods for details) is plotted against the relatedness estimated from the pedigree information. The dotted line indicates the 1:1 relationship. </p>   </div> <p>The heritabilities of egg‐laying date calculated from the different GRMs were consistently, although not significantly, lower than the estimate based on pedigree information (Table <a href="#evl3103-tbl-0001">1</a>). At the same time, the permanent environment variance was considerably lower in the model using pedigree‐based relatedness. When the contribution of the GRM and pedigree to the relatedness estimates used in the analysis was varied, the results were very similar. The estimated heritability was highest when most weight was given to the GRM and decreased with decreasing weight of the GRM (Table <a href="#evl3103-tbl-0002">2</a>). </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Estimates (and SE) for the additive genetic ( <i>V <sub>A</sub> </i>), permanent environment ( <i>V <sub>PE</sub> </i>), and residual ( <i>V</i> <sub>res</sub>) variance component, heritability ( <i>h</i> <sup>2</sup>) of egg‐laying date and accuracies of GEBVs from animal models with different relatedness estimators (Yang: following Yang et al. ( <a href="#evl3103-bib-0079">2010</a>), VanRaden: following VanRaden ( <a href="#evl3103-bib-0071">2008</a>), Yang‐ped: as Yang but scaled for pedigree, VanRaden‐ped, as VanRaden but scaled for pedigree) </p>  <table> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <thead> <tr> <th>Relatedness estimator</th> <th> <i>V <sub>A</sub> </i> </th> <th> <i>h</i> <sup>2</sup> </th> <th> <i>V <sub>PE</sub> </i> </th> <th> <i>V</i> <sub>res</sub> </th> <th>Accuracy</th> <th>95% CI</th> <th>RMSE</th> </tr> </thead> <tbody> <tr> <td>Pedigree</td> <td>5.77 (1.72)</td> <td>0.24 (0.07)</td> <td>2.64 (1.71)</td> <td>15.7 (0.58)</td> <td>0.197 (0.045)</td> <td>0.115–0.281</td> <td>4.723</td> </tr> <tr> <td>VanRaden</td> <td>3.91 (1.31)</td> <td>0.16 (0.05)</td> <td>4.48 (1.38)</td> <td>15.6 (0.58)</td> <td>0.210 (0.055)</td> <td>0.105–0.304</td> <td>4.746</td> </tr> <tr> <td>Yang</td> <td>3.84 (1.31)</td> <td>0.16 (0.05)</td> <td>4.55 (1.38)</td> <td>15.6 (0.58)</td> <td>0.206 (0.055)</td> <td>0.097–0.312</td> <td>4.746</td> </tr> <tr> <td>VanRaden‐ped</td> <td>4.00 (1.33)</td> <td>0.17 (0.06)</td> <td>4.39 (1.39)</td> <td>15.6 (0.58)</td> <td>0.207 (0.054)</td> <td>0.104–0.304</td> <td>4.745</td> </tr> <tr> <td>Yang‐ped</td> <td>3.94 (1.33)</td> <td>0.16 (0.06)</td> <td>4.45 (1.39)</td> <td>15.6 (0.58)</td> <td>0.206 (0.055)</td> <td>0.103–0.303</td> <td>4.746</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>The 95% confidence intervals of the accuracy as well as their root mean square error (RMSE) are also given. </p> </div> </div> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd.</span>, </span> </div> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Estimates (and SE) for the additive genetic ( <i>V <sub>A</sub> </i>), permanent environment ( <i>V <sub>PE</sub> </i>), and residual ( <i>V</i> <sub>res</sub>) variance component, heritability ( <i>h</i> <sup>2</sup>) of egg‐laying date and accuracies of GEBVs from animal models weighting pedigree and marker‐based relatedness to a different degree </p>  <table> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <thead> <tr> <th>Alpha</th> <th> <i>V <sub>A</sub> </i> </th> <th> <i>h</i> <sup>2</sup> </th> <th> <i>V <sub>PE</sub> </i> </th> <th> <i>V</i> <sub>res</sub> </th> <th>Accuracy</th> <th>95% CI</th> <th>RMSE</th> </tr> </thead> <tbody> <tr> <td>0.05</td> <td>5.87 (1.72)</td> <td>0.24 (0.07)</td> <td>2.55 (1.71)</td> <td>15.6 (0.58)</td> <td>0.199 (0.045)</td> <td>0.117–0.281</td> <td>4.723</td> </tr> <tr> <td>0.50</td> <td>5.53 (1.62)</td> <td>0.23 (0.207)</td> <td>2.87 (1.62)</td> <td>15.6 (0.58)</td> <td>0.207 (0.046)</td> <td>0.120–0.286</td> <td>4.731</td> </tr> <tr> <td>0.80</td> <td>4.64 (1.45)</td> <td>0.19 (0.06)</td> <td>3.75 (1.49)</td> <td>15.6 (0.58)</td> <td>0.209 (0.051)</td> <td>0.119–0.300</td> <td>4.739</td> </tr> <tr> <td>0.95</td> <td>4.16 (1.36)</td> <td>0.17 (0.06)</td> <td>4.23 (1.42)</td> <td>15.6 (0.58)</td> <td>0.209 (0.053)</td> <td>0.109–0.312</td> <td>4.744</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>The 95% confidence intervals of the accuracy as well as their root means square error (RMSE) are also given. A high value for alpha means a high weight for marker‐based relatedness and vice versa. </p> </div> </div> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">John Wiley &amp;amp; Sons, Ltd.</span>, </span> </div> </div> <p>The accuracies of (pedigree‐based) EBVs (Table <a href="#evl3103-tbl-0001">1</a>) or of GEBVs with a strong contribution of the pedigree relative to the GRM (Table <a href="#evl3103-tbl-0002">2</a>) were lower, but not substantially so, than accuracies based solely on the GRM. Which relatedness estimator, VanRaden ( <a href="#evl3103-bib-0071">2008</a>) or Yang et al. (Yang et al. <a href="#evl3103-bib-0079">2010</a>), was used and whether estimates were scaled by the pedigree had little effect on the accuracy of GEBVs (Fig. <a href="#evl3103-fig-0002">2</a>, Table <a href="#evl3103-tbl-0001">1</a>). The root mean square errors (RMSEs) gave a slightly different picture with RMSEs being slightly lower for pedigree‐based analyses (Tables <a href="#evl3103-tbl-0001">1</a> and <a href="#evl3103-tbl-0002">2</a>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Estimated accuracies (and 95% confidence interval) for EBVs and GEBVs of egg‐laying date for four different estimation methods (ped: pedigree only, vR: vanRaden ( <a href="#evl3103-bib-0071">2008</a>), Y: Yang et al. ( <a href="#evl3103-bib-0079">2010</a>), vR‐ped: vanRaden ( <a href="#evl3103-bib-0071">2008</a>) including the pedigree in the estimation, Y‐ped: Yang et al. ( <a href="#evl3103-bib-0079">2010</a>) including the pedigree in the estimation). See Methods for details on estimation procedure. </p>   </div> <p>The accuracy increased with the number of SNPs used to calculate the GRM (Fig. <a href="#evl3103-fig-0003">3</a>). Using only 1000 SNPs led to a very low accuracy, also with CIs overlapping with zero. Using 50,000 SNPs led to an improved but still reduced accuracy. Increasing the number of SNPs to 100,000 improved the accuracy still but whether 250,000, 400,000 or all SNPs were used made very little difference. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Accuracy of GEBVs (and 95% confidence interval) as a function of the number of markers used to estimate genomic relatedness (using VanRaden 2008). </p>   </div> <p>The predicted accuracy based on a heritability of 0.17, an effective population size of 570,000 individuals and a genome size of 20.1 M was much smaller than the empirical accuracy, namely only 0.017, when <i>M <sub>e</sub> </i> was calculated from <i>N <sub>e</sub> </i>, following Goddard ( <a href="#evl3103-bib-0032">2009</a>) giving a value of 1,298,858. When <i>M <sub>e</sub> </i> was calculated empirically from the variation in the relatedness, following Goddard et al. ( <a href="#evl3103-bib-0033">2011</a>) giving a value of 7230, the predicted accuracy was 0.21 and thus very close to the empirical accuracy. </p> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Depending on the used relatedness estimator and whether estimates were scaled according to the pedigree the estimated accuracy of GEBVs for egg‐laying date varied between 0.197 and 0.210. This is lower than what is normally found in domesticated species. One study in dairy cattle on a variety of traits found accuracies that ranged from 0.63 to 0.83, with an average of 0.70 (VanRaden et al. <a href="#evl3103-bib-0072">2009</a>). An early review study on dairy cattle based on considerably smaller training populations of only 332 to 637 individuals still found accuracies of 0.60 on average (Hayes et al. <a href="#evl3103-bib-0037">2009b</a>). The average accuracy in a large number of studies on plants, mainly crop species, was ca. 0.60 (reviewed in Lin et al. <a href="#evl3103-bib-0049">2014</a>). It hence seems that the accuracies estimated here are smaller than the ones typically found in domesticated animal and plant species. This is maybe not too surprising as genomic prediction relies on the markers being in linkage disequilibrium with the loci determining the trait. Effective population sizes of cattle breeds can be as low as ∼100 (e.g., de Roos et al. <a href="#evl3103-bib-0019">2008</a>). Due to the larger effective population sizes of natural populations and the therefore higher number of independently segregating genome segments fewer causal loci will be "tagged" by the markers in natural populations resulting in lower expected accuracy of GEBVs (e.g., Goddard <a href="#evl3103-bib-0032">2009</a>; Hayes et al. <a href="#evl3103-bib-0038">2009c</a>; Daetwyler et al. <a href="#evl3103-bib-0017">2010</a>). Another factor determining the accuracy is the number of individuals in the training population, i.e. the number of individuals of which both phenotype and genotype are known. For example, increasing the size of the training population from 1151 to 3576 individuals increased the accuracy of the predicted GEBVs from 0.35 to 0.53 in Holstein dairy cattle (VanRaden et al. <a href="#evl3103-bib-0072">2009</a>). Our training population of about 2000 individuals was smaller than typical datasets from animal and plant breeding, especially recent studies that have been based on &amp;gt;20,000 genotyped individuals (e.g., Lin et al. <a href="#evl3103-bib-0049">2014</a>; Schöpke and Swalve <a href="#evl3103-bib-0063">2016</a>). The accuracy of GEBVs also depends on the trait's heritability (e.g., Goddard et al. <a href="#evl3103-bib-0033">2011</a>; Su et al. <a href="#evl3103-bib-0067">2012</a>; Brito et al. <a href="#evl3103-bib-0009">2017</a>), which was 0.24 when estimated from the pedigree and 0.17 when estimated using the markers and hence roughly comparable to the traits analyzed in, for example, VanRaden et al. ( <a href="#evl3103-bib-0072">2009</a>). </p> <p>The predicted accuracy of the GEBVs for egg‐laying date was lower than the estimated accuracy and also very low in absolute terms, about 0.01, when estimating <i>M <sub>e</sub> </i> from the effective population size in Laine et al. ( <a href="#evl3103-bib-0047">2016</a>). The factor driving this low predicted accuracy was the very large effective population size in great tits, more than half a million (Laine et al. <a href="#evl3103-bib-0047">2016</a>). While there is very little genetic differentiation between European great tit populations (Kvist et al. <a href="#evl3103-bib-0046">2003</a>; Laine et al. <a href="#evl3103-bib-0047">2016</a>), this very large effective population size may still be an overestimate. When we calculated <i>M <sub>e</sub> </i> from the variation in genomic relatedness, the predicted and estimated accuracies corresponded much better (0.21 vs. 0.18). This illustrates that estimating <i>M <sub>e</sub> </i> from <i>N <sub>e</sub> </i> can be challenging (Brard and Ricard <a href="#evl3103-bib-0008">2015</a>). </p> <p>Goddard et al. ( <a href="#evl3103-bib-0033">2011</a>) showed that the number of markers also affects the accuracy of GEBVs because if the number of markers is too small the true relatedness at the causal loci will be estimated too imprecisely. The expected accuracy increased with the number of SNPs but started to plateau at around 3000 SNPs (Goddard et al. <a href="#evl3103-bib-0033">2011</a>). This is in line with earlier findings that increasing the number of SNPs from ∼10,000 to ∼38,000 had very little effect on estimated accuracies in Holstein dairy cattle (VanRaden et al. <a href="#evl3103-bib-0072">2009</a>). We used here more than ten times more markers, 502,951 SNPs, which seemed to have "compensated" for the larger effective population size and the hence increased number of independently segregating genome segments. We also could demonstrate the previously expected pattern that the accuracy plateaued with an increasing number of markers indicating that the number of markers used here was sufficient (Fig. <a href="#evl3103-fig-0003">3</a>). </p> <p>The accuracy of the estimated breeding values (EBVs) from a pedigree‐based animal model was very similar to the accuracies of GEBVs but the pedigree‐based heritability was higher (Table <a href="#evl3103-tbl-0001">1</a>). The effect that using the pedigree rather than the GRM led to higher heritabilities could also be seen when more weight was given to the pedigree matrix relative to the GRM in a combined, weighted relatedness matrix (Table <a href="#evl3103-tbl-0002">2</a>). One potential reason for the higher pedigree‐based estimate is environmentally caused similarity among relatives inflating estimates of <i>V <sub>A</sub> </i> and <i>h</i> <sup>2</sup> (van der Jeugd and McCleery <a href="#evl3103-bib-0069">2002</a>). Animal models have been thought to suitably account for this potential bias (e.g., Kruuk <a href="#evl3103-bib-0044">2004</a>; Wilson et al. <a href="#evl3103-bib-0077">2010</a>) but a simulation study showed that not all pedigrees, including the one of the Hoge Veluwe study population, may be informative enough to allow this (Gienapp et al. <a href="#evl3103-bib-0031">2013b</a>). The low relatedness information in the pedigree‐based analysis means that it is difficult for the model to disentangle the genetic and permanent environmental effects with too much of the permanent environment variance being absorbed into the additive genetic variance and the EBVs. Using genomic instead of pedigree relatedness has been shown to improve the ability of the model to disentangle variances of possibly confounded effects (Lee et al. <a href="#evl3103-bib-0048">2010</a>). Thus, the additional information contained in the GRM compared to the pedigree may have been able to remove more of the environmentally caused similarity among relatives, which would mean that the pedigree‐based heritability estimate would be inflated. However, to ultimately address this, cross‐fostering experiments (e.g., Kruuk and Hadfield <a href="#evl3103-bib-0045">2007</a>) or models directly fitting the spatial correlation would be necessary (e.g., Stopher et al. <a href="#evl3103-bib-0066">2012</a>). </p> <p>While GEBVs have an obvious use in animal and plant breeding, they also have the potential for useful applications in natural populations. Whether this will be possible and useful obviously depends on the accuracy of the estimated GEBVs, which will depend on species‐ and population‐specific parameters, which determine <i>N <sub>e</sub> </i> and <i>M <sub>e</sub> </i>, and the number of available markers. Whether these potential constraints prohibit the useful applications of GEBVs in certain species or populations, requires further investigation. GEBVs could, for example, be used to select individuals for assisted migration or release programs. Assisted migration aims to mitigate negative effects of climate change by transplanting suitable individuals to new locations that could only be slowly reached by natural dispersal, that is "assisting" their dispersal, which has already been used in forest tree management (Aitken and Whitlock <a href="#evl3103-bib-0003">2013</a>). In breeding programs for endangered species GEBVs could also play a role by helping to identify individuals that are well adapted to the, potentially altered, release environments (Griffiths and Pavajeau <a href="#evl3103-bib-0034">2008</a>). Currently, genotyping is more costly than measuring phenotypes but with dropping genotyping costs this may change, especially for phenotypes that are difficult to measure. Even though the variance in estimated (genomic) breeding values is downward biased in comparison to the additive genetic variance (e.g., Hadfield et al. <a href="#evl3103-bib-0035">2010</a>) it may still serve as a useful proxy. Genotyping individuals in newly studied populations and predicting their GEBVs could allow us to (roughly) predict the evolutionary potential of these populations without the need for, potentially very laborious, quantitative genetic studies. Potential limitations of GEBVs for such applications can, however, arise from Genotype‐by‐Environment interactions (G × E) or different genetic trait architectures in different populations. G × E is common in many traits (e.g., Pigliucci <a href="#evl3103-bib-0059">2001</a>; Wood and Brodie <a href="#evl3103-bib-0078">2016</a>), which would mean that the "ranking" of individual breeding values in different environments can differ. As a consequence of this, the accuracy of predicting GEBVs across environments can be substantially reduced. Predicting GEBVs across populations, while ignoring this in the model, assumes that the genetic architecture of the trait is identical, that is that the same loci segregate in both populations. Cross‐breed prediction of GEBVs in domesticated populations has been attempted but accuracies are much lower than typical within‐breed accuracies (Moghaddar et al. <a href="#evl3103-bib-0056">2014</a>; Hidalgo et al. <a href="#evl3103-bib-0039">2016</a>). </p> <p>High‐density marker panels have been available in animal and plant breeding and have been used in these fields for considerably longer than in evolutionary ecology to estimate relatedness from markers or even genomic selection (Gienapp et al. <a href="#evl3103-bib-0024">2017a</a>). Early approaches aimed at estimating relatedness in natural populations from markers failed due to the limited number of markers available at that time (Coltman <a href="#evl3103-bib-0014">2005</a>; Csilléry et al. <a href="#evl3103-bib-0015">2006</a>), which is again stressed by our results that about 100,000 SNPs were necessary to obtain reliable accuracies of GEBVs. More recently, studies in natural populations of mammals, birds, and plants successfully estimated additive genetic variances in a number of traits and were even able to predict GEBVs (Robinson et al. <a href="#evl3103-bib-0062">2013</a>; Stanton‐Geddes et al. <a href="#evl3103-bib-0065">2013</a>; Beaulieu et al. <a href="#evl3103-bib-0006">2014</a>; Bérénos et al. <a href="#evl3103-bib-0007">2014</a>). Consequently, genomic prediction and also marker‐based quantitative genetics could become possible in a wide range of species, which has the potential to widen our understanding of evolutionary dynamics in natural populations (Gienapp et al. <a href="#evl3103-bib-0024">2017a</a>). </p> </div> <div class="sec" title="sec"> <p>Associate Editor: A. Charmantier</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data">  <p>Supplementary Material</p>  <div class="media" title="media"> <a href="EVL3-3-142-s001.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGMENTS</div> <p>Funding for this project was provided by an ERC advanced grant (339092‐E‐Response) to M.E.V. Two anonymous reviewers provided constructive comments. </p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
AUTHOR CONTRIBUTIONS</div> <p>P.G., M.P.L.C., and M.E.V. designed the study and the analysis. M.E.V. and P.G. compiled the phenotypic data. V.N.L. prepared the genomic data. P.G. ran all analyses and wrote the manuscript. All coauthors commented on the manuscript and approved the final version. </p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
DATA ACCESSIBILITY</div> <p>All data will be deposited in a public archive (Dryad) after acceptance and before publication of the manuscript. </p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
LITERATURE CITED</div> <li tag="ref"> <a name="evl3103-bib-0001" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Abzhanov</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">W. P.</span> <span class="surname" tagx="surname" title="surname">Kuo</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C.</span> <span class="surname" tagx="surname" title="surname">Hartmann</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. R.</span> <span class="surname" tagx="surname" title="surname">Grant</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. R.</span> <span class="surname" tagx="surname" title="surname">Grant</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> <span class="surname" tagx="surname" title="surname">Tabin</span> </span>. <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">The calmodulin pathway and evolution of elongated beak morphology in Darwin's finches</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">442</span>: <span class="fpage" tagx="fpage" title="fpage">563</span>– <span class="lpage" tagx="lpage" title="lpage">567</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16885984">16885984</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0002" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Abzhanov</span>, <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M.</span> <span class="surname" tagx="surname" title="surname">Protas</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. R.</span> <span class="surname" tagx="surname" title="surname">Grant</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. R.</span> <span class="surname" tagx="surname" title="surname">Grant</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> <span class="surname" tagx="surname" title="surname">Tabin</span> </span>. <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Bmp4 and morphological variation of beaks in Darwin's finches</span>. <span class="source" tagx="source" title="source">Science</span> <span class="volume" tagx="volume" title="volume">305</span>: <span class="fpage" tagx="fpage" title="fpage">1462</span>– <span class="lpage" tagx="lpage" title="lpage">1465</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15353802">15353802</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0003" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Aitken</span>, <span class="given-names" tagx="given-names" title="given-names">S. N.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. C.</span> <span class="surname" tagx="surname" title="surname">Whitlock</span> </span>. <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Assisted gene flow to facilitate local adaptation to climate change</span>. <span class="source" tagx="source" title="source">Ann. Rev. Ecol. Evol. Syst.</span> <span class="volume" tagx="volume" title="volume">44</span>: <span class="fpage" tagx="fpage" title="fpage">367</span>– <span class="lpage" tagx="lpage" title="lpage">388</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0004" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Barson</span>, <span class="given-names" tagx="given-names" title="given-names">N. J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T.</span> <span class="surname" tagx="surname" title="surname">Aykanat</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K.</span> <span class="surname" tagx="surname" title="surname">Hindar</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M.</span> <span class="surname" tagx="surname" title="surname">Baranski</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">G. H.</span> <span class="surname" tagx="surname" title="surname">Bolstad</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P.</span> <span class="surname" tagx="surname" title="surname">Fiske</span> </span>, et al. <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Sex‐dependent dominance at a single locus maintains variation in age at maturity in salmon </span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">528</span>: <span class="fpage" tagx="fpage" title="fpage">405</span>– <span class="lpage" tagx="lpage" title="lpage">408</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26536110">26536110</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0005" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Barton</span>, <span class="given-names" tagx="given-names" title="given-names">N. H.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> <span class="surname" tagx="surname" title="surname">Etheridge</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A.</span> <span class="surname" tagx="surname" title="surname">Véber</span> </span>. <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">The infinitesimal model: definition, derivation, and implications</span>. <span class="source" tagx="source" title="source">Theor. Pop. Biol.</span> <span class="volume" tagx="volume" title="volume">118</span>: <span class="fpage" tagx="fpage" title="fpage">50</span>– <span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28709925">28709925</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0006" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Beaulieu</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T.</span> <span class="surname" tagx="surname" title="surname">Doerksen</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">S.</span> <span class="surname" tagx="surname" title="surname">Clément</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">MacKay</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">Bousquet</span> </span>. <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Accuracy of genomic selection models in a large population of open‐pollinated families in white spruce </span>. <span class="source" tagx="source" title="source">Heredity</span> <span class="volume" tagx="volume" title="volume">113</span>: <span class="fpage" tagx="fpage" title="fpage">343</span>– <span class="lpage" tagx="lpage" title="lpage">352</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24781808">24781808</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0007" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bérénos</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. A.</span> <span class="surname" tagx="surname" title="surname">Ellis</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> <span class="surname" tagx="surname" title="surname">Pilkington</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> <span class="surname" tagx="surname" title="surname">Pemberton</span> </span>. <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Estimating quantitative genetic parameters in wild populations: a comparison of pedigree and genomic approaches </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">23</span>: <span class="fpage" tagx="fpage" title="fpage">3434</span>– <span class="lpage" tagx="lpage" title="lpage">3451</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24917482">24917482</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0008" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Brard</span>, <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A.</span> <span class="surname" tagx="surname" title="surname">Ricard</span> </span>. <span class="year" tagx="year" title="year">2015</span> <span class="mixed-article-title" title="mixed-article-title">Is the use of formulae a reliable way to predict the accuracy of genomic selection</span>? <span class="source" tagx="source" title="source">J. Anim. Breed. Genet.</span> <span class="volume" tagx="volume" title="volume">132</span>: <span class="fpage" tagx="fpage" title="fpage">207</span>– <span class="lpage" tagx="lpage" title="lpage">217</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25377121">25377121</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0009" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Brito</span>, <span class="given-names" tagx="given-names" title="given-names">L. F.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> <span class="surname" tagx="surname" title="surname">Clarke</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> <span class="surname" tagx="surname" title="surname">McEwan</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">S. P.</span> <span class="surname" tagx="surname" title="surname">Miller</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">N. K.</span> <span class="surname" tagx="surname" title="surname">Pickering</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">W. E.</span> <span class="surname" tagx="surname" title="surname">Bain</span> </span>, et al. <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">Prediction of genomic breeding values for growth, carcass and meat quality traits in a multi‐breed sheep population using a HD SNP chip </span>. <span class="source" tagx="source" title="source">BMC Genet.</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">7</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28122512">28122512</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0010" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Bürger</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M.</span> <span class="surname" tagx="surname" title="surname">Lynch</span> </span>. <span class="year" tagx="year" title="year">1995</span> <span class="mixed-article-title" title="mixed-article-title">Evolution and extinction in a changing environment: a quantitative‐genetic analysis</span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">49</span>: <span class="fpage" tagx="fpage" title="fpage">151</span>– <span class="lpage" tagx="lpage" title="lpage">163</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28593664">28593664</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0011" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Calus</span>, <span class="given-names" tagx="given-names" title="given-names">M. P. L.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">Vandenplas</span> </span>. <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Calc_grm—a program to compute pedigree, genomic, and combined relationship matrices</span>. ABGC, Wageningen UR Livestock Research. </span> </li> <li tag="ref"> <a name="evl3103-bib-0012" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Charmantier</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span>, <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garant</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span>, and <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kruuk</span> <span class="given-names" tagx="given-names" title="given-names">L. E. B.</span> </span> </span>, eds. <span class="year" tagx="year" title="year">2014</span> <span class="source" tagx="source" title="source">Quantitative genetics in the wild</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford Univ. Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0013" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Colosimo</span>, <span class="given-names" tagx="given-names" title="given-names">P. F.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> <span class="surname" tagx="surname" title="surname">Peichel</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K.</span> <span class="surname" tagx="surname" title="surname">Nereng</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. K.</span> <span class="surname" tagx="surname" title="surname">Blackman</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. D.</span> <span class="surname" tagx="surname" title="surname">Shapiro</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D.</span> <span class="surname" tagx="surname" title="surname">Schluter</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> <span class="surname" tagx="surname" title="surname">Kingsley</span> </span>. <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">The genetic architecture of parallel armor plate reduction in threespine sticklebacks</span>. <span class="source" tagx="source" title="source">PLoS Biol</span>. <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">e109</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15069472">15069472</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0014" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Coltman</span>, <span class="given-names" tagx="given-names" title="given-names">D. W.</span> </span> <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Testing marker‐based estimates of heritability in the wild</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">2593</span>– <span class="lpage" tagx="lpage" title="lpage">2599</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15969737">15969737</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0015" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Csilléry</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T.</span> <span class="surname" tagx="surname" title="surname">Johnson</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D.</span> <span class="surname" tagx="surname" title="surname">Beraldi</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T.</span> <span class="surname" tagx="surname" title="surname">Clutton‐Brock</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D. W.</span> <span class="surname" tagx="surname" title="surname">Coltman</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B.</span> <span class="surname" tagx="surname" title="surname">Hansson</span> </span>, et al. <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Performance of marker‐based relatedness estimators in natural populations of outbred vertebrates </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">173</span>: <span class="fpage" tagx="fpage" title="fpage">2091</span>– <span class="lpage" tagx="lpage" title="lpage">2101</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16783017">16783017</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0016" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Daetwyler</span>, <span class="given-names" tagx="given-names" title="given-names">H. D.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. P. L.</span> <span class="surname" tagx="surname" title="surname">Calus</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R.</span> <span class="surname" tagx="surname" title="surname">Pong‐Wong</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">G.</span> <span class="surname" tagx="surname" title="surname">de los Campos</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> <span class="surname" tagx="surname" title="surname">Hickey</span> </span>. <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">193</span>: <span class="fpage" tagx="fpage" title="fpage">347</span>– <span class="lpage" tagx="lpage" title="lpage">365</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23222650">23222650</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0017" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Daetwyler</span>, <span class="given-names" tagx="given-names" title="given-names">H. D.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R.</span> <span class="surname" tagx="surname" title="surname">Pong‐Wong</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B.</span> <span class="surname" tagx="surname" title="surname">Villanueva</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> <span class="surname" tagx="surname" title="surname">Wooliams</span> </span>. <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">The impact of genetic architecture on genome‐wide evaluation methods</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">185</span>: <span class="fpage" tagx="fpage" title="fpage">1021</span>– <span class="lpage" tagx="lpage" title="lpage">1031</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20407128">20407128</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0018" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">de los Campos</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. H.</span> <span class="surname" tagx="surname" title="surname">Hickey</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R.</span> <span class="surname" tagx="surname" title="surname">Pong‐Wong</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">H. D.</span> <span class="surname" tagx="surname" title="surname">Daetwyler</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. P. L.</span> <span class="surname" tagx="surname" title="surname">Calus</span> </span>. <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Whole‐genome regression and prediction methods applied to plant and animal breeding</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">193</span>: <span class="fpage" tagx="fpage" title="fpage">327</span>– <span class="lpage" tagx="lpage" title="lpage">345</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22745228">22745228</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0019" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">de Roos</span>, <span class="given-names" tagx="given-names" title="given-names">A. P. W.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. J.</span> <span class="surname" tagx="surname" title="surname">Hayes</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R. J.</span> <span class="surname" tagx="surname" title="surname">Spelman</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>. <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Linkage disequilibrium and persistence of phase in Holstein–Friesian, Jersey and Angus cattle </span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">179</span>: <span class="fpage" tagx="fpage" title="fpage">1503</span>– <span class="lpage" tagx="lpage" title="lpage">1512</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18622038">18622038</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0020" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gao</span>, <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. S.</span> <span class="surname" tagx="surname" title="surname">Lund</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">Y.</span> <span class="surname" tagx="surname" title="surname">Zhang</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">G.</span> <span class="surname" tagx="surname" title="surname">Su</span> </span>. <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Accuracy of genomic prediction using different models and response variables in the Nordic Red cattle population </span>. <span class="source" tagx="source" title="source">J. Anim. Breed. Genet.</span> <span class="volume" tagx="volume" title="volume">130</span>: <span class="fpage" tagx="fpage" title="fpage">333</span>– <span class="lpage" tagx="lpage" title="lpage">340</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24074170">24074170</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0021" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Garant</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">L. E. B.</span> <span class="surname" tagx="surname" title="surname">Kruuk</span> </span>. <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">How to use molecular marker data to measure evolutionary parameters in wild populations</span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">14</span>: <span class="fpage" tagx="fpage" title="fpage">1843</span>– <span class="lpage" tagx="lpage" title="lpage">1859</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15910310">15910310</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0022" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Garant</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">L. E. B.</span> <span class="surname" tagx="surname" title="surname">Kruuk</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R. H.</span> <span class="surname" tagx="surname" title="surname">McCleery</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> <span class="surname" tagx="surname" title="surname">Sheldon</span> </span>. <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Evolution in a changing environment: a case study with great tit edging mass</span>. <span class="source" tagx="source" title="source">Am. Nat.</span> <span class="volume" tagx="volume" title="volume">164</span>: <span class="fpage" tagx="fpage" title="fpage">E115</span>– <span class="lpage" tagx="lpage" title="lpage">E129</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15540146">15540146</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0023" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gianola</span>, <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Priors in whole‐genome regression: the Bayesian alphabet returns</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">194</span>: <span class="fpage" tagx="fpage" title="fpage">573</span>– <span class="lpage" tagx="lpage" title="lpage">596</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23636739">23636739</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0024" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">S.</span> <span class="surname" tagx="surname" title="surname">Fior</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">F.</span> <span class="surname" tagx="surname" title="surname">Guillaume</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> <span class="surname" tagx="surname" title="surname">Lasky</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">V. L.</span> <span class="surname" tagx="surname" title="surname">Sork</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K.</span> <span class="surname" tagx="surname" title="surname">Csilléry</span> </span>. <span class="year" tagx="year" title="year">2017a</span> <span class="mixed-article-title" title="mixed-article-title">Genomic quantitative genetics to study evolution in the wild</span>. <span class="source" tagx="source" title="source">Trends Ecol. Evol.</span> <span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">897</span>– <span class="lpage" tagx="lpage" title="lpage">908</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29050794">29050794</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0025" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">L.</span> <span class="surname" tagx="surname" title="surname">Hemerik</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Visser</span> </span>. <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">A new statistical tool to predict phenology under climate change scenarios</span>. <span class="source" tagx="source" title="source">Glob. Change Biol.</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">600</span>– <span class="lpage" tagx="lpage" title="lpage">606</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0026" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">V. N.</span> <span class="surname" tagx="surname" title="surname">Laine</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. C.</span> <span class="surname" tagx="surname" title="surname">Mateman</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K.</span> <span class="surname" tagx="surname" title="surname">van Oers</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Visser</span> </span>. <span class="year" tagx="year" title="year">2017b</span> <span class="mixed-article-title" title="mixed-article-title">Environment‐dependent genotype‐phenotype associations in avian breeding time</span>. <span class="source" tagx="source" title="source">Front. Genet.</span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">102</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28824697">28824697</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0027" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M.</span> <span class="surname" tagx="surname" title="surname">Lof</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T. E.</span> <span class="surname" tagx="surname" title="surname">Reed</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">McNamara</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">S.</span> <span class="surname" tagx="surname" title="surname">Verhulst</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Visser</span> </span>. <span class="year" tagx="year" title="year">2013a</span> <span class="mixed-article-title" title="mixed-article-title">Predicting demographically‐sustainable rates of adaptation: can great tit breeding time keep pace with climate change </span>? <span class="source" tagx="source" title="source">Phil. Trans. R Soc. B</span> <span class="volume" tagx="volume" title="volume">368</span>: <span class="fpage" tagx="fpage" title="fpage">20120289</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23209174">23209174</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0028" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">Merilä</span> </span>. <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Disentangling plastic and genetic changes in body mass of Siberian jays</span>. <span class="source" tagx="source" title="source">J. Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">27</span>: <span class="fpage" tagx="fpage" title="fpage">1849</span>– <span class="lpage" tagx="lpage" title="lpage">1858</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24930732">24930732</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0029" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">E.</span> <span class="surname" tagx="surname" title="surname">Postma</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Visser</span> </span>. <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Why breeding time has not responded to selection for earlier breeding in a songbird population </span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">60</span>: <span class="fpage" tagx="fpage" title="fpage">2381</span>– <span class="lpage" tagx="lpage" title="lpage">2388</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17236428">17236428</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0030" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T. E.</span> <span class="surname" tagx="surname" title="surname">Reed</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Visser</span> </span>. <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Why climate change will invariably lead to selection on phenology</span>. <span class="source" tagx="source" title="source">Proc. R Soc. B</span> <span class="volume" tagx="volume" title="volume">281</span>: <span class="fpage" tagx="fpage" title="fpage">20141611</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0031" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Gienapp</span>, <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> <span class="surname" tagx="surname" title="surname">van Noordwijk</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Visser</span> </span>. <span class="year" tagx="year" title="year">2013b</span> <span class="mixed-article-title" title="mixed-article-title">Genetic background, and not ontogenetic effects, affects avian seasonal timing of reproduction </span>. <span class="source" tagx="source" title="source">J. Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">26</span>: <span class="fpage" tagx="fpage" title="fpage">2739</span>– <span class="lpage" tagx="lpage" title="lpage">2745</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24118141">24118141</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0032" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Goddard</span>, <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span> <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Genomic selection: prediction of accuracy and maximisation of long term response</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">136</span>: <span class="fpage" tagx="fpage" title="fpage">245</span>– <span class="lpage" tagx="lpage" title="lpage">257</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0033" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Goddard</span>, <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. J.</span> <span class="surname" tagx="surname" title="surname">Hayes</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T. H. E.</span> <span class="surname" tagx="surname" title="surname">Meuwissen</span> </span>. <span class="year" tagx="year" title="year">2011</span> <span class="mixed-article-title" title="mixed-article-title">Using the genomic relationship matrix to predict the accuracy of genomic selection</span>. <span class="source" tagx="source" title="source">J. Anim. Breed. Genet.</span> <span class="volume" tagx="volume" title="volume">128</span>: <span class="fpage" tagx="fpage" title="fpage">409</span>– <span class="lpage" tagx="lpage" title="lpage">421</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22059574">22059574</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0034" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Griffiths</span>, <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">L.</span> <span class="surname" tagx="surname" title="surname">Pavajeau</span> </span>. <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Captive breeding, reintroduction, and the conservation of amphibians</span>. <span class="source" tagx="source" title="source">Conserv. Biol.</span> <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">852</span>– <span class="lpage" tagx="lpage" title="lpage">861</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18616746">18616746</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0035" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hadfield</span>, <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> <span class="surname" tagx="surname" title="surname">Wilson</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D.</span> <span class="surname" tagx="surname" title="surname">Garant</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> <span class="surname" tagx="surname" title="surname">Sheldon</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">L. E. B.</span> <span class="surname" tagx="surname" title="surname">Kruuk</span> </span>. <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">The misuse of BLUP in ecology and evolution</span>. <span class="source" tagx="source" title="source">Am. Nat.</span> <span class="volume" tagx="volume" title="volume">175</span>: <span class="fpage" tagx="fpage" title="fpage">116</span>– <span class="lpage" tagx="lpage" title="lpage">125</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19922262">19922262</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0036" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hayes</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> <span class="surname" tagx="surname" title="surname">Bowman</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. C.</span> <span class="surname" tagx="surname" title="surname">Chamberlain</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K.</span> <span class="surname" tagx="surname" title="surname">Verbyla</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>. <span class="year" tagx="year" title="year">2009a</span> <span class="mixed-article-title" title="mixed-article-title">Accuracy of genomic breeding values in multi‐breed dairy cattle populations</span>. <span class="source" tagx="source" title="source">Genet. Sel. Evol.</span> <span class="volume" tagx="volume" title="volume">41</span>: <span class="fpage" tagx="fpage" title="fpage">51</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19930712">19930712</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0037" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hayes</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> <span class="surname" tagx="surname" title="surname">Bowman</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> <span class="surname" tagx="surname" title="surname">Chamberlain</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>. <span class="year" tagx="year" title="year">2009b</span> <span class="mixed-article-title" title="mixed-article-title">Genomic selection in dairy cattle: progress and challenges</span>. <span class="source" tagx="source" title="source">J. Dairy Sci.</span> <span class="volume" tagx="volume" title="volume">92</span>: <span class="fpage" tagx="fpage" title="fpage">433</span>– <span class="lpage" tagx="lpage" title="lpage">443</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19164653">19164653</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0038" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hayes</span>, <span class="given-names" tagx="given-names" title="given-names">B. J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. M.</span> <span class="surname" tagx="surname" title="surname">Visscher</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>. <span class="year" tagx="year" title="year">2009c</span> <span class="mixed-article-title" title="mixed-article-title">Increased accuracy of artificial selection by using the realized relationship matrix</span>. <span class="source" tagx="source" title="source">Genet. Res.</span> <span class="volume" tagx="volume" title="volume">91</span>: <span class="fpage" tagx="fpage" title="fpage">47</span>– <span class="lpage" tagx="lpage" title="lpage">60</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0039" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hidalgo</span>, <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. W. M.</span> <span class="surname" tagx="surname" title="surname">Bastiaansen</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. S.</span> <span class="surname" tagx="surname" title="surname">Lopes</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. P. L.</span> <span class="surname" tagx="surname" title="surname">Calus</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> <span class="surname" tagx="surname" title="surname">de Koning</span> </span>. <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Accuracy of genomic prediction of purebreds for cross bred performance in pigs</span>. <span class="source" tagx="source" title="source">J. Anim. Breed. Genet.</span> <span class="volume" tagx="volume" title="volume">133</span>: <span class="fpage" tagx="fpage" title="fpage">443</span>– <span class="lpage" tagx="lpage" title="lpage">451</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27087113">27087113</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0040" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hill</span>, <span class="given-names" tagx="given-names" title="given-names">W. G.</span> </span> <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Quantitative genetics in the genomics era</span>. <span class="source" tagx="source" title="source">Curr. Genom.</span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">196</span>– <span class="lpage" tagx="lpage" title="lpage">206</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0041" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Hoerl</span>, <span class="given-names" tagx="given-names" title="given-names">A. E.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R. W.</span> <span class="surname" tagx="surname" title="surname">Kennard</span> </span>. <span class="year" tagx="year" title="year">1970</span> <span class="mixed-article-title" title="mixed-article-title">Ridge regression: biased estimation for nonorthogonal problems</span>. <span class="source" tagx="source" title="source">Technometrics</span> <span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">55</span>– <span class="lpage" tagx="lpage" title="lpage">67</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0042" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Jannink</span>, <span class="given-names" tagx="given-names" title="given-names">J.‐L.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> <span class="surname" tagx="surname" title="surname">Lorenz</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">H.</span> <span class="surname" tagx="surname" title="surname">Iwata</span> </span>. <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Genomic selection in plant breeding: from theory to practice</span>. <span class="source" tagx="source" title="source">Brief. Funct. Genom.</span> <span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">166</span>– <span class="lpage" tagx="lpage" title="lpage">177</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0043" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kim</span>, <span class="given-names" tagx="given-names" title="given-names">J.‐M.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. W.</span> <span class="surname" tagx="surname" title="surname">Santure</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">H. J.</span> <span class="surname" tagx="surname" title="surname">Barton</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. L.</span> <span class="surname" tagx="surname" title="surname">Quinn</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">E. F.</span> <span class="surname" tagx="surname" title="surname">Cole</span> </span>, <span class="collab'">collab: Great Tit HapMap Consortium</span> , et al. <span class="year" tagx="year" title="year">2018</span> <span class="mixed-article-title" title="mixed-article-title">A high‐density SNP chip for genotyping great tit ( <i>Parus major</i>) populations and its application to studying the genetic architecture of exploration behaviour </span>. <span class="source" tagx="source" title="source">Mol. Ecol. Res.</span> <span class="volume" tagx="volume" title="volume">18</span>: <span class="fpage" tagx="fpage" title="fpage">877</span>– <span class="lpage" tagx="lpage" title="lpage">891</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0044" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kruuk</span>, <span class="given-names" tagx="given-names" title="given-names">L. E. B.</span> </span> <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Estimating genetic parameters in natural populations using the ‘animal model’</span>. <span class="source" tagx="source" title="source">Phil. Trans. R Soc. Lond. B</span> <span class="volume" tagx="volume" title="volume">359</span>: <span class="fpage" tagx="fpage" title="fpage">873</span>– <span class="lpage" tagx="lpage" title="lpage">890</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15306404">15306404</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0045" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kruuk</span>, <span class="given-names" tagx="given-names" title="given-names">L. E. B.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> <span class="surname" tagx="surname" title="surname">Hadfield</span> </span>. <span class="year" tagx="year" title="year">2007</span> <span class="mixed-article-title" title="mixed-article-title">How to separate genetic and environmental causes of similarity between relatives</span>. <span class="source" tagx="source" title="source">J. Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">1890</span>– <span class="lpage" tagx="lpage" title="lpage">1903</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17714306">17714306</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0046" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Kvist</span>, <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">Martens</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">H.</span> <span class="surname" tagx="surname" title="surname">Higuchi</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> <span class="surname" tagx="surname" title="surname">Nazarenko</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">O. P.</span> <span class="surname" tagx="surname" title="surname">Valchuk</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M.</span> <span class="surname" tagx="surname" title="surname">Orell</span> </span>. <span class="year" tagx="year" title="year">2003</span> <span class="mixed-article-title" title="mixed-article-title">Evolution and genetic structure of the great tit ( <i>Parus major</i>) complex </span>. <span class="source" tagx="source" title="source">Proc. R Soc. Lond. B</span> <span class="volume" tagx="volume" title="volume">270</span>: <span class="fpage" tagx="fpage" title="fpage">1447</span>– <span class="lpage" tagx="lpage" title="lpage">1454</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0047" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Laine</span>, <span class="given-names" tagx="given-names" title="given-names">V. N.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T. I.</span> <span class="surname" tagx="surname" title="surname">Gossmann</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> <span class="surname" tagx="surname" title="surname">Schachtschneider</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> <span class="surname" tagx="surname" title="surname">Garroway</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">O.</span> <span class="surname" tagx="surname" title="surname">Madsen</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K. J. F.</span> <span class="surname" tagx="surname" title="surname">Verhoeven</span> </span>, et al. <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Evolutionary signals of selection on cognition from the great tit genome and methylome</span>. <span class="source" tagx="source" title="source">Nat. Comm</span>. <span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">10474</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0048" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lee</span>, <span class="given-names" tagx="given-names" title="given-names">S. H.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. M.</span> <span class="surname" tagx="surname" title="surname">Visscher</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. H. J.</span> <span class="surname" tagx="surname" title="surname">van der Werf</span> </span>. <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Using the realized relationship matrix to disentangle confounding factors for the estimation of genetic variance components of complex traits </span>. <span class="source" tagx="source" title="source">Genet. Sel. Evol.</span> <span class="volume" tagx="volume" title="volume">42</span>: <span class="fpage" tagx="fpage" title="fpage">22</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20546624">20546624</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0049" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lin</span>, <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. J.</span> <span class="surname" tagx="surname" title="surname">Hayes</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">H. D.</span> <span class="surname" tagx="surname" title="surname">Daetwyler</span> </span>. <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Genomic selection in crops, trees and forages: a review</span>. <span class="source" tagx="source" title="source">Crop Pasture Sci</span>. <span class="volume" tagx="volume" title="volume">65</span>: <span class="fpage" tagx="fpage" title="fpage">1177</span>– <span class="lpage" tagx="lpage" title="lpage">1191</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0050" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lynch</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R.</span> <span class="surname" tagx="surname" title="surname">Lande</span> </span>. <span class="year" tagx="year" title="year">1993</span> <span class="chapter-title" tagx="chapter-title" title="chapter-title">Evolution and extinction in response to environmental change</span> <i>in</i> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kareiva</span> <span class="given-names" tagx="given-names" title="given-names">P. M.</span> </span> </span>, <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kingsolver</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> </span>, and <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huey</span> <span class="given-names" tagx="given-names" title="given-names">R. B.</span> </span> </span>, eds. <span class="source" tagx="source" title="source">Biotic interactions and global change</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer Ass.</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sunderland, Massachusetts</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0051" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Lynch</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B.</span> <span class="surname" tagx="surname" title="surname">Walsh</span> </span>. <span class="year" tagx="year" title="year">1998</span> <span class="source" tagx="source" title="source">Genetics and analysis of quantitative traits</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sunderland, Massachusetts</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0052" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Manolio</span>, <span class="given-names" tagx="given-names" title="given-names">T. A.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">F. S.</span> <span class="surname" tagx="surname" title="surname">Collins</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">N. J.</span> <span class="surname" tagx="surname" title="surname">Cox</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D. B.</span> <span class="surname" tagx="surname" title="surname">Goldstein</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">L. A.</span> <span class="surname" tagx="surname" title="surname">Hindorff</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> <span class="surname" tagx="surname" title="surname">Hunter</span> </span>, et al. <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Finding the missing heritability of complex diseases</span>. <span class="source" tagx="source" title="source">Nature</span> <span class="volume" tagx="volume" title="volume">461</span>: <span class="fpage" tagx="fpage" title="fpage">747</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19812666">19812666</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0053" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">McCleery</span>, <span class="given-names" tagx="given-names" title="given-names">R. H.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> <span class="surname" tagx="surname" title="surname">Pettifor</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P.</span> <span class="surname" tagx="surname" title="surname">Armbruster</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K.</span> <span class="surname" tagx="surname" title="surname">Meyer</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> <span class="surname" tagx="surname" title="surname">Sheldon</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> <span class="surname" tagx="surname" title="surname">Perrins</span> </span>. <span class="year" tagx="year" title="year">2004</span> <span class="mixed-article-title" title="mixed-article-title">Components of variance underlying fitness in a natural population of the great tit <i>Parus major</i> </span>. <span class="source" tagx="source" title="source">Am. Nat</span>. <span class="volume" tagx="volume" title="volume">164</span>: <span class="fpage" tagx="fpage" title="fpage">E62</span>– <span class="lpage" tagx="lpage" title="lpage">E72</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15478083">15478083</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0054" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Meuwissen</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B.</span> <span class="surname" tagx="surname" title="surname">Hayes</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>. <span class="year" tagx="year" title="year">2001</span> <span class="mixed-article-title" title="mixed-article-title">Prediction of total genetic value using genome‐wide dense marker maps</span>. <span class="source" tagx="source" title="source">Genetics</span> <span class="volume" tagx="volume" title="volume">157</span>: <span class="fpage" tagx="fpage" title="fpage">1819</span>– <span class="lpage" tagx="lpage" title="lpage">1829</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11290733">11290733</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0055" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Meuwissen</span>, <span class="given-names" tagx="given-names" title="given-names">T.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B.</span> <span class="surname" tagx="surname" title="surname">Hayes</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>. <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Genomic selection: a paradigm shift in animal breeding</span>. <span class="source" tagx="source" title="source">Anim. Front.</span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="fpage" tagx="fpage" title="fpage">6</span>– <span class="lpage" tagx="lpage" title="lpage">14</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0056" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Moghaddar</span>, <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> <span class="surname" tagx="surname" title="surname">Swan</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. H. J.</span> <span class="surname" tagx="surname" title="surname">van der Werf</span> </span>. <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Comparing genomic prediction accuracy from purebred, crossbred and combined purebred and crossbred reference populations in sheep </span>. <span class="source" tagx="source" title="source">Genet. Sel. Evol.</span> <span class="volume" tagx="volume" title="volume">46</span>: <span class="fpage" tagx="fpage" title="fpage">58</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25927315">25927315</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0057" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Mrode</span>, <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span> <span class="year" tagx="year" title="year">2014</span> <span class="source" tagx="source" title="source">Linear models for the prediction of animal breeding values</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">CABI Publishing</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Wallingford</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0058" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Nelson</span>, <span class="given-names" tagx="given-names" title="given-names">R. M.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Petterson</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">Ö.</span> <span class="surname" tagx="surname" title="surname">Carlborg</span> </span>. <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">A century after Fisher: time for a new paradigm in quantitative genetics</span>. <span class="source" tagx="source" title="source">Trends Genet</span>. <span class="volume" tagx="volume" title="volume">29</span>: <span class="fpage" tagx="fpage" title="fpage">669</span>– <span class="lpage" tagx="lpage" title="lpage">676</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24161664">24161664</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0059" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Pigliucci</span>, <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="year" tagx="year" title="year">2001</span> <span class="source" tagx="source" title="source">Phenotypic plasticity</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Hopkins Univ. Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Baltimore</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0060" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Powell</span>, <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. M.</span> <span class="surname" tagx="surname" title="surname">Visscher</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> <span class="surname" tagx="surname" title="surname">Goddard</span> </span>. <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Reconciling the analysis of IBD and IBS in complex trait studies</span>. <span class="source" tagx="source" title="source">Nat. Rev. Genet.</span> <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">800</span>– <span class="lpage" tagx="lpage" title="lpage">805</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20877324">20877324</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0061" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Ritland</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Marker‐based method for inferences about quantitative inheritance in natural populations</span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">50</span>: <span class="fpage" tagx="fpage" title="fpage">1062</span>– <span class="lpage" tagx="lpage" title="lpage">1073</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28565279">28565279</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0062" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Robinson</span>, <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. W.</span> <span class="surname" tagx="surname" title="surname">Santure</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">I.</span> <span class="surname" tagx="surname" title="surname">DeCauwer</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> <span class="surname" tagx="surname" title="surname">Sheldon</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">Slate</span> </span>. <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Partitioning of genetic variation across the genome using multimarker methods in a wild bird population </span>. <span class="source" tagx="source" title="source">Mol. Ecol.</span> <span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">3963</span>– <span class="lpage" tagx="lpage" title="lpage">3980</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23848161">23848161</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0063" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Schöpke</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">H. H.</span> <span class="surname" tagx="surname" title="surname">Swalve</span> </span>. <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Opportunities and challenges for small populations of dairy cattle in the era of genomics</span>. <span class="source" tagx="source" title="source">Animal</span> <span class="volume" tagx="volume" title="volume">10</span>: <span class="fpage" tagx="fpage" title="fpage">1050</span>– <span class="lpage" tagx="lpage" title="lpage">1060</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26957010">26957010</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0064" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Seutin</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. N.</span> <span class="surname" tagx="surname" title="surname">White</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P. T.</span> <span class="surname" tagx="surname" title="surname">Boag</span> </span>. <span class="year" tagx="year" title="year">1991</span> <span class="mixed-article-title" title="mixed-article-title">Preservation of avian blood and tissue samples for DNA analyses</span>. <span class="source" tagx="source" title="source">Can. J. Zool.</span> <span class="volume" tagx="volume" title="volume">69</span>: <span class="fpage" tagx="fpage" title="fpage">82</span>– <span class="lpage" tagx="lpage" title="lpage">90</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0065" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stanton‐Geddes</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. B.</span> <span class="surname" tagx="surname" title="surname">Yoder</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R.</span> <span class="surname" tagx="surname" title="surname">Briskine</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">N. D.</span> <span class="surname" tagx="surname" title="surname">Young</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P.</span> <span class="surname" tagx="surname" title="surname">Tiffin</span> </span>. <span class="year" tagx="year" title="year">2013</span> <span class="mixed-article-title" title="mixed-article-title">Estimating heritability using genomic data</span>. <span class="source" tagx="source" title="source">Methods Ecol. Evol.</span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="fpage" tagx="fpage" title="fpage">1151</span>– <span class="lpage" tagx="lpage" title="lpage">1158</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0066" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Stopher</span>, <span class="given-names" tagx="given-names" title="given-names">K. V.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> <span class="surname" tagx="surname" title="surname">Walling</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A.</span> <span class="surname" tagx="surname" title="surname">Morris</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">F. E.</span> <span class="surname" tagx="surname" title="surname">Guinness</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T. H.</span> <span class="surname" tagx="surname" title="surname">Clutton‐Brock</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> <span class="surname" tagx="surname" title="surname">Pemberton</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D. H.</span> <span class="surname" tagx="surname" title="surname">Nussey</span> </span>. <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Shared spatial effects on quantitative genetic parameters: accounting for spatial autocorrelation and home range overlap reduces estimates of heritability in wild red deer </span>. <span class="source" tagx="source" title="source">Evolution</span> <span class="volume" tagx="volume" title="volume">66</span>: <span class="fpage" tagx="fpage" title="fpage">2411</span>– <span class="lpage" tagx="lpage" title="lpage">2426</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22834741">22834741</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0067" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Su</span>, <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P.</span> <span class="surname" tagx="surname" title="surname">Madsen</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">U. S.</span> <span class="surname" tagx="surname" title="surname">Nielsen</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> <span class="surname" tagx="surname" title="surname">Mäntysaari</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">G. P.</span> <span class="surname" tagx="surname" title="surname">Aamand</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">O. F.</span> <span class="surname" tagx="surname" title="surname">Christensen</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. S.</span> <span class="surname" tagx="surname" title="surname">Lund</span> </span>. <span class="year" tagx="year" title="year">2012</span> <span class="mixed-article-title" title="mixed-article-title">Genomic prediction for Nordic Red Cattle using one‐step and selection index blending</span>. <span class="source" tagx="source" title="source">J. Dairy Sci.</span> <span class="volume" tagx="volume" title="volume">95</span>: <span class="fpage" tagx="fpage" title="fpage">909</span>– <span class="lpage" tagx="lpage" title="lpage">917</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22281355">22281355</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0068" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Tibshirani</span>, <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="year" tagx="year" title="year">1996</span> <span class="mixed-article-title" title="mixed-article-title">Regression shrinkage and selection via the lasso</span>. <span class="source" tagx="source" title="source">J. R. Stat. Soc. B</span> <span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">267</span>– <span class="lpage" tagx="lpage" title="lpage">288</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0069" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">van der Jeugd</span>, <span class="given-names" tagx="given-names" title="given-names">H. P.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R.</span> <span class="surname" tagx="surname" title="surname">McCleery</span> </span>. <span class="year" tagx="year" title="year">2002</span> <span class="mixed-article-title" title="mixed-article-title">Effects of spatial autocorrelation, natal philopatry and phenotypic plasticity on the heritability of laying date </span>. <span class="source" tagx="source" title="source">J. Evol. Biol.</span> <span class="volume" tagx="volume" title="volume">15</span>: <span class="fpage" tagx="fpage" title="fpage">380</span>– <span class="lpage" tagx="lpage" title="lpage">387</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0070" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">van Oers</span>, <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. W.</span> <span class="surname" tagx="surname" title="surname">Santure</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">I.</span> <span class="surname" tagx="surname" title="surname">De Cauwer</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">N. E. M.</span> <span class="surname" tagx="surname" title="surname">van Bers</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R. P. M. A.</span> <span class="surname" tagx="surname" title="surname">Crooijmans</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. C.</span> <span class="surname" tagx="surname" title="surname">Sheldon</span> </span>, et al. <span class="year" tagx="year" title="year">2014</span> <span class="mixed-article-title" title="mixed-article-title">Replicated high‐density genetic maps of two great tit populations reveal fine‐scale genomic departures from sex‐equal recombination rates </span>. <span class="source" tagx="source" title="source">Heredity</span> <span class="volume" tagx="volume" title="volume">112</span>: <span class="fpage" tagx="fpage" title="fpage">307</span>– <span class="lpage" tagx="lpage" title="lpage">316</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24149651">24149651</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0071" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">VanRaden</span>, <span class="given-names" tagx="given-names" title="given-names">P. M.</span> </span> <span class="year" tagx="year" title="year">2008</span> <span class="mixed-article-title" title="mixed-article-title">Efficient methods to compute genomic predictions</span>. <span class="source" tagx="source" title="source">J. Dairy Sci.</span> <span class="volume" tagx="volume" title="volume">91</span>: <span class="fpage" tagx="fpage" title="fpage">4414</span>– <span class="lpage" tagx="lpage" title="lpage">4423</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18946147">18946147</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0072" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">VanRaden</span>, <span class="given-names" tagx="given-names" title="given-names">P. M.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. P.</span> <span class="surname" tagx="surname" title="surname">van Tassell</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">G. R.</span> <span class="surname" tagx="surname" title="surname">Wiggans</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">T. S.</span> <span class="surname" tagx="surname" title="surname">Sonstegard</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">R. D.</span> <span class="surname" tagx="surname" title="surname">Schnabel</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> <span class="surname" tagx="surname" title="surname">Taylor</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">F. S.</span> <span class="surname" tagx="surname" title="surname">Schenkel</span> </span>. <span class="year" tagx="year" title="year">2009</span> <span class="mixed-article-title" title="mixed-article-title">Reliability of genomic predictions for North American Holstein bulls</span>. <span class="source" tagx="source" title="source">J. Dairy Sci.</span> <span class="volume" tagx="volume" title="volume">92</span>: <span class="fpage" tagx="fpage" title="fpage">16</span>– <span class="lpage" tagx="lpage" title="lpage">24</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19109259">19109259</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0073" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Visscher</span>, <span class="given-names" tagx="given-names" title="given-names">P. M.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">S. E.</span> <span class="surname" tagx="surname" title="surname">Medland</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. A. R.</span> <span class="surname" tagx="surname" title="surname">Ferreira</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">K. I.</span> <span class="surname" tagx="surname" title="surname">Morley</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">G.</span> <span class="surname" tagx="surname" title="surname">Zhu</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. K.</span> <span class="surname" tagx="surname" title="surname">Cornes</span> </span>, et al. <span class="year" tagx="year" title="year">2006</span> <span class="mixed-article-title" title="mixed-article-title">Assumption‐free estimation of heritability from genome‐wide identity‐by‐descent sharing between full siblings </span>. <span class="source" tagx="source" title="source">PLoS Genet</span>. <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">e41</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16565746">16565746</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0074" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Visscher</span>, <span class="given-names" tagx="given-names" title="given-names">P. M.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">N. R.</span> <span class="surname" tagx="surname" title="surname">Wray</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">Q.</span> <span class="surname" tagx="surname" title="surname">Zhang</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">P.</span> <span class="surname" tagx="surname" title="surname">Sklar</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. I.</span> <span class="surname" tagx="surname" title="surname">McCarthy</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> <span class="surname" tagx="surname" title="surname">Brown</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J.</span> <span class="surname" tagx="surname" title="surname">Yang</span> </span>. <span class="year" tagx="year" title="year">2017</span> <span class="mixed-article-title" title="mixed-article-title">10 years of GWAS discovery: biology, function, and translation</span>. <span class="source" tagx="source" title="source">Am. J. Hum. Genet.</span> <span class="volume" tagx="volume" title="volume">101</span>: <span class="fpage" tagx="fpage" title="fpage">5</span>– <span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28686856">28686856</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0075" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Visser</span>, <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C.</span> <span class="surname" tagx="surname" title="surname">Both</span> </span>. <span class="year" tagx="year" title="year">2005</span> <span class="mixed-article-title" title="mixed-article-title">Shifts in phenology due to global climate change: the need for a yardstick</span>. <span class="source" tagx="source" title="source">Proc. R Soc. Lond. B</span> <span class="volume" tagx="volume" title="volume">272</span>: <span class="fpage" tagx="fpage" title="fpage">2561</span>– <span class="lpage" tagx="lpage" title="lpage">2569</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0076" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Visser</span>, <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. J.</span> <span class="surname" tagx="surname" title="surname">van Noordwijk</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> <span class="surname" tagx="surname" title="surname">Tinbergen</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> <span class="surname" tagx="surname" title="surname">Lessells</span> </span>. <span class="year" tagx="year" title="year">1998</span> <span class="mixed-article-title" title="mixed-article-title">Warmer springs lead to mistimed reproduction in great tits ( <i>Parus major</i>) </span>. <span class="source" tagx="source" title="source">Proc. R. Soc. Lond. B</span> <span class="volume" tagx="volume" title="volume">265</span>: <span class="fpage" tagx="fpage" title="fpage">1867</span>– <span class="lpage" tagx="lpage" title="lpage">1870</span>. </span> </li> <li tag="ref"> <a name="evl3103-bib-0077" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wilson</span>, <span class="given-names" tagx="given-names" title="given-names">A. J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D.</span> <span class="surname" tagx="surname" title="surname">Réale</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. N.</span> <span class="surname" tagx="surname" title="surname">Clements</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> <span class="surname" tagx="surname" title="surname">Morrissey</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">E.</span> <span class="surname" tagx="surname" title="surname">Postma</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> <span class="surname" tagx="surname" title="surname">Walling</span> </span>, et al. <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">An ecologist's guide to the animal model</span>. <span class="source" tagx="source" title="source">J. Anim. Ecol.</span> <span class="volume" tagx="volume" title="volume">79</span>: <span class="fpage" tagx="fpage" title="fpage">13</span>– <span class="lpage" tagx="lpage" title="lpage">26</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20409158">20409158</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0078" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Wood</span>, <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, and <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">E. D. I.</span> <span class="surname" tagx="surname" title="surname">Brodie</span> </span>. <span class="year" tagx="year" title="year">2016</span> <span class="mixed-article-title" title="mixed-article-title">Evolutionary response when selection and genetic variation covary across environments</span>. <span class="source" tagx="source" title="source">Ecol. Lett.</span> <span class="volume" tagx="volume" title="volume">19</span>: <span class="fpage" tagx="fpage" title="fpage">1189</span>– <span class="lpage" tagx="lpage" title="lpage">1200</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27531600">27531600</a> </span> </span> </li> <li tag="ref"> <a name="evl3103-bib-0079" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="string-name" title="string-name"> <span class="surname" tagx="surname" title="surname">Yang</span>, <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B.</span> <span class="surname" tagx="surname" title="surname">Benyamin</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">B. P.</span> <span class="surname" tagx="surname" title="surname">McEvoy</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">S.</span> <span class="surname" tagx="surname" title="surname">Gordon</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> <span class="surname" tagx="surname" title="surname">Henders</span> </span>, <span class="string-name" title="string-name"> <span class="given-names" tagx="given-names" title="given-names">D. R.</span> <span class="surname" tagx="surname" title="surname">Nyholt</span> </span>, et al. <span class="year" tagx="year" title="year">2010</span> <span class="mixed-article-title" title="mixed-article-title">Common SNPs explain a large proportion of the heritability for human height</span>. <span class="source" tagx="source" title="source">Nat. Genet</span>. <span class="volume" tagx="volume" title="volume">42</span>: <span class="fpage" tagx="fpage" title="fpage">565</span>– <span class="lpage" tagx="lpage" title="lpage">569</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20562875">20562875</a> </span> </span> </li> </ul> </div> </div>  </body></html>