<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Scientifica (Cairo)</span> <span class="iso-abbrev" title="iso-abbrev">Scientifica (Cairo)</span> <span class="publisher-id" title="publisher-id">SCIENTIFICA</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Scientifica</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2090-908X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Hindawi</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5884125">5884125</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1155/2018/9230143">10.1155/2018/9230143</a> </span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Analysis of Bacterial Diversity in Different Heavy Oil Wells of a Reservoir in South Oman with Alkaline pH </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-5339-2333</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shibulal</span> <span class="given-names" tagx="given-names" title="given-names">Biji</span> </span> <a href="#I1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-6335-5982</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Bahry</span> <span class="given-names" tagx="given-names" title="given-names">Saif N.</span> </span> <span class="email" tagx="email" title="email">snbahry@squ.edu.om</span> <a href="#I1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9580-6274</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Wahaibi</span> <span class="given-names" tagx="given-names" title="given-names">Yahya M.</span> </span> <a href="#I2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5128-7166</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elshafie</span> <span class="given-names" tagx="given-names" title="given-names">Abdulkadir E.</span> </span> <a href="#I1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3535-2586</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Bemani</span> <span class="given-names" tagx="given-names" title="given-names">Ali S.</span> </span> <a href="#I2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">Sanket J.</span> </span> <a href="#I1"> <sup>1</sup> </a> <a href="#I3"> <sup>3</sup> </a> </span> </div> <span class="citation_author_institution" id="I1">[], <sup>1</sup> </span> <span class="citation_author_institution" id="I2">[], <sup>2</sup> </span> <span class="citation_author_institution" id="I3">[], <sup>3</sup> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-other" title="other"> <p>Academic Editor: Joaquim Ruiz</p> </div> </div> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-3-3</span> </span> <span class="volume" tagx="volume" title="volume">2018</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">9230143</span> <span class="history" title="history"> <span class="received" title="received">received: 2017-10-29</span> <span class="accepted" title="accepted">accepted: 2018-2-15</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open access article distributed under the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The identification of potential hydrocarbon utilizing bacteria is an essential requirement in microbial enhanced oil recovery (MEOR). Molecular approaches like proteomic and genomic characterization of the isolates are replacing the traditional method of identification with systemic classification. Genotypic profiling of the isolates includes fingerprint or pattern-based technique and sequence-based technique. Understanding community structure and dynamics is essential for studying diversity profiles and is challenging in the case of microbial analysis. The present study aims to understand the bacterial community composition from different heavy oil contaminated soil samples collected from geographically related oil well areas in Oman and to identify spore-forming hydrocarbon utilizing cultivable bacteria. V4 region of 16S rDNA gene was the target for Ion PGM™. A total of 825081 raw sequences were obtained from Ion torrent from all the 10 soil samples. The species richness and evenness were found to be moderate in all the samples with four main phyla, Firmicutes, Bacteroidetes, Proteobacteria, and Actinobacteria, the most abundant being Firmicutes. <i> Bacillus </i>sp. ubiquitously dominated in all samples followed by <i> Paenibacillus</i>, which was followed by <i> Brevibacillus</i>, <i> Planococcus</i>, and <i> Flavobacterium</i>. Principal Coordinate Analysis (PCoA) and UPGMA dendrogram clustered the 10 soil samples into four main groups. Weighted UniFrac significance test determined that there was significant difference in the communities present in soil samples examined. It can be concluded that the microbial community was different in all the 10 soil samples with <i> Bacillus</i> and <i> Paenibacillus </i>sp. as predominating genus. The 16S rDNA sequencing of cultivable spore-forming bacteria identified the hydrocarbon utilizing bacteria as <i> Bacillus</i> and <i> Paenibacillus</i> sp. and the nucleotide sequences were submitted to NCBI GenBank under accession numbers KP119097–KP119115. <i> Bacillus</i> and <i> Paenibacillus</i> sp., which were relatively abundant in the oil fields, can be recommended to be chosen as candidates for hydrocarbon utilization study. </p> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Oil production has been experiencing decline in many parts of the world due to oilfield's maturity and an example of such includes the major oilfields in the North Sea [ <a href="#B1">1</a>]. Another major concern is the increasing energy demands due to global population growth and the difficulty in discovering new oilfields as an alternative to the exploited oil fields. Therefore, there is an urge to find out alternative technologies to increase oil recovery from existing oilfields around the world. It is a fact that fossil fuels will still remain the key source of energy, regardless of the gross investments in other energy sources such as biofuels, solar energy, and wind energy. This fact is highlighted by the current global energy production from fossil fuels which currently stand at about 80–90% with oil and gas representing about 60% [ <a href="#B2">2</a>]. During oil production, primary oil recovery can account for 30–40% oil productions, while additional 15–25% can be recovered by secondary methods such as water injection leaving behind about 35–55% of oil as residual oil in the reservoirs [ <a href="#B3">3</a>]. This residual oil is usually the target of many enhanced oil recovery technologies and it amounts to about 2–4 trillion barrels [ <a href="#B4">4</a>] or about 67% of the total oil reserves [ <a href="#B5">5</a>]. Residual oil recovery is at present necessitated for many oil companies and so there is a constant hunt for a cheap and efficient technology which will raise the global oil production as well as the productive life of many oilfields. The recovery of this residual oil is accomplished by enhanced oil recovery (EOR) or tertiary recovery methods which are used in oil industry to increase the production of crude oil. Indigenous or in situ microbial enhanced oil recovery (IMEOR) is one of the techniques in which the inhabitant microbes in the oil reservoirs were stimulated to enhance oil recovery [ <a href="#B6">6</a>]. IMEOR is reported as a cost-effective method [ <a href="#B7">7</a>]. The microbial community composition is influenced by oil reservoir geological conditions and other external factors like nutrient and water flooding, biosurfactant and biopolymer application, and steam injection [ <a href="#B8">8</a>]. </p> <p>For the advancement of a reliable MEOR protocol, understanding the microflora that exist in the oil reservoir is essential. Conventional culture-based techniques have been widely used for bacterial identification and enumeration in oil fields. Culture-based technique is selective and aids in the identification of only a few of the microbes. The problem is even worsened in extreme environmental conditions such as oil fields, where only microbes that can withstand strict circumstances will survive. Therefore, genomic analysis has become an important tool for understanding ecological biodiversity. It evades the need for laboratory cultivation and isolation of individual isolates. </p> <p>Complex environmental samples need some technique which can read multiple sequences in parallel. Next-Generation Sequencing (NGS) can obtain DNA sequences directly from environmental samples [ <a href="#B9">9</a>, <a href="#B10">10</a>]. In the study, the distribution of bacteria in steam-injected heavy oil wells (South Oman) with alkaline pH was characterized using high-throughput sequencing technique, Ion Torrent-Personal Genome Machine (PGM). The spore-forming bacteria in the samples were also identified using culture-based method. </p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <div class="2.1.samplecollectionandpreparation" title="sec"> <div class="title" tagx="title" title="title">
2.1. Sample Collection and Preparation</div> <p>The heavy crude oil contaminated subsurface soil was collected from different heavy oil wells in Southern Oman as described previously [ <a href="#B11">11</a>]. Total ten soil samples were collected aseptically in sterile sampling bags and stored at 4°C for further studies. The samples were kindly provided by a local oil company. </p> </div> <div class="2.2.genomicdnaisolation" title="sec"> <div class="title" tagx="title" title="title">
2.2. Genomic DNA Isolation</div> <p>The genomic DNA (gDNA) was isolated from cultivable spore-forming bacteria by boiling suspended soil samples in 10 ml distilled water at 90°C for 30 min to kill all the vegetative cells and enrichment of the bacteria in Bushnell-Haas media (BH media) containing 1% heavy crude oil. The flasks inoculated with the boiled sample supernatant were incubated at 40°C for 24 h and plated on fresh BH agar plates to obtain pure cultures. The gDNA from the isolates and the soil samples was isolated using PowerSoil® DNA Isolation Kit (Mo Bio Laboratories Inc.). The nucleic acid concentration and purity were measured by Thermo Fischer NanoDrop™ 2000/2000 c spectrophotometer. </p> </div> <div class="2.3.identificationofcultivablespore-formingisolatesusing16srdnasequencing" title="sec"> <div class="title" tagx="title" title="title">
2.3. Identification of Cultivable Spore-Forming Isolates Using 16S rDNA Sequencing</div> <p>Cultivable spore-forming bacterial isolates were identified by 16S rDNA sequencing using 27F and 1492R universal primers as described previously [ <a href="#B11">11</a>], where briefly the genomic DNA was extracted using PowerSoil DNA Isolation Kit (Mo Bio Laboratories Inc.), and the amplification was performed using T100™ thermal cycler (Bio-Rad, USA). The PCR products were purified using QIAquick PCR purification kit (QIAGen). The BigDye® Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems™) was used for de novo sequencing. The sequencing was done using 3130 XL Genetic Analyzer (Applied Biosystem, Hitachi) at Central Analytical and Applied Research Unit (CAARU), Sultan Qaboos University, and was submitted to NCBI GenBank USA. The dendrogram was constructed using maximum likelihood (ML) methods, respectively, using PHYLIP, the Phylogeny Inference Package program. The ML program uses a Hidden Markov Model (HMM) method of inferring different rates of evolution at different sites [ <a href="#B12">12</a>]. </p> </div> <div class="2.4.ngsanalysisofmicrobialcommunityinsoilsamples" title="sec"> <div class="title" tagx="title" title="title">
2.4. NGS Analysis of Microbial Community in Soil Samples</div> <p>The metagenomics analysis of all the 10 soil samples was done following the instruction manual of the Ion PGM System. The total gDNA extracted from the soil samples was amplified using 16S Primer Sets targeting hypervariable V4 region: 515F (5′-GTGCCAGCMGCCGCGGTAA-3′) and 806R (5′-GGACTACVSGGGTATCTAAT-3′). The PCR procedure was with an initial denaturation at 95°C for 10 min, 25 cycles of denaturation for 30 sec at 95°C, annealing at 58°C, extension at 72°C for 20 sec, followed by holding at 72°C for 7 min and a final hold at 4°C for ∞. The amplified product was purified using Agencourt® AMPure® XP Reagent and 70% ethanol. The DNA input for library preparation was calculated using Agilent® 2100 Bioanalyzer® and the library was prepared using Ion Plus Fragment Library Kit, following the user instructions. The pooled short amplicons were end-repaired using 5x End Repair Buffer and End Repair Enzyme and purified using Agencourt AMPure XP Reagent and 70% ethanol. Barcoded libraries were prepared using Ion Xpress™ Barcode Adapters 1–16 Kit. The adapters are ligated using DNA Ligase and the nicks were repaired using Nick Repair Polymerase. The DNA template for Ion PGM System was prepared using the Ion PGM Template OT2 400 Kit and the Ion OneTouch™ 2 System following the instructions in Ion PGM Template OT2 400 Kit User Guide. And the library was sequenced using the Ion Personal Genome Machine® (PGM™) System and the Ion PGM Sequencing 400 Kit. </p> </div> <div class="2.5.dataanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.5. Data Analysis</div> <p>Primary data analysis was performed with Torrent Suite™ Software v4.0 with automated secondary analysis using Ion Reporter™ Software v4.0. Further analysis was done using a variety of computer packages including Past3, XLstat, NCSS 2007,  "R" and NCSS 2010. Alpha diversity analysis was conducted using QIIME pipeline (version 1.8.0) (QIIME, 2016). Significance reported for any analysis is defined as <i>p</i> &amp;lt; 0.05. </p> <p>Short sequences &amp;lt; 200 bp were removed after depleting primers and barcodes; sequences with ambiguous base calls were also removed. Sequences with homopolymer runs exceeding 6 bp were removed, as the threshold used was 6 bp. Sequences were then denoised and chimeras were removed. Operational taxonomic units (OTUs) were formed at 97% similarity (3% divergence) using UCLUST after removing singleton sequences [ <a href="#B13">13</a>]. RDP classifier version 2.2 was used for taxonomic assignments within the GreenGenes taxonomy [ <a href="#B14">14</a>, <a href="#B15">15</a>]. The OTUs were rarefied (randomly subsampled) to 30000 sequences and the signified sequences were aligned to <i> 16S </i>reference sequences with PyNAST [ <a href="#B16">16</a>]. The aligned multiple sequences were used to create tree in FastTree [ <a href="#B17">17</a>]. </p> <p>Measures of <i>α</i>-diversity, the diversity within the community such as (a) Shannon-Wiener index, also termed the Shannon-Wiener index ( <i>H</i>′), which evaluates the relative abundance and refers to species richness and species evenness [ <a href="#B18">18</a>] and Simpson index ( <i>D</i>) [ <a href="#B19">19</a>] and (b) rarefaction, a method for species estimation based on the presence-absence data [ <a href="#B20">20</a>] and (c) Chao 1 [ <a href="#B21">21</a>] plots, based on the number of rare OTUs found in sample were measured using QIIME pipeline [ <a href="#B22">22</a>]. </p> <p>Shannon index is <i>H</i>′ = ∑⁡ <i>p</i> <sub> <i>i</i> </sub>ln⁡ <i>p</i> <sub> <i>i</i> </sub>, where <i>p</i> <sub> <i>i</i> </sub> is the proportion of individuals found in species <i>i</i>, <i>p</i> <sub> <i>i</i> </sub> = <i>n</i> <sub> <i>i</i> </sub>/ <i>N</i>, where <i>n</i> <sub> <i>i</i> </sub> is the number of individuals in species <i>i</i> and <i>N</i> is the total number of individuals in the community. </p> <p>Simpson <i>ʼ</i>s index = 1 − <i>D</i>, where <i>D</i> is the dominance. </p> <p> <i>D</i> = ∑ <i>p</i> <sub> <i>i</i> </sub> <sup>2</sup>, where <i>p</i> <sub> <i>i</i> </sub> is the proportion of individuals found in species <i>i</i>. </p> <p>SHE analysis was done which is based on the mathematical approach that diversity (Shannon diversity, <i>H</i>) is related directly to species richness ( <i>S</i>) and evenness ( <i>E</i>) of the distribution [ <a href="#B23">23</a>]. </p> <p> <i>H</i> = ln⁡( <i>S</i>)  +  ln⁡( <i>E</i>), where <i>H</i> is Shannon index; <i>S</i> is species richness; and <i>E</i> is evenness of distribution. </p> <p>From all OTUs, representative sequences were selected and using QIIME and RDP classifier, taxonomy was assigned for them. Only those phyla which appeared in 75% of the samples were selected. The relative abundance of the genera present in all 10 soil samples was plotted by a heatmap and graphically represented by interactive Krona charts. </p> <p> <i>β</i>-Diversity or community similarity analysis was done by determining the pairwise distances using the phylogenetic distance metric, UniFrac generated by QIIME [ <a href="#B24">24</a>]. Monte Carlo simulations (weighted UniFrac significance test) were done to test the significant difference in OTUs in the samples in the Unweighted Pair Group Method with Arithmetic Mean (UPGMA) tree generated by UniFrac analysis [ <a href="#B25">25</a>], followed by Bonferroni correction [ <a href="#B26">26</a>] to reduce type 1 error rate. </p> </div> </div> <div class="3.results" title="sec"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1.samplecollectionandpreparation" title="sec"> <div class="title" tagx="title" title="title">
3.1. Sample Collection and Preparation</div> <p>The soil moisture was determined to be 0.03–0.08 m <sup>3</sup>/m <sup>3</sup>, measured using EM50 Digital data logger, and pH was measured by Jenway 3505 pH meter as 8.5. The viscosity of the heavy crude oil from the well head, which contaminated the soil, was estimated as 4.57°API using RheolabQC Rotation Viscometer. The eTPH of the soil samples ranged within 3.2–4.8%. </p> </div> <div class="3.2.genomicdnaisolationandidentificationofcultivablespore-formingisolatesusing16srdnasequencing" title="sec"> <div class="title" tagx="title" title="title">
3.2. Genomic DNA Isolation and Identification of Cultivable Spore-Forming Isolates Using 16S rDNA Sequencing </div> <p>The gDNA from all the isolates ( <a href="#fig1">Figure 1(a)</a>) and the soil samples ( <a href="#fig1">Figure 1(b)</a>) was extracted using PowerSoil DNA Isolation Kit and was successfully amplified using universal primers for bacteria, 27F and 1492R. The amplified product was 1400 bp ( <a href="#fig2">Figure 2</a>). </p> <p>The purified PCR product was sequenced using 3130 XL Genetic Analyzer and the sequences were submitted to NCBI under the accession numbers KP119097–KP119115 ( <a href="#tab1">Table 1</a>). </p> <p>The sequences were aligned with the closely related species sequences from NCBI nucleotide blast. The branches with bootstrap values above 70% are reliable. The dendrogram was made using ML (maximum likelihood) method, which assumes every single site of the multiple sequence alignment independently with bootstrapping of 1000 replications. The ln likelihood for the cladogram was −6866.09722. The closer the value of likelihood to zero, the better is the dendrogram ( <a href="#fig3">Figure 3</a>). Among the listed isolates in <a href="#tab1">Table 1</a>, <i> Paenibacillus ehimensis</i> BS1 showed very good potential for applications in MEOR and heavy crude oil biodegradation [ <a href="#B11">11</a>]. </p> </div> <div class="3.3.ngsanalysisofmicrobialcommunityinsoilsamples" title="sec"> <div class="title" tagx="title" title="title">
3.3. NGS Analysis of Microbial Community in Soil Samples</div> <p>The gDNA was amplified using specific primers targeting the V4 region of the 16S rRNA bacterial gene using 515F (5′-GTGCCAGCMGCCGCGGTAA-3′) and 806R (5′-GGACTACVSGGGTATCTAAT-3′) for Ion Torrent PGM analysis. A total of 825081 raw sequences were obtained from Ion torrent from all the 10 soil samples. After quality filtering 758305 reads were obtained. Clustering with UCLUST provided 755423 OTUs. The average reads per sample was 75542. The OTUs were rarefied to 30000 sequences and jackknifed at 25000 sequences. </p> <p>The <i>α</i>-diversity measures of the samples were done in QIIME pipeline and Past3. Shannon index ( <i>H</i>′) and Simpson index ( <i>D</i>) were calculated for all the samples ( <a href="#tab2">Table 2</a>), which revealed that the species richness and relative abundance were present in all the samples, soil sample 2 being with the least diversity. </p> <p>Rarefaction curves with chao1 and Shannon diversity were plotted with the average number of OTUs at each interval against the size of the subsample [ <a href="#B27">27</a>] ( <a href="#fig4">Figure 4</a>). It was found that, for all the 10 soil samples, the curve reached a plateau at approximately 5000 sequences indicating that sequencing depth was sufficient to capture the full scope of microbial diversity. </p> <p>Taxonomic comparison of the OTUs identified four main phyla, Firmicutes, Bacteroidetes, Proteobacteria, and Actinobacteria, the most abundant being Firmicutes followed by Bacteroidetes ( <a href="#fig5">Figure 5</a>). Further comparison of genus belonging to these phyla showed that <i> Bacillus</i> was the most abundant genus followed by <i> Paenibacillus</i>, which was followed by <i> Brevibacillus</i>, <i> Planococcus</i>, <i> Flavobacterium</i>, and so forth ( <a href="#fig6">Figure 6</a>). </p> <p>The result was confirmed by heatmap analysis in which the highest relative abundance of <i> Bacillus</i>, <i> Paenibacillus</i>, and <i> Brevibacillus</i> was found in all the 10 soil samples and the soil samples with more similar microbial populations were mathematically clustered closer together. The genera (consortium) were used for clustering. Thus, the samples with more similar consortium of genera cluster closer together with the length of connecting lines (top of heatmap) ( <a href="#fig7">Figure 7</a>) related to the similarity; shorter lines between two samples indicate closely matched microbial consortium. The heatmap represents the relative percentages of each genus. The predominant genera are represented along the right <i>y</i>-axis. </p> <p>SHE analysis was performed to evaluate whether species proportion was similar in all of the 10 soil samples to assess the microbial diversity. The community structure was determined as a log-normal distribution, that is, a few species with high or low abundance and many with intermediate abundance [ <a href="#B28">28</a>] ( <a href="#fig8">Figure 8</a>). </p> <p>The resemblance between the bacterial communities, <i>β</i>-diversity, was measured using UniFrac analysis, which provided a tree-based ( <a href="#fig9">Figure 9(a)</a>) Principal Coordinate Analysis (PCoA) graph ( <a href="#fig9">Figure 9(b)</a>), which grouped the soil samples to 4 main clusters. The eigenvalues for PC1, PC2, and PC3 were 0.014, 0.002, and 0.001, respectively, and accounted for 74%, 12%, and 6% of the total variance. Both the two approaches revealed the same pattern. </p> <p>UniFrac <i>p</i> values were based on comparisons to 1,000 randomized trees. The <i>p</i> values are significant only if they were &amp;lt;0.05 ( <a href="#tab3">Table 3</a>). </p> </div> </div> <div class="4.discussion" title="sec"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>Ion PGM™ was used to delineate the bacterial community structure of 10 soil samples contaminated with heavy crude oil collected from near oil wells used in the study. The hypervariable V4 region of 16S rDNA sequence was the target region. Stringent quality sequence curation of a total of 825081 raw reads obtained resulted in 84% reduction of initial reads. There are reports stating 50–80% filtering of initial reads [ <a href="#B29">29</a>– <a href="#B31">31</a>]. </p> <p>The relative abundance of each phyla varied among the 10 soil samples, the predominant phyla observed to be Firmicutes, Bacteroidetes, Proteobacteria, and Actinobacteria, the most abundant being Firmicutes followed by Bacteroidetes. The possible reason for abundance of Firmicutes ( <i>Bacillus </i>sp.) could be their ability to form endospores to resist adverse conditions of the oil fields, as in desert habitats of Oman [ <a href="#B32">32</a>– <a href="#B35">35</a>]. Also, because of their metabolic and physiologic adaptability and ability to produce enzyme inhibitors and antibiotics, Firmicutes are considered to be better competitors in natural environment [ <a href="#B36">36</a>]. The moisture content of the soil samples was too low, about 0.03–0.08 m <sup>3</sup>/m <sup>3</sup>, which can be another possible reason. The heatmap analysis of genera showed that <i> Bacillus</i> had the highest relative abundance followed by <i> Paenibacillus</i> and <i> Brevibacillus</i> in all the 10 soil samples. </p> <p>Measures of <i>α</i>-diversity such as Shannon index and Simpson index showed that there was diversity of species OTUs within the community in all the 10 soil samples. Multiple rarefaction curves assembled from each sample's Shannon diversity index reached a plateau at approximately 5000 sequences suggesting that the sequencing depth was sufficient to capture the full scope of microbial diversity [ <a href="#B37">37</a>]. SHE analysis revealed the constant proportion of species in all 10 samples. </p> <p> <i>β</i>-Diversity measured using UniFrac analysis provided a tree-based PCoA (Principal Coordinate Analysis) graph. The two approaches revealed the same pattern of clustering; the Monte Carlo simulation test with Bonferroni corrections revealed that there was significant difference in the communities present in the soil samples examined [ <a href="#B24">24</a>]. </p> <p>The microbial community in soils is determined by the physicochemical parameters such as moisture content, temperature, salinity, and pH [ <a href="#B38">38</a>, <a href="#B39">39</a>]. The extreme temperature conditions along with low moisture content (0.03–0.08 m <sup>3</sup>/m <sup>3</sup>) and a slightly alkaline pH will have an impact on the diversity of bacterial community in the soil. Similar results were reported earlier for Tibetan plateau [ <a href="#B40">40</a>]. The presence of heavy crude oil can be another limiting factor for the microbial community. </p> <p>The identification of cultivable spore-forming isolates by 16S rDNA sequencing from the soil samples resulted in <i> Paenibacillus</i> and <i> Bacillus</i> sp. The relative abundance of <i> Bacillus</i> sp. in the microbial community in heavy crude oil sludge was reported in Saudi Arabia, Oman, and Nigeria. [ <a href="#B41">41</a>– <a href="#B43">43</a>]. <i> Bacillus</i> and <i> Paenibacillus</i> sp. which were relatively abundant in the oil fields can be recommended to be chosen as candidates for hydrocarbon utilization study. One of our isolates, <i> Paenibacillus ehimensis</i> BS1, showed maximum growth in presence of heavy oil and biotransformed it to lighter aliphatic and aromatic compounds demonstrating its potential in EOR and environmental bioremediation under aerobic and anaerobic reservoir conditions [ <a href="#B11">11</a>]. </p> </div> </div> <div class="back" title="back"> <div class="conflictsofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare that they have no conflicts of interest.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"> <a name="B1" /> <span class="label" tagx="label" title="label">1</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aleklett</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Höök</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jakobsson</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lardelli</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Snowden</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Söderbergh</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The Peak of the Oil Age - Analyzing the world oil production Reference Scenario in World Energy Outlook 2008 </span> <span class="source" tagx="source" title="source"> <i>Energy Policy</i> </span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">38</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="fpage" tagx="fpage" title="fpage">1398</span> <span class="lpage" tagx="lpage" title="lpage">1414</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.enpol.2009.11.021">10.1016/j.enpol.2009.11.021</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="label" tagx="label" title="label">2</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Graus</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roglieri</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jaworski</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alberio</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Worrell</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The promise of carbon capture and storage: Evaluating the capture-readiness of new EU fossil fuel power plants </span> <span class="source" tagx="source" title="source"> <i>Climate Policy</i> </span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">789</span> <span class="lpage" tagx="lpage" title="lpage">812</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.3763/cpol.2008.0615">10.3763/cpol.2008.0615</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="label" tagx="label" title="label">3</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cosse</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koch</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Basics of reservoir engineering</span> <span class="source" tagx="source" title="source"> <i>Pure and Applied Geophys</i> </span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">140</span> <span class="fpage" tagx="fpage" title="fpage">739</span> <span class="lpage" tagx="lpage" title="lpage">741</span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="label" tagx="label" title="label">4</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tharakan</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hallock</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cleveland</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jefferson</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Hydrocarbons and the evolution of human culture</span> <span class="source" tagx="source" title="source"> <i>Nature</i> </span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">426</span> <span class="issue" tagx="issue" title="issue">6964</span> <span class="fpage" tagx="fpage" title="fpage">318</span> <span class="lpage" tagx="lpage" title="lpage">322</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nature02130">10.1038/nature02130</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14628060">14628060</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="label" tagx="label" title="label">5</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stepp</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bertus</span> <span class="given-names" tagx="given-names" title="given-names">K. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burchfield</span> <span class="given-names" tagx="given-names" title="given-names">T. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dennis</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial-Enhanced Waterflooding Field Pilots</span> <span class="source" tagx="source" title="source"> <i>Developments in Petroleum Science</i> </span> <span class="year" tagx="year" title="year">1993</span> <span class="volume" tagx="volume" title="volume">39</span> <span class="issue" tagx="issue" title="issue">C</span> <span class="fpage" tagx="fpage" title="fpage">289</span> <span class="lpage" tagx="lpage" title="lpage">306</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0376-7361(09)70067-6">10.1016/S0376-7361(09)70067-6</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="label" tagx="label" title="label">6</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sunde</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beeder</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nilsen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Torsvik</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Aerobic microbial enhanced oil recovery for offshore use</span> <span class="source" tagx="source" title="source"> <i>SPE/DOE Enhanced Oil Recovery Symposium</i> </span> <span class="year" tagx="year" title="year">1992</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Tulsa, Oklahoma</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Society of Petroleum Engineers SPE-24204-MS</span> <span class="fpage" tagx="fpage" title="fpage">22</span> <span class="lpage" tagx="lpage" title="lpage">24</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2118/24204-MS">10.2118/24204-MS</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="label" tagx="label" title="label">7</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geetha</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banat</span> <span class="given-names" tagx="given-names" title="given-names">I. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosurfactants: Production and potential applications in microbial enhanced oil recovery (MEOR) </span> <span class="source" tagx="source" title="source"> <i>Biocatalysis and Agricultural Biotechnology</i> </span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">32</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bcab.2018.01.010">10.1016/j.bcab.2018.01.010</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="label" tagx="label" title="label">8</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Magot</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ollivier</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Magot</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Indigenous microbial communities in oil fields</span> <span class="source" tagx="source" title="source"> <i>Petroleum Microbiology</i> </span> <span class="year" tagx="year" title="year">2005</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington DC, USA</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">ASM Press</span> <span class="fpage" tagx="fpage" title="fpage">21</span> <span class="lpage" tagx="lpage" title="lpage">34</span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="label" tagx="label" title="label">9</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schuster</span> <span class="given-names" tagx="given-names" title="given-names">S. C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Next-generation sequencing transforms today's biology</span> <span class="source" tagx="source" title="source"> <i>Nature Methods</i> </span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">16</span> <span class="lpage" tagx="lpage" title="lpage">18</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth1156">10.1038/nmeth1156</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18165802">18165802</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="label" tagx="label" title="label">10</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sogin</span> <span class="given-names" tagx="given-names" title="given-names">M. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morrison</span> <span class="given-names" tagx="given-names" title="given-names">H. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huber</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial diversity in the deep sea and the underexplored ‘rare biosphere’</span> <span class="source" tagx="source" title="source"> <i>Proceedings of the National Acadamy of Sciences of the United States of America</i> </span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">103</span> <span class="issue" tagx="issue" title="issue">32</span> <span class="fpage" tagx="fpage" title="fpage">12115</span> <span class="lpage" tagx="lpage" title="lpage">12120</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0605127103">10.1073/pnas.0605127103</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B11" /> <span class="label" tagx="label" title="label">11</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shibulal</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Bahry</span> <span class="given-names" tagx="given-names" title="given-names">S. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Wahaibi</span> <span class="given-names" tagx="given-names" title="given-names">Y. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elshafie</span> <span class="given-names" tagx="given-names" title="given-names">A. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Bemani</span> <span class="given-names" tagx="given-names" title="given-names">A. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The potential of indigenous Paenibacillus ehimensis BS1 for recovering heavy crude oil by biotransformation to light fractions </span> <span class="source" tagx="source" title="source"> <i>PLoS ONE</i> </span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0171432">10.1371/journal.pone.0171432</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B12" /> <span class="label" tagx="label" title="label">12</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Felsenstein</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Churchill</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A Hidden Markov Model approach to variation among sites in rate of evolution</span> <span class="source" tagx="source" title="source"> <i>Molecular Biology and Evolution</i> </span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">13</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">93</span> <span class="lpage" tagx="lpage" title="lpage">104</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a025575">10.1093/oxfordjournals.molbev.a025575</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8583911">8583911</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="label" tagx="label" title="label">13</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Search and clustering orders of magnitude faster than BLAST</span> <span class="source" tagx="source" title="source"> <i>Bioinformatics</i> </span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="issue" tagx="issue" title="issue">19</span> <span class="fpage" tagx="fpage" title="fpage">2460</span> <span class="lpage" tagx="lpage" title="lpage">2461</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btq461">10.1093/bioinformatics/btq461</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691">20709691</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="label" tagx="label" title="label">14</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Desantis</span> <span class="given-names" tagx="given-names" title="given-names">T. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andersen</span> <span class="given-names" tagx="given-names" title="given-names">G. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Accurate taxonomy assignments from 16S rRNA sequences produced by highly parallel pyrosequencers </span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Research</i> </span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">36</span> <span class="issue" tagx="issue" title="issue">18, article no. e120</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkn491">10.1093/nar/gkn491</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="label" tagx="label" title="label">15</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schloss</span> <span class="given-names" tagx="given-names" title="given-names">P. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westcott</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Assessing and improving methods used in operational taxonomic unit-based approaches for 16S rRNA gene sequence analysis </span> <span class="source" tagx="source" title="source"> <i>Applied and Environmental Microbiology</i> </span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="fpage" tagx="fpage" title="fpage">3219</span> <span class="lpage" tagx="lpage" title="lpage">3226</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.02810-10">10.1128/AEM.02810-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21421784">21421784</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="label" tagx="label" title="label">16</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caporaso</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bittinger</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bushman</span> <span class="given-names" tagx="given-names" title="given-names">F. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeSantis</span> <span class="given-names" tagx="given-names" title="given-names">T. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andersen</span> <span class="given-names" tagx="given-names" title="given-names">G. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">PyNAST: a flexible tool for aligning sequences to a template alignment</span> <span class="source" tagx="source" title="source"> <i>Bioinformatics</i> </span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">26</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">266</span> <span class="lpage" tagx="lpage" title="lpage">267</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btp636">10.1093/bioinformatics/btp636</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19914921">19914921</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="label" tagx="label" title="label">17</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">M. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dehal</span> <span class="given-names" tagx="given-names" title="given-names">P. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arkin</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FastTree 2—approximately maximum-likelihood trees for large alignments</span> <span class="source" tagx="source" title="source"> <i>PLoS ONE</i> </span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">3</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0009490">10.1371/journal.pone.0009490</a> </span> <span class="pub-id" /> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B18" /> <span class="label" tagx="label" title="label">18</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shannon</span> <span class="given-names" tagx="given-names" title="given-names">C. E.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Communication theory of secrecy systems</span> <span class="source" tagx="source" title="source"> <i>Bell Labs Technical Journal</i> </span> <span class="year" tagx="year" title="year">1949</span> <span class="volume" tagx="volume" title="volume">28</span> <span class="fpage" tagx="fpage" title="fpage">656</span> <span class="lpage" tagx="lpage" title="lpage">715</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/j.1538-7305.1949.tb00928.x">10.1002/j.1538-7305.1949.tb00928.x</a> </span> <span class="pub-id" /> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B19" /> <span class="label" tagx="label" title="label">19</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simpson</span> <span class="given-names" tagx="given-names" title="given-names">E. H.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Measurement of diversity</span> <span class="source" tagx="source" title="source"> <i>Nature</i> </span> <span class="year" tagx="year" title="year">1949</span> <span class="volume" tagx="volume" title="volume">163, article 688</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/163688a0">10.1038/163688a0</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B20" /> <span class="label" tagx="label" title="label">20</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Simberloff</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Properties of the Rarefaction Diversity Measurement</span> <span class="source" tagx="source" title="source"> <i>The American Naturalist</i> </span> <span class="year" tagx="year" title="year">1972</span> <span class="volume" tagx="volume" title="volume">106</span> <span class="issue" tagx="issue" title="issue">949</span> <span class="fpage" tagx="fpage" title="fpage">414</span> <span class="lpage" tagx="lpage" title="lpage">418</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1086/282781">10.1086/282781</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="label" tagx="label" title="label">21</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chao</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nonparametric estimation of the number of classes in a population</span> <span class="source" tagx="source" title="source"> <i>Scandinavian Journal of Statistics</i> </span> <span class="year" tagx="year" title="year">1984</span> <span class="volume" tagx="volume" title="volume">11</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">265</span> <span class="lpage" tagx="lpage" title="lpage">270</span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B22" /> <span class="label" tagx="label" title="label">22</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meyer</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paarmann</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D'Souza</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The metagenomics RAST server—a public resource for the automatic phylogenetic and functional analysis of metagenomes </span> <span class="source" tagx="source" title="source"> <i>BMC Bioinformatics</i> </span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">article 386</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-9-386">10.1186/1471-2105-9-386</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B23" /> <span class="label" tagx="label" title="label">23</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayek</span> <span class="given-names" tagx="given-names" title="given-names">L. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buzas</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> </span> <span class="source" tagx="source" title="source"> <i>Surveying Natural Populations: Quantitative Tools for Assessing Biodiversity</i> </span> <span class="year" tagx="year" title="year">1997</span> <span class="volume" tagx="volume" title="volume">563</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Columbia University Press</span> <span class="pub-id"> <a href="https://dx.doi.org/10.7312/haye14620">10.7312/haye14620</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="label" tagx="label" title="label">24</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lozupone</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">UniFrac: a new phylogenetic method for comparing microbial communities</span> <span class="source" tagx="source" title="source"> <i>Applied and Environmental Microbiology</i> </span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">71</span> <span class="issue" tagx="issue" title="issue">12</span> <span class="fpage" tagx="fpage" title="fpage">8228</span> <span class="lpage" tagx="lpage" title="lpage">8235</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/aem.71.12.8228-8235.2005">10.1128/aem.71.12.8228-8235.2005</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16332807">16332807</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="label" tagx="label" title="label">25</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lozupone</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Global patterns in bacterial diversity</span> <span class="source" tagx="source" title="source"> <i>Proceedings of the National Acadamy of Sciences of the United States of America</i> </span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">104</span> <span class="issue" tagx="issue" title="issue">27</span> <span class="fpage" tagx="fpage" title="fpage">11436</span> <span class="lpage" tagx="lpage" title="lpage">11440</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0611525104">10.1073/pnas.0611525104</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="label" tagx="label" title="label">26</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tenesa</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Farrington</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prendergast</span> <span class="given-names" tagx="given-names" title="given-names">J. G. D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21 </span> <span class="source" tagx="source" title="source"> <i>Nature Genetics</i> </span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">631</span> <span class="lpage" tagx="lpage" title="lpage">637</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ng.133">10.1038/ng.133</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18372901">18372901</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="label" tagx="label" title="label">27</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gotelli</span> <span class="given-names" tagx="given-names" title="given-names">N. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Colwell</span> <span class="given-names" tagx="given-names" title="given-names">R. K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Quantifying biodiversity: procedures and pitfalls in the measurement and comparison of species richness </span> <span class="source" tagx="source" title="source"> <i>Ecology Letters</i> </span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">4</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">379</span> <span class="lpage" tagx="lpage" title="lpage">391</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1461-0248.2001.00230.x">10.1046/j.1461-0248.2001.00230.x</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B28" /> <span class="label" tagx="label" title="label">28</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Buzas</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayek</span> <span class="given-names" tagx="given-names" title="given-names">L.-A. C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">On richness and evenness within and between communities</span> <span class="source" tagx="source" title="source"> <i>Paleobiology</i> </span> <span class="year" tagx="year" title="year">2005</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="fpage" tagx="fpage" title="fpage">199</span> <span class="lpage" tagx="lpage" title="lpage">220</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1666/0094-8373(2005)031[0199:ORAEWA]2.0.CO;2">10.1666/0094-8373(2005)031[0199:ORAEWA]2.0.CO;2</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="label" tagx="label" title="label">29</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gloor</span> <span class="given-names" tagx="given-names" title="given-names">G. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hummelen</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macklaim</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbiome profiling by illumina sequencing of combinatorial sequence-tagged PCR products</span> <span class="source" tagx="source" title="source"> <i>PLoS ONE</i> </span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0015406">10.1371/journal.pone.0015406</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B30" /> <span class="label" tagx="label" title="label">30</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Indugu</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bittinger</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vecchiarelli</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pitta</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A comparison of rumen microbial profiles in dairy cows as retrieved by 454 Roche and Ion Torrent (PGM) sequencing platforms </span> <span class="source" tagx="source" title="source"> <i>PeerJ 4</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">2016</span> <span class="issue" tagx="issue" title="issue">2, article no. 1599</span> <span class="pub-id"> <a href="https://dx.doi.org/10.7717/peerj.1599">10.7717/peerj.1599</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B31" /> <span class="label" tagx="label" title="label">31</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caporaso</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lauber</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Walters</span> <span class="given-names" tagx="given-names" title="given-names">W. A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample</span> <span class="source" tagx="source" title="source"> <i>Proceedings of the National Acadamy of Sciences of the United States of America</i> </span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">108</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">4516</span> <span class="lpage" tagx="lpage" title="lpage">4522</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.1000080107">10.1073/pnas.1000080107</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="label" tagx="label" title="label">32</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vreeland</span> <span class="given-names" tagx="given-names" title="given-names">R. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosenzweig</span> <span class="given-names" tagx="given-names" title="given-names">W. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Powers</span> <span class="given-names" tagx="given-names" title="given-names">D. W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Isolation of a 250 million-year-old halotolerant bacterium from a primary salt crystal</span> <span class="source" tagx="source" title="source"> <i>Nature</i> </span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">407</span> <span class="issue" tagx="issue" title="issue">6806</span> <span class="fpage" tagx="fpage" title="fpage">897</span> <span class="lpage" tagx="lpage" title="lpage">900</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/35038060">10.1038/35038060</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11057666">11057666</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="label" tagx="label" title="label">33</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jang</span> <span class="given-names" tagx="given-names" title="given-names">L. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">P. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Findley</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yen</span> <span class="given-names" tagx="given-names" title="given-names">T. F.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Selection of bacteria with favorable transport properties through porous rock for the application of microbial-enhanced oil recovery </span> <span class="source" tagx="source" title="source"> <i>Applied and Environmental Microbiology</i> </span> <span class="year" tagx="year" title="year">1983</span> <span class="volume" tagx="volume" title="volume">46</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="fpage" tagx="fpage" title="fpage">1066</span> <span class="lpage" tagx="lpage" title="lpage">1072</span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16346414">16346414</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="label" tagx="label" title="label">34</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clark</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Munnecke</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenneman</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Insitu microbial enhancement of oil production</span> <span class="source" tagx="source" title="source"> <i>Developments in Industrial Microbiology</i> </span> <span class="year" tagx="year" title="year">1981</span> <span class="volume" tagx="volume" title="volume">22</span> <span class="fpage" tagx="fpage" title="fpage">695</span> <span class="lpage" tagx="lpage" title="lpage">701</span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="label" tagx="label" title="label">35</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McInerney</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Menzie</span> <span class="given-names" tagx="given-names" title="given-names">D. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jenneman</span> <span class="given-names" tagx="given-names" title="given-names">G. E.</span> </span> </span> <span class="source" tagx="source" title="source"> <i>The Use of Microorganisms in Enhanced Oil Recovery</i> </span> <span class="year" tagx="year" title="year">1983</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">US Department of Energy</span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="label" tagx="label" title="label">36</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandey</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sree</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dash</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sethi</span> <span class="given-names" tagx="given-names" title="given-names">D. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chowdhury</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Diversity of marine bacteria producing beta-glucosidase inhibitors</span> <span class="source" tagx="source" title="source"> <i>Microbial Cell Factories</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">12</span> <span class="issue" tagx="issue" title="issue">1, article no. 35</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1475-2859-12-35">10.1186/1475-2859-12-35</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="label" tagx="label" title="label">37</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aravindraja</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Viszwapriya</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karutha Pandian</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ultradeep 16S rRNA sequencing analysis of geographically similar but diverse unexplored marine samples reveal varied bacterial community composition </span> <span class="source" tagx="source" title="source"> <i>PLoS ONE</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="issue" tagx="issue" title="issue">10</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0076724">10.1371/journal.pone.0076724</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B38" /> <span class="label" tagx="label" title="label">38</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L.-Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Duan</span> <span class="given-names" tagx="given-names" title="given-names">R.-Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J.-F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">S.-Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">J.-D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mu</span> <span class="given-names" tagx="given-names" title="given-names">B.-Z.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Molecular analysis of the microbial community structures in water-flooding petroleum reservoirs with different temperatures </span> <span class="source" tagx="source" title="source"> <i>Biogeosciences</i> </span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="issue" tagx="issue" title="issue">11</span> <span class="fpage" tagx="fpage" title="fpage">4645</span> <span class="lpage" tagx="lpage" title="lpage">4659</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.5194/bg-9-4645-2012">10.5194/bg-9-4645-2012</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="label" tagx="label" title="label">39</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">S.-S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z.-Z.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of bacterial diversity in two oil blocks from two low-permeability reservoirs with high salinities </span> <span class="source" tagx="source" title="source"> <i>Scientific Reports</i> </span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1038/srep19600">10.1038/srep19600</a> </span> <span class="pub-id" /> </span> </li> <li tag="ref"> <a name="B40" /> <span class="label" tagx="label" title="label">40</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">H. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">G. D.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Soil moisture effect on bacterial and fungal community in Beilu River (Tibetan Plateau) permafrost soils with different vegetation types </span> <span class="source" tagx="source" title="source"> <i>Journal of Applied Microbiology</i> </span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">114</span> <span class="issue" tagx="issue" title="issue">4</span> <span class="fpage" tagx="fpage" title="fpage">1054</span> <span class="lpage" tagx="lpage" title="lpage">1065</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/jam.12106">10.1111/jam.12106</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23241008">23241008</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="label" tagx="label" title="label">41</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albokari</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mashhour</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alshehri</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boothman</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Enezi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Characterization of microbial communities in heavy crude oil from Saudi Arabia</span> <span class="source" tagx="source" title="source"> <i>Annals of Microbiology</i> </span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">65</span> <span class="issue" tagx="issue" title="issue">1</span> <span class="fpage" tagx="fpage" title="fpage">95</span> <span class="lpage" tagx="lpage" title="lpage">104</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s13213-014-0840-0">10.1007/s13213-014-0840-0</a> </span> <span class="pub-id" /> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25705147">25705147</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="label" tagx="label" title="label">42</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allamin</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ijah</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ismail</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Riskuwa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Occurrence of hydrocarbon degrading bacteria in soil in Kukawa, Borno State</span> <span class="source" tagx="source" title="source"> <i>International Journal of Environment</i> </span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="issue" tagx="issue" title="issue">2</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3126/ije.v3i2.10503">10.3126/ije.v3i2.10503</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="label" tagx="label" title="label">43</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Sayegh</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Wahaibi</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Bahry</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elshafie</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Bemani</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbial enhanced heavy crude oil recovery through biodegradation using bacterial isolates from an Omani oil field </span> <span class="source" tagx="source" title="source"> <i>Microbial Cell Factories</i> </span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="issue" tagx="issue" title="issue">1, article no. 141</span> <span class="pub-id" /> <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12934-015-0330-5">10.1186/s12934-015-0330-5</a> </span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>(a) Purified gDNA from isolates electrophoresed on a 1.5% agarose gel and stained with ethidium bromide. First lane contains the 10000 bp molecular weight markers from 300 bp to 10200 bp in length. Other lanes are gDNA samples extracted from the isolates using PowerSoil DNA Isolation Kit; (b) DNA extracted from 10 heavy crude soil samples in 2% agarose gel. Lane 1 is the 1 Kb DNA ladder. 1–10 represent the gDNA from 10 soil samples. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>16S rDNA amplified product electrophoresed on a 2% agarose gel and stained with ethidium bromide. First lane contains the 10000 bp molecular weight markers from 300 bp to 10200 bp in length. Other lanes are PCR products amplified using 27F and 1492R primers. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Dendrogram of the isolates using maximum likelihood method (the green dots represent the isolates). The length of each tree segment in the dendrogram represents in units of expected nucleotide substitutions per sites. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Rarefaction curves showing Shannon diversity index and Chao1 at various sequencing depths. The curve reached a plateau at approximately 5000 sequences. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Relative abundance (%) of bacteria phyla. Firmicutes were found to be the most abundant phyla in all the soil samples. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>Relative abundance (%) of bacterial genera. <i> Bacillus</i> was the most abundant genus followed by <i> Paenibacillus</i> and <i> Brevibacillus</i>. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 7</span> </a> </div>   <p>Dual hierarchal dendrogram of the taxonomic classification data, with each sample clustered on the <i>x</i>-axis. The relative abundance of <i> Bacillus</i>, <i> Paenibacillus</i>, and <i> Brevibacillus</i> was highest in all the 10 soil samples and the soil samples with more similar microbial populations being mathematically clustered closer together. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 8</span> </a> </div>   <p>SHE analysis of soil samples showing constant proportion of species in all 10 samples.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 9</span> </a> </div>   <p>Weighted UniFrac generated (a) Jackknifed UPGMA clustering of soil samples based on the observed OTUs (b) 2D PCoA. </p>   </div> <div class="table-wrap_UNKNOWN" id="tab1" orientation="portrait" position="float"> <span class="label" tagx="label" title="label">Table 1</span>  <p>NCBI GenBank accession numbers for the identified isolates.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Accession No.</th> <th align="center" rowspan="1" colspan="1">Identification</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">KP119097</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus subtilis </i>strain BG1 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119098</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis </i>strain BG2 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119099</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus foraminis </i>strain BG3 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119100</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus firmus </i>strain BG4 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119101</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus halodurans </i>strain BG5 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119102</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus subtilis </i>strain BG7 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119103</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus halodurans </i>strain BG8 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119104</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis </i>strain BG9 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119105</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis </i>strain BG10 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119106</td> <td align="center" rowspan="1" colspan="1"> <i>Paenibacillus ehimensis </i>strain BS1 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119107</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus subtilis </i>strain BS2 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119108</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus subtilis </i>strain BS3 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119109</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus subtilis</i> strain BS4 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119110</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis </i>strain BS5 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119111</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis </i>strain BS6 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119112</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus subtilis </i>strain BS7 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119113</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis </i>strain BS8 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119114</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis </i>strain BS9 </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">KP119115</td> <td align="center" rowspan="1" colspan="1"> <i>Bacillus licheniformis</i> strain BS10 </td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="tab2" orientation="portrait" position="float"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Shannon index and Simpson index.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Samples</th> <th align="center" rowspan="1" colspan="1">Shannon index ( <i>H</i>′) </th> <th align="center" rowspan="1" colspan="1">Simpson index ( <i>D</i>) </th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">(1)</td> <td align="center" rowspan="1" colspan="1">2.67</td> <td align="center" rowspan="1" colspan="1">0.89</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(2)</td> <td align="center" rowspan="1" colspan="1">1.81</td> <td align="center" rowspan="1" colspan="1">0.73</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(3)</td> <td align="center" rowspan="1" colspan="1">2.26</td> <td align="center" rowspan="1" colspan="1">0.81</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(4)</td> <td align="center" rowspan="1" colspan="1">2.61</td> <td align="center" rowspan="1" colspan="1">0.86</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(5)</td> <td align="center" rowspan="1" colspan="1">2.69</td> <td align="center" rowspan="1" colspan="1">0.90</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(6)</td> <td align="center" rowspan="1" colspan="1">2.08</td> <td align="center" rowspan="1" colspan="1">0.79</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(7)</td> <td align="center" rowspan="1" colspan="1">2.42</td> <td align="center" rowspan="1" colspan="1">0.85</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(8)</td> <td align="center" rowspan="1" colspan="1">2.03</td> <td align="center" rowspan="1" colspan="1">0.71</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(9)</td> <td align="center" rowspan="1" colspan="1">2.78</td> <td align="center" rowspan="1" colspan="1">0.90</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(10)</td> <td align="center" rowspan="1" colspan="1">2.37</td> <td align="center" rowspan="1" colspan="1">0.84</td> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="tab3" orientation="portrait" position="float"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Weighted UniFrac significance test with Bonferroni correction.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="1" colspan="1">Sample</th> <th align="center" rowspan="1" colspan="1">1</th> <th align="center" rowspan="1" colspan="1">2</th> <th align="center" rowspan="1" colspan="1">3</th> <th align="center" rowspan="1" colspan="1">4</th> <th align="center" rowspan="1" colspan="1">5</th> <th align="center" rowspan="1" colspan="1">6</th> <th align="center" rowspan="1" colspan="1">7</th> <th align="center" rowspan="1" colspan="1">8</th> <th align="center" rowspan="1" colspan="1">9</th> <th align="center" rowspan="1" colspan="1">10</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">(1)</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(2)</td> <td align="center" rowspan="1" colspan="1">0.14</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(3)</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.01 <sup> <i>∗</i> </sup> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(4)</td> <td align="center" rowspan="1" colspan="1">0.87</td> <td align="center" rowspan="1" colspan="1">0.00 <sup> <i>∗</i> </sup> </td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(5)</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.22</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.95</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(6)</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.00 <sup> <i>∗</i> </sup> </td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(7)</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.75</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.4</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(8)</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.00 <sup> <i>∗</i> </sup> </td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(9)</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.00 <sup> <i>∗</i> </sup> </td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.63</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1"> </td> <td align="center" rowspan="1" colspan="1"> </td> </tr> <tr> <td align="left" rowspan="1" colspan="1">(10)</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">0.00 <sup> <i>∗</i> </sup> </td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1">1</td> <td align="center" rowspan="1" colspan="1"> </td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <sup> <i>∗</i> </sup> <i>p</i> &amp;lt; 0.05: significant. </p> </div> </div> </div> </div>  </body></html>